This manuscript
(\href{https://scherzerthesis.github.io/thesis/v/ea3aab225306cad9bf38fdfcc0c9e01659bbf0fb/}{permalink})
was automatically generated
from \href{https://github.com/scherzerthesis/thesis/tree/ea3aab225306cad9bf38fdfcc0c9e01659bbf0fb}{scherzerthesis/thesis@ea3aab2}
on April 1, 2022.

\hypertarget{authors}{%
\subsection{Authors}\label{authors}}

\begin{itemize}
\tightlist
\item
  \textbf{Michael}
  \includegraphics[width=0.16667in,height=0.16667in]{images/orcid.svg}
  \href{https://orcid.org/XXXX-XXXX-XXXX-XXXX}{XXXX-XXXX-XXXX-XXXX}
  · \includegraphics[width=0.16667in,height=0.16667in]{images/github.svg}
  \href{https://github.com/johndoe}{johndoe}
  · \includegraphics[width=0.16667in,height=0.16667in]{images/twitter.svg}
  \href{https://twitter.com/johndoe}{johndoe}
  Department of Something, University of Whatever
  · Funded by Grant XXXXXXXX
\end{itemize}

\hypertarget{chapter-one-introduction}{%
\subsection{Chapter One: Introduction}\label{chapter-one-introduction}}

\textbf{NSCLC} Lung cancer is a global hazard on human health. The historic consumption of cigarette smoke has caused lung cancer to be the most common and deadly form of cancer. Fortunately, smoking-rates have decreased in the past 30 years with lung cancer rates also on the decline. However, even if cigarette consumption were eradicated, lung cancer would still significantly affect human health. Indeedm it is estimated that approximately 25\% of all lung cancer cases arise in never smokers (Warren and Cummings, 2013). Secondary causes of lung cancer include Radon exposure, high-heat cooking, family history, air pollution (Malhotra et al., 2016). An individual's risk of lung cancer also increases with age due to decreases in the efficiency of tumor supressive mechanisms such as tumor immunity and apoptosis(de Groot and Munden, 2012).

Lung cancer can be broadly classified into two major histological categories that are named based on how the malignant cells look under a microscope: Non-small Cell Lung Cancer (NSCLC) and Small Cell Lung Cancer (SCLC). Within NSCLC, tumors are further characterized based on histological structures, anatomical location, cell-of-origin, as well as genetic drivers(Herbst et al., 2008). For example, lung adenocarcinoma is the most common subtype and is characterized by glandular and papillary structures while squamous cell lung carcinoma is characterized by keratin-pearls(Nikitin et al., 2004). Moreover, adenocarcinomas arise from alveolar-type-2 (AT2) cells in the distal lung and alveoli and are typified by genomic alterations throughout the MAPK pathway, such as EGFR, KRAS, BRAF, PI3K (Ferone et al., 2020; Jackson et al., 2001). On the other hand, squamous cells carcinomas likely arise from basal cells and typically harbor SOX2 gene amplification (Mollaoglu et al., 2018; Mukhopadhyay and Oliver, 2014).

The 5-year survival rate for patients with lung cancer was 25\% in 2020 but varies depending on stage of disease at time of diagnosis (Siegel et al., 2021).Like other solid cancers, lung cancers are most deadly after primary tumors have metastaized to other organ systems like the liver, kidney, bone, or nervous system (Matsunaga and Shida, 1971). Lung cancer patients with no local or distant metastasis typically have a 5-year survival percentage of approximately 40\% compared to approximately 5\% with distant metastasis{[}Woodard et al.; 27740970{]}. Ultimately, the cause of death of many lung cancer patients include primary lung tumor burden, infection, pulmonary embolism or hemorrhage, or complications of metastatic dissemination(Nichols et al., 2012). Although patient survival has improved due to improved targeted and immune therapies, the still poor prognosis reflects a need to better understand the molecular mechanisms underlying lung adenocarcinoma initiation, progression, maintenance and response to targeted- or immune-therapy.

The current standard of care for Lung adenocarcinoma patients are often a combination of several conventional chemotherapeutics including Cisplatin, Carboplatin, Paclitaxel, or Pemetrexed(Ghimessy et al., 2020). Fortunately for patients who have been identified to have EGFR, ALK, ROS, TRK, or BRAF (V600E) genetic alternations are often offered pathway-targeted therapies(Arteaga and Engelman, 2014; Planchard et al., 2016). As of late, many patients receive immunotherapy first, followed by pathway-targeted therapy after progression on immunotherapy. Furthermore, in the last seven years, exciting therapeutic options for \emph{KRAS} mutant lung cancer have been developed and tested, such as a combination of MEK-inhibition and autophagy-inhibtion, and direct inhibitors of \emph{KRAS} G12c and \emph{KRAS} G12D (Kinsey et al., 2019; Reita et al., 2022; Zheng et al., 2022)

\textbf{MAPK signaling} standard activation from the top of he MAPK pathway begins with extracellular ligands (EGF, NGF, etc.) binding to Receptor-Tyrosine Kinases (RTKs). This leads to an loading of RAS with GTP, which activates RAS. Many RAS mutations lead to either more efficient loading of GTP or less effciency GTP hydrolysis which both serve to activate RAS. GTP-loaded RAS recruits and activates RAF kinases (ARAF, BRAF, CRAF) which then phosphorylates and activates MEK1/2 kinases which then phosphorylates and activates ERK1/2. This ultimately leads to the activation of a suite of cellular processes that are required for proliferation, differentiation, migration, and cell survival. Therefore, activating mutation in this pathway, from RTKs to kinases, serve to keep the pathway in a constitutively active state. For example, the most common mutation in \emph{BRAF} is V600E. This valine to glutamic acid substitution is located in the kinase domain and mimics phosphorylation, allowing BRAF(V600E) to become independet of upstream regulations like RAS. This active state drives many of the hallmarks of cancer that is required for tumor formation. It is important to note that oncogenic activation of the MAPK pathway differs from normal ligand-mediated pathway activation. For example, mitogens, such as EGF, only transiently activate the pathway even with constant ligand stimulation. This immediate and strong spike in pathway activation is integrated by various biological processes in the cell that results in transcription of certain target genes. In contrast, oncogenic signaling thru mutational activation of BRAF(V600E) results in constant kinase signaling that is integrated differently than normal pathway stimulation. Thus, an oncogene-specific transcriptional program is activated.

The importance of this pathway is reflected in the strong anti-tumor responses that are seen when small molecule inhibitors are used to target various molecules involved in the MAPK pathway. For example, Dabrafenib and Trametinib, inhibitors of BRAF and MEK kinases, respectfully, are used in patients that harbor BRAF(V600E) melanomas and lung cancers(Planchard et al., 2021; Robert et al., 2014). Although there is strong pre-clinical and clinical evidence that BRAF+MEK blockade can lead to tumor regression, not all patients respond to treatment(Tabbò et al., 2021). Therefore, to see a complete response, multiple pathways need to be targeted, such as PI3K or autophagy (Deuker et al., 2014; Kinsey et al., 2019; Truong et al., 2020).

Great effort has elucidated the proto-oncogenes that drive cancer formation when mutated, such as \emph{MYC}, \emph{RAS}, \emph{PI3K}, and others. Yet we still do not fully understand the mechanistic details that determine how genes cooperate to drive cancer.

\textbf{Genetically-engineered mouse models} Genetically-Engineered mouse models (GEMMs) of human cancer have allowed for the reproducible analysis of cancer biology driven by specific mutations frequently found in cancer patients. These models are helpful in understanding the molecular mechanisms that drive tumor formation and testing tumor responses to targeted or immune therapies. Early mouse models were not engineered but instead selected inbred mouse strains that were natural hosts of tumor transplantation(Strong, 1978). However, these models suffer from long latencies, incomplete pentrance, and often not relevant to human cancers. Therefore, carcinogen-induced models such as UV, Urethane, radiation, cigarette consumption, or MNU treatment models were developed to ressemble carinogene-induced human cancers(Kemp et al., 1994; Westcott et al., 2014).After techniques were developed to genetically manipulate embryonic stem-cells came the ability to engineer mice with specific mutations that could drive tumor formation. Since genes involved in cancer are also important for normal mouse development, conditional gene manipulation was developed to allow for temporal control.

The most commonly used conditional gene expression system relies on inducible transcription factors, such as tetracycline-dependent regulatory system, where the treatment of mice with doxycycline toggles gene transcriptial levels of transgenes. More recently, Cre-Lox recombination strategies have been utilized to get more cell type specific control. In this system, genes of interested are constructed to contain LoxP-sites that flank a particular genetic element and upon delivery of Cre-recombinase the DNA sequence in between LoxP sites are excised. With this approach, one can knockout a gene by engineering LoxP sites around critical exons, or one could induce gene transcription by flanking a strong stop signal such as a polyadenylation sequence upstream of a gene-of-interest sequence such as KRAS(G12D)(Jackson et al., 2001).

Genetically engineered mouse models of human cancer are important preclinical models, because they resemble the physiological environment of tumor growth in which tumors arise as progeny from a single initiating cell. These tumors can approximate the genetic alterations, transcriptional landscape, histology, and responsive disposition or lack therefore, seen in human cancers(cite). However, these model often take months to develop tumors and compound genetic alterations take time to develop. Later in this thesis, we will utilize TUBA-SEQ to both quantify tumor burden and cooperation of BRAF(V600E) with other common alterations in lung cancer.

Due to the robust connection between lung adenocarcinomas and mutational activation of genes involved in the MAPK-pathway, such as \emph{EGFR}, \emph{KRAS}, and \emph{BRAF}, many labs have generated genetically-engineered mouse models (GEMMs) that harbor conditionally-activated cancer specific mutant alleles of either EGFR (L858R), KRAS (G12D), or BRAF(V600E).

Our lab has previously developed a conditional mouse model of BRAF (V600E) human lung cancers(Dankort et al., 2007; Shai et al., 2015; van Veen et al., 2019). In the \emph{Braf(CAT)} model, normal BRAF is expressed from a conditional allele prior to Cre-mediated recombination . LoxP sites flank human \emph{BRAF} cDNA encoding normal \emph{BRAF} exons 15-18. Downstream of the LoxP sites is the mutant exon 15 that encodes the murine equivalent of the T1799A mutation that gives rise to the BRAF (V600E) onco-protein. Downstream of the mutant exon there is a P2A element and a CAAX-tagged \emph{TdTomato}. Therefore, after Cre-mediated recombination, the BRAF (V600E) oncoprotein and fluorophore TdTomato is expressed at normal levels(van Veen et al., 2019).

\textbf{BRAF (V600E) Tumorigenesis and Cooperating Events for Progression} Mutational activation of the MAPK pathway forms lung tumors in mice. specifically, expression of BRAF (V600E) in the Surfactant-Protein C expression cells (alveolar type 2 pneumocytes) elicits benign tumor formation that fails to progress to lung adenocarcinoma. It is hypothesized that the cell cycle arrest is dependent on the tumor suppressors P53 and CDKN2A. Loss of either P53 or CDKN2A allows tumors to progress to malignant and deadly lung adenocarcinoma. Also, activation of WNT signaling and PI3K signaling similarly allows BRAF (V600E) adenomas to progress to adenocarcinoma (Green et al., 2015; Juan et al., 2014; van Veen et al., 2019). These observations are in concordance to the "multiple-hit" hypothesis that states that cancers do not arise from single mutagenic events, but are the consequence of sequential assaults on the genome that activate more than one oncogenic pathway. These multiple genetic hits allow cells to adopt traits to form cancer, which is coined by Bob Weinberg and Douglas Hanahan as the "Hallmarks of Cancer" (Hanahan, 2022; Hanahan and Weinberg, 2011). The traits include suppression of apoptosis, active proliferation, altered metabolism, changes in cell identity, and forming blood-vessels to supply tumor cells with adequate resources for growth(Hanahan, 2022). BRAF(V600E) expression alone is sufficient to activate many of the hallmarks of cancer but additional pathways need to be activated, or suppresed, to develop to malignant tumors. Great effort has elucidated the proto-oncogenes that drive cancer formation when mutated, such as \emph{MYC}, \emph{RAS}, \emph{PI3K}, and others. Yet we still do not fully understand the mechanistic details that determine how genes cooperate to overide tumor suppressive mechanisms and drive cancer progression.

\textbf{Talk here about the prevalence of mutations in the egfr-ras-raf-mek-erk pathway}

\textbf{Identification of this pathway as central to luad tumorigenesis}

\textbf{Development of gemms to study BRAF\textsuperscript{V600E}}

\textbf{Proliferation arrest by this model}

\textbf{How does p53 loss lead to sustained wnt-signalling? Do dominant negative recapitulate?}

\textbf{P53- mediated tumor suppression}
Cancers pose an evoltionary challenge for organisms regardless of their life span. Therefore, supression of the outgrowth of neoplastic cells is built into many organisms. One such mechanism is the engagment of the p53 pathway. At the time of writing this dissertation, there are approximately 105,000 manuscripts on PUBMED that mention P53. P53 is so well studied because of both its apparent role in tumor suppression, but also because it is the most frequently altered gene in human cancer.

The p53 protein was originally discovered as a protein that bound large T antigene of the SV40 virus in infected cells (Linzer and Levine, 1979).P53 is a transcriptional factor that has a DNA-binding domain that specifically recognizes two decameric half-sites. P53 also contains two N-terminal transactivation domains and a C-terminal oligomerization domain that are critical for P53-target gene activation(Laptenko and Prives, 2006). Central to the ability of P53 to induce transcriptional activation is tetramerization. Thus, if one or more P53 proteins in the complex is compromised in transactivation or DNA binding, transcriptional activity is compromised. Therefore, if one copy of P53 is mutated, the organism's ability to suppress tumor formation is compromised.

The importance of p53 is best demonstrated in people, and in mice, that have just one mutated copy of P53. In humans with the familial- inherited Li-Fraumeni syndrome, in which they have one or more mutations in p53 are almost certain to develop cancer early on and throughout their life(Li et al., 1988). Similarly, mice lacking two function P53 genes are prone to leukemias and lymphomas and have a shortened lifespan due to cancer incidence(Donehower et al., 1992). On the other hand, organisms such as the african and asian elephants, which have an abnormally low level of cancer incidence, have been shown to higher multiple copies of \emph{TP53} in their genomes(Abegglen et al., 2015).

The most well-known cellular functions of P53 include its ability to induce cell-cycle arrest in response to DNA damage(el-Deiry et al., 1993). Many stress signals, including oncogene activation has been shown to stimulate a reversible or irreversible cell cycle arrest(McNeal et al., 2021; Zhu et al., 1998). However, the strength of P53 induced cell cycle arrest or apoptosis is likely cell type- and cellular stress- specific and is a active area of p53 research.

Once a cell encounters a stressor such as DNA damage or oncogene activation, P53 is stabilized and accumulates in the nucleus to activate a suite of target genes that can lead to a number of cellular phenotypes, such as cell-cycle arrest or apoptosis.

P53 was originally thought to be an oncogene as P53 is accumulated in lung cancer, which is not common in normal tissues(Bartek et al., 1990; Iggo et al., 1990). Moreover, ectopic expression of a P53 cDNA was found to aide in the transformation of primary cells induced by RAS. However, early studies erroneously used mutated P53 instead of wild-type leading to the misclassification of P53 as an oncogene. We now know P53 suppresses cell growth and transformation(Levine and Oren, 2009).

Often times, P53 is mutated in the DNA-binding domain at sites commonly referred to as "hotspots" due to there extraordinary frequency. These hotspot mutations poison the ability of p53 to bind DNA therefore blocking its function. Interestingly, since one mutated P53 can disrupt the entire P53 tetramer complex, a single mutation can exert dominant-negative effects by inhibiting the normal tumor suppressive functions of P53. Although one mutant P53 allele is enough to compromised transcriptional activity, there is still selective pressure to lose the other wild -type copy. Loss of hemizygosity implies there is still residual tumor suppression in the presence of a wild-type P53 allele(Muzumdar et al., 2016).

Early on in P53 studies, there were reports of certain P53 mutations having a "gain-of-function" (GOF) effect. For example, Li-Fraumeni patients with certain missense mutations in P53 would develop tumors earlier than Li-Fraumeni patients with loss-of-function (LOF) mutations (Bougeard et al., 2008). Furthermore, there were experimental cell biology studies that would express mutant-p53 in P53-null cells and demonstrate enhanced tumorigenic potential (Levine, 2020). Further evidence for GOF mutant-P53 in mice indicated that missense mutant P53 induced different cancer types and enhanced metastasis than LOF P53 (Olive et al., 2004). Furthermore, many groups have shown mutant-p53 can alter signal transduction affecting chemoresistance and altering metabolism.

The cellular mechanisms that GOF P53 are involved in are also well-characterized but context-dependent. For example, in Pancreatic cancers driven by KRAS(G12D) Mutant P3 interacts with CREB to induce FOXA1 transcription which enhances Beta-Catenin signaling to augment liver metastasis (Kim et al., 2021).

\hypertarget{chapter-two-quantitative-tumor-burden-and-small-library-screen-using-tuba-seq}{%
\subsection{Chapter Two: Quantitative tumor burden and small library screen using TUBA-SEQ}\label{chapter-two-quantitative-tumor-burden-and-small-library-screen-using-tuba-seq}}

\{.page\_break\_before\}

\hypertarget{abstract}{%
\section{Abstract}\label{abstract}}

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

Next generation sequencing has allowed for the unbiased identification of genes that are mutated in human cancer (2014; Hoadley et al., 2014; Kandoth et al., 2013).
The wealth of knowledge gained from such large-scale sequencing projects has revealed actionable mutationas that can be targeted with small molecule inhibitors, such as dabrafenib and trametinib.
However, Identificaton of possible targets is not enough, as many cancers with similarly mutated oncogenes, such as BRAF\textsuperscript{V600E}, respond differentially to BRAF\textsuperscript{V600E}-targeed therapy.
For example, mealanoma patients with BRAF\textsuperscript{V600E} mutations respond well to BRAF\textsuperscript{V600E}-targeted therapy whereas colorectal cancer patients with BRAF\textsuperscript{V600E} mutations do not respond(Corcoran et al., 2012).
The disconnect between putative driver oncogenes identified in human cancers and response to targeted therapy is a problem that can be solved with rigorous pre-clinical experimentation, such as in GEMMs.

Genomic sequencing of human tumors has also been crucial insight into how cancers form and what signaling nodes might be important for progressive stages of cancer evolution.
For example, pancreatic cancers have a very stereotypical histiological progression that is brought about by known mutational activation.
Specifically, mutations in \emph{KRAS} initiate tumorigenesis and loss of \emph{TP53} function and \emph{SMAD3} are associated with cancer progression(Orth et al., 2019).
However, this is more difficult to do in lung adenocarcinoma because of the high mutational burden often due to the consumption of cigarettes(Alexandrov et al., 2016; Yoshida et al., 2020).

Indeed, in highly mutated cancers such as melanoma and lung cancer, recurrent mutations are not necessarily predictive of biological importance.
To better understand the importance of candidate tumor suppressors or oncogenes, researchers exploit tumor cell lines and GEMMs to identify bona-fide genetic drivers of cancer.
However, these systems are suboptimal, as they are either not physiologically relevant (tissue culture) or are not scalable to determine the effect of many mutations (GEMMs).
Moreover, techniques to measure cooperation of genes to drive lung cancer are qualitative and are not sufficiently quantitative to allow for large-scale studies.
Typically, lung adenocarcinoma is quantified by lung weight or histological analyses such as tumor burden (Percentage of lung that is tumor), tumor size, and immunohistochemistry (IHC) of tumor markers.
Although important and informative, these analyses are not high-throughput and rely on many mice to acheive statistical confidence. Furthermore, there is intrinsic bias when calculating tumor size and tumor burden, as tissue-sectioning results in tumor sections that do not accurately portray true tumor size. Therefore, more advanced techniques are required for large-scale analysis of mouse tumors that are both quantitative and high throughput.

Recently, Monte Winslow and his lab at Stanford University has developed \textbf{Tu}mor- \textbf{Ba}rcode \textbf{Seq}uencing (Tuba-Seq), a rapid, multiplexed, quantitative assay for analyzing the contribution of tumor-suppressor loss \emph{in vivo}.
This technique works by delivering a uniquely barcoded lentiviral endoing Cre recombinase, to activate Cre-mediated oncogenes, and a guide RNA, to inactivate a single tumor suppressor gene.
Therefore, each tumor will be driven by the same oncogene, but will have different tumor suppressors inactivated.
After a period of time (6-24 weeks) tissue is harvested for whole-lung DNA extraction and targeted DNA-sequencing is performed to quantify relative tumor size to spike-in controls(Rogers et al., 2017).
Therefore, contribution of a single tumor suppressor can be quantified in single animal.

BRAF\textsuperscript{V600E} has previously been shown to be a potent driver of lung tumorigenesis(Dankort et al., 2007).
However, tumor fails to progress past adenoma-like tumors.
Given the theory that BRAF-\textsuperscript{V600E} induces growth arrest after an initial burst of neoplastic growth, it would be of interest to know if all tumors grow to a predetermined and absolute size, or if there is a spectrum of tumor sizes that are determined by extrinsic factors such as proximity to blood vessels or certain stromal cells.
Therfore, we used Tuba-Seq combined with CRISPR/CAS9 genome editing to explore the contribution of 11 tumor suppressor genes and 5 controls in the context of our BRAF \textsuperscript{V600E}- driven lung tumor models.

\hypertarget{results}{%
\section{Results}\label{results}}

Adenoviral vectors and lentiviral vectors introduced in the lungs of GEMMs offer temporal control and can be titrated to acheive an optimal level of tumor numerosity.
However, only lentiviral vectors can be used to barcode tumors because they integrate into the genome while adenoviruses do not(DuPage et al., 2009; Rogers et al., 2017).
Together with CRISPR/CAS9 gene editing, lentiviruses have been used to analyzed many genetic interactions of oncogenic \emph{Kras} driven lung adenocarcinoma(Rogers et al., 2017; Winters et al., 2017).
To enable simultaneous actvation of BRAF\textsuperscript{V600E} and loss of tumor supressor genes, we bred \emph{Braf\textsuperscript{CAT}}; \emph{H11b\textsuperscript{LSL-CAS9/+}} mice.
To determine the size of BRAF \textsuperscript{V600E} tumors and the contribution of 11 tumor suppressor genes (\emph{Trp53, Rb1, Cdkn2a, Apc, Rbm10, Stk11, Setd2, Arid1a, Atm, Keap1, Smad4)} on BRAF \textsuperscript{V600E} lung tumorigenesis, we obtained tittered barcoded-lentiviral plasmids that encode Cre-recombinase as well as a sgRNA that inactivates one of eleven tumor suppressors common in lung adenocarcinoma.
All genes tested are putative tumor suppressor genes commonly lost in human lung cancer patients. Each lentiviral solution contained \textasciitilde1000 individually barcoded lentiviruses.
We infected both \emph{LSL-Cas9} and \emph{CAS9} deficient mice in order to determine the contribution of genetic manipulation and/or deletion of the targeted gene.
Mice meant to be analyzed after 8 weeks received a viral titer of 1x10\^{}5 while mice mean to be analyzed after 16 weeks received a viral titer of 1x10\^{}4. After euthanasia, lungs were collected for genomic DNA extraction, and barcode amplification followed by preperation of sequencing libraries.
Before genomic extraction and library preparation, a known quantity of known-barcoded cells was added in order to benchmark sequencing reads to a known cell number(figure).

We determined the mean size of a BRAF\textsuperscript{V600E} driven tumor to be 21,248 thousand cells (figure).
Interestingly, there is a widespread distribution of tumor sizes with the largest BRAF\textsuperscript{V600E} tumor to be \textasciitilde300,000 cells along with many small tumors (figure).

Gross histologic images of experimental lungs are show an increase in tumor burden with both time (Figure xxx) and CAS9 status (figure xxx).
Therefore, based on macroscopic inspection, at least one tumor suppressor appeared to cooperate with BRAF \textsuperscript{V600E} to increase tumor burden.
In order to determine the fitness landscape of any one particular tumor suppressor, we calculated the relative number of tumor cells in a given tumor using the Tuba-Seq genomics pipeline.
After 8 weeks post tumor initiation there no statistically significant contribution of tumor suppressor loss (Figure).
SETD2 loss was trending towards statistical significance.
However, 16 weeks post initiation revealed both SETD2 loss and APC loss cooperated with BRAF \textsuperscript{V600E} to form larger tumors (Figure x).
APC loss was a suspected hit as we have previously shown that activation of Wnt-beta-catenin signaling promoted BRAF\textsuperscript{V600E} tumorigenesis.
As APC loss is predicted to increase the activity of Beta-catenin signaling, this result was unsurprising to us.
However, loss of SETD2 was a novel finding in the context of BRAF \textsuperscript{V600E} lung tumorigenesis.

Given our findings of SETD2 loss increased tumor burden concurrent with BRAF\textsuperscript{V600E} we next validated SETD2 in a new cohort of mice with single-guide RNAs targeting SETD2.
Indeed, loss of SETD2 increased tumor burden from 5\% to over 30\% in some animals.
Since SETD2 antibodies are unreliable in both western blots and immunohistochemistry we used H3K36me3 as a surrogate marker for SETD2 activity by immunohistochemistry.
Whereas 90\% of nuclei in BRAF \textsuperscript{V600E} only tumors were H3K36me3, more than 60\% tumors of mice infected with SETD2 CRISPR virus were negative for H3K36me3.
Furthermore, when classified by H3K36me3-positive or -negative, there is even stronger evidence that SETD2 loss leads to larger tumor formation.

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

The mutational landscape of lung adenocarcinoma is complex due lifelong bombardment on DNA by carcinogen ingested into the distal lung epithelium{[}23770567{]}.
While understanding which altered genes play causal roles in tumor formation and progression is required for personalized medicine, the noisy mutation profiles in lung adenocarcinoma makes it hard to do so.
However, it is clear that there is a central role for the MAPK pathway during the initiation and progression of lung adenocarcinoma (Cicchini et al., 2017).
Oncogenic BRAF\textsuperscript{V600E} is a major driver of lung adenocarcinoma.
However, mouse models with activation of BRAF \textsuperscript{V600E} only lead to benign tumorigenesis(Dankort et al., 2007; Shai et al., 2015).
Although it is known that activation of WNT and PI3k signaling, as well as lost of \emph{Trp53} and \emph{Cdkn2a} genes allow BRAF\textsuperscript{V600E} benign lung tumors to progress to malignant disease, it is a time-consuming and laborious process to effects of individual tummor suppressors in GEMMs.
Therefore, we sought out to systematically test a variety of known tumor suppressors in lung adenocarcinoma.
Furthermore,until now, we did not know the absolute number of cells in a BRAF\textsuperscript{V600E} adenoma, only its area, which is confounded by tissue sectioning as well as stromal cell contamination.
Prior to Tuba-Seq, it was very difficult to accurately determine the size of BRAF\textsuperscript{V600E} adenomas.
Based on population doublings, as well as cross sections of BRAF \textsuperscript{V600E} tumors in histological sections, it is estimated that BRAF\textsuperscript{V600E}-driven lung adenomas grow about 15-20 populations doublings and reaching a size of 1mm\textsuperscript{3}(Juan et al., 2014).
With Tuba-Seq, we accurately discerned the absolute size of all BRAF\textsuperscript{V600E} lung adenomas, regardless of size.
Interestingly, our results have revealed BRAF\textsuperscript{V600E}-driven tumors widely differ in absloute tumor cell number. Our results support the hypothesis that each tumor-initiating cell has its own unique barriers for tumor outgrowth.
This heterogenous reponse to BRAF\textsuperscript{V600E} activation likely reflects intrinisic differences between individual cell states.
Indeed, tumorigenic potential can vary greatly due to different cell types (AT2 vs Club cell vs AT1), or can be influenced by subtle differences within a cell type, such proximity to supporting stromal cells such as fibroblast or immune cells.
Since we used a ubiquitous promoter to drive BRAF\textsuperscript{V600E} and CAS9 expression, it is likely that not all sequence reads come from adenomas or tumors, but rather from normal cell turnover during tissue homeostasis.
Therefore, in out quantifcations and analysis, we focused on tumors that were greater than 5,000 cells.

Combining Tuba-Seq with CRISPR/CAS9 gene editing of eleven putative tumor supressor genes with four inert control sgRNAs, we identified both APC and SETD2 loss as a cooperating genetic event that allows BRAF\textsuperscript{V600E} tumors to grow larger.
Therefore, both APC and SETD2 are bona fide tumor suppressors in lung adenocarinoma.
APC loss is known to activate the Wnt-signaling pathway and is a well establish tumor supressor in colon cancer(Powell et al., 1992).
Furthermore, our lab has previously published a requirement for WNT-signaling for progression of BRAF\textsuperscript{V600E} lung tumors(Juan et al., 2014).
Specifically, inhibition of WNT signaling by a Porcupine inhibitor prevent tumor onset, while constitutive activation of the pathway drive tumor progression to high grade.
Therefore, it is not surprising that APC loss cooperates with BRAF\textsuperscript{V600E} to induce tumor growth.

We have also identified \emph{SETD2} loss as a potent tumor supressor(Figure 2).
\emph{SETD2} is mutated in other solid malignancies on cBioPortal, such as Kidney and bladder cancers(Cerami et al., 2012).
SETD2 is a histone methyltransferase that deposits trimethylation marks on H3K36(Bhattacharya and Workman, 2020).
H3K36me\textsuperscript{3} is used by chromatin modifiers to mark actively trascribed genes(Edmunds et al., 2007).
mining of TCGA mRNA expression and survival analysis further revealed that \emph{SETD2} mRNA is lower in lung adenocarcinoma and is associated with poorer overall survival(Figure 2).
Interestingly, \emph{SETD2} alterations are also significantly associated with BRAF\textsuperscript{V600E} mutations (Figure 2).
The unique association of \emph{BRAF\textsuperscript{V600E}} mutation and \emph{SETD2} loss may reflect an interesting synergistic effect between MAPK signaling and loss of chromatin regulation.
One hypothesis is that SETD2 loss drives spurious transcriptional activation.\\
Similarily, Recent work by Monte Winslow and also David Feldser's labs have indentified SETD2 loss as a cooperating event in KRAS-G12D lung tumorigenesis(Walter et al., 2017).
However, the mechanism by which \emph{SETD2} loss drives tumor progression in either BRAF\textsuperscript{V600E} or KRAS-G12D remain elusive.

Although we have identified novel tumor suppressors in BRAF\textsuperscript{V600E} lung tumorigenesis using Tuba-Seq, we have failed to confirm previously identified tumor suppressors \emph{CdkN2a} and \emph{Trp53} as cooperating events.
One hypothesis is that the sgRNAs targeting \emph{Trp53} and \emph{Cdkn2a} are inefficient. However, previous Tuba-Seq experiments did confirm \emph{Trp53} editing and even demonstrated tumorigeneic cooperation with KRAS-G12D(Rogers et al., 2018).
Another hypothesis is that the effect size of \emph{trp53} loss is so small that many mice and tumors are needed to detect a robust cooperation.
Indeed, p53 loss is not that to suddenly make tumors immediately proliferate.
Rather, p53 loss merely allows for tumor cells to sample various transcriptional programs that can be co-opted for increased fitness in the face of stress.
Along similar lines, perhaps 16-weeks is too little time to allow for \emph{Trp53} or \emph{Cdkn2a} deficient tumors to proliferate.
Likely, it is a combination of small effect size, ineficient gene editing, and insufficient time that prevented us from identifying \emph{Trp53} as a tumor supressor in BRAF6V600E\^{} driven lung tumorigenesis using Tuba-Seq combined with CRISPR/CAS9.

Since GEMMs lack the mutational burden exhibited by human lung adenocarcinomas, they are good models to study gene-gene interactions in tumorigenesis.
Our data provides further insight into what biological mechanisms are important for BRAF\textsuperscript{V600E} lung tumorigenesis.
Additional studies of SETD2's role in lung adenocarcinoma will further reveal how it alters the evolution of lung adenocarcinomas.
Furthermore, given its striking association with \emph{BRAF\textsuperscript{V600E}} mutations, it will be important to know how \emph{SETD2} alters response to BRAF\textsuperscript{V600E}-targeted therapy.
Full understanding of the consequences and outcomes of genetic cooperation between BRAF\textsuperscript{V600E} and tumor supressors or oncogenes will help clinicians better understand tumor evolution and paths to drug resistance.
Utltimetly, aiding to the refinement of cancer biology knowledge and precision oncology.

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/tuba1.png}
\caption{\textbf{Figure 2.1. Overview of Tuba-Seq experimentation} (A) Lenti-sgRNA-Cre plasmid library with collection of targeted tumor supressors. (B) \emph{Braf\textsuperscript{CAT/+}} and \emph{Braf\textsuperscript{CAT/+};H11b\textsuperscript{LSL-Cas9/+}} were infected with either 10\textsuperscript{4} (16 week \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}}) or 10\textsuperscript{5} (8 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}} and 16 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}}) (B) Representative gross images of experimental mouse lungs.}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:05}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/tuba2.png}
\caption{\textbf{Figure 2.2. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline.} Tumor size corresponds to dot size. Mean tumor number for 16 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{+/+}} is 21,248 cells. Mean tumor number for 8 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}} is 9,095 cells. Mean tumor number for 16 week \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}} is 29,867 cells.}\label{fig:05}
}
\end{figure}

\begin{figure}
\hypertarget{fig:06}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/tuba3.png}
\caption{\textbf{Figure 2.3. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline seperated by sgID.} (A) 16 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{+/+}}(negative control) has no apparent cooperative genetic events. (B) 8 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}} identifed Apc loss as a cooperating genetic event based on increase in tumor cell number (C) 16 week \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}} identified Apc and Setd2 loss as cooperating events.}\label{fig:06}
}
\end{figure}

\begin{figure}
\hypertarget{fig:07}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/tuba4.png}
\caption{\textbf{Figure 2.4. Validation of \emph{Setd2} loss as cooperating genetic event in BRAF\textsuperscript{V600E} lung cancer.} (A) Representative images of lung tumor histology of 16 weeks \emph{Braf\textsuperscript{CAT/+}; H11b\textsuperscript{LSL-Cas9/+}} mice infected with 2.5x10\textsuperscript{4} PFU Lenti-sgSetd2-Cre or Lenti-sgNT1-Cre. (B) Quantification of tumor size based on histology, n= 30 tumors. * = \_p-value\textless.05.}\label{fig:07}
}
\end{figure}

\begin{figure}
\hypertarget{fig:08}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/tuba5.png}
\caption{\textbf{Figure 2.5. H3K36me\textsuperscript{3} is lost in larger tumors in \emph{Set2d}-edited lung tumors.}(A) Representative images of H3K36me3 (surrogate for \emph{setd2} loss) depicting either ``high'' or ``low'' staining indicative of protein expression. (B) Quantification of H3K36me\textsuperscript{3} positive tumors in either \emph{sgSetd2} or \emph{sgNT1} targeted mice. \emph{= \emph{p-value\textless.05} for negative tumors. (C) Quantification of tumor size paired with H3K36me\textsuperscript{3} negative tumors. Tumor size and H3K36me\textsuperscript{3} positivty were simuletaneously measured using QuantCenter and binned in either ``low'' or ``High'' and then average tumor size was determined for each group. }= \emph{p-value\textless.05}.}\label{fig:08}
}
\end{figure}

\begin{figure}
\hypertarget{fig:09}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/tuba6.png}
\caption{\textbf{Figure 2.6. \emph{SETD2} loss is common in human cancers, an indicator of poor prognosis and frequently occurs with \emph{BRAF\textsuperscript{V600E}} mutations.}(A) Survival analysis with ``Survival'' R package indicates a trend towards poorer overall survival with low \emph{SETD2} mRNA in TGCA Lung adenocarcinoma dataset. (B) \emph{SETD2} mRNA is attenuated in Lung adenocarcinoma compared to normal lung tissue (C) \emph{SETD2} mutations significantly co-occur with BRAF\textsuperscript{V600E} mutations and are present in 40\% of \_BRAF\textsuperscript{V600E}) mutant Lung Adenocarcinoma.}\label{fig:09}
}
\end{figure}

\hypertarget{chapter-three}{%
\subsection{Chapter Three}\label{chapter-three}}

\hypertarget{abstract-1}{%
\subsection{Abstract}\label{abstract-1}}

\textsuperscript{TP53} missense BRAF\textsuperscript{V600E} mutations both frequently occur in lung adenocarcinoma. While genetically engineered mouse models (GEMMs) of lung adenocarcinoma are invaluable for understanding the initiation, progression, and response to therapeutics, little is known about how different hotspot point mutations in p53 affect disease progression and therapeutic responses. To this end, we use GEMMs, tumor-derived cell lines, and human lung cancer cell lines to test how murine homologues of p53 hotspot mutants R172H and R245W compare to p53-deletion. We find that conformation mutation p53\textsuperscript{R172H} drives more aggressive lung adenocarcinomas and dedifferentiation. We also observe p53\textsuperscript{R172H} and p53\textsuperscript{R245W} accumulation in only high-grade tumors that have activated ARF expression. In vitro and in vivo studies demonstrate enhanced resistance to BRAF+MEK blockade with p53\textsuperscript{R172H} and p53\textsuperscript{R245W}, a common treatment regimen for BRAF\textsuperscript{V600E} mutant patients. Mechanistically, mutant-p53 alters the transcriptome of cancer cells and enhances entry into the cell cycle.

\hypertarget{introduction-1}{%
\subsection{Introduction}\label{introduction-1}}

Lung adenocarcinoma (LUAD) remains deadliest form of cancer since 1987 (Siegel, Miller et al.~2021). It is characterized by alterations in the MAP kinase (MAPK) pathway, including activating mutations in the oncoproteins EGFR, KRAS (G12C, G12D, G12V) or BRAF\textsuperscript{V600E} (2014; Dankort et al., 2007; Jackson et al., 2001; Politi et al., 2006). The recognized predominant cell of origin for LUAD, lung alveolar type-2 pneumocyte cells (AT2), are exquisitely responsive to the transformative properties of constitutive activation of the MAPK pathway (Sainz de Aja and Kim, 2020; Sutherland et al., 2014). Indeed, conditionally genetically engineered mouse models (GEMMs) of mutationally activated EGFR, KRAS, or BRAF form lung tumors that strongly resemble key characteristics of human LUAD. However, tumors formed in all three models (KRAS:G12D, EGFR: Deletion 19 or L858R, or BRAF:V600E) develop only into histologically low- to mid-grade cancer and are growth arrested (Dankort et al., 2007; Jackson et al., 2001; Politi et al., 2006). Prior research has determined that oncogene-induced tumor growth arrest occurs through activation of several tumor suppression mechanisms, most notably, p53-mediated tumor suppression (Shai et al., 2015).

Alterations in the \emph{TP53} tumor suppressor gene are common in all cancer types, including LUAD (Bouaoun et al., 2016). P53 encodes a transcription factor that responds to a variety of cell stresses, such as DNA damage and oncogene activation. In response to stress, p53 is stabilized and induces anti-proliferative and genome-mending effects through trans-activation of a diverse set of P53-target genes (Boutelle and Attardi, 2021; Vogelstein et al., 2000). p53 mutations observed in cancer frequently occur in the DNA binding domain where a single amino acid change results in dysfunctional DNA-binding and failed transcriptional activation (Olivier et al., 2010). Therefore, mutated p53 that is defunct in p53 transcriptional regulation results in failed tumor suppression and cancer outgrowth. Furthermore, defective p53 signaling will permit future genomic insults to remain unchecked thus fostering an environment primed for malignant potential. Since mutant-p53 lacks transcriptional activity through its inability to bind DNA, it is unable to induce expression of its negative regulators, ARF, MDM2 and MDMX (Perry et al., 1993; Wu et al., 1993). Lack of negative feedback results in mutant-p53 stabilization. Accumulation of mutant p53 has been reported to elicit gain-of-function p53 activity. Thus, any gain-of-function activity in mutant-p53 cannot be studied with p53-genetic deletion. Many research groups have characterized gain-of-function phenotypes of mutant-p53 in cancer. For instance, mice expressing mutant-p53 have altered disease phenotype compared to p53-null models (Lang et al., 2004; Olive et al., 2004). Specifically in lung cancer model systems, mutant-p53 has been observed to modulate SMAD3, NF\(\kappa\)B, p63, TGF\(\beta\), and others to drive various hallmarks of cancer such as invasion, chemoresistance, metabolism, and propagation of a stem cell fate (Klemke et al., 2021 ; Adorno et al., 2009; Gaiddon et al., 2001). Although the mechanisms that leads to these altered processes are controversial, it is thought that the main affect is through mutant-p53 binding and sequestering known p53-associated proteins. This often leads to activation of transcription factors such as NF-Y (Kim et al., 2021).

In BRAF\textsuperscript{V600E} GEMMs, when p53 function is abolished by \emph{Trp53} gene deletion or by \emph{Cdkn2a} deletion, lung tumors can escape growth arrest and undergo malignant transformation (Dankort et al., 2007; Shai et al., 2015). Furthermore, p53 pathway dysregulation allows a cell to sample diverse phenotypic landscapes that allow for enhanced cell fitness and tumor outgrowth (Marjanovic et al., 2020). However, BRAF\textsuperscript{V600E} lung tumorigenesis and progression to malignant lung adenocarcinoma has only been studied in the context of p53 genetic deletion, not p53 mutation, which are more frequently seen in human lung cancer. Specifically, p53-R172H and p53-R245W are murine homologues of common hotspots in human cancer R175H and R245W, respectively.

Here, we characterize the differential impact of p53(null), as well as p53 hotspot mutations p53\textsuperscript{R712H} (R175H in humans) and p53\textsuperscript{R245W} (R248W in humans) on BRAF\textsuperscript{V600E} lung tumorigenesis. We observed significantly differential effects of p53-R172H on BRAF\textsuperscript{V600E}-driven lung tumorigenesis that was not observed when p53 is deleted, such as enhanced tumor size and decreased survival. Our findings highlight functional differences in mutant-p53 biology, as well as the importance and the utility of clinically relevant mutant-p53 mouse models.

\hypertarget{results-1}{%
\subsection{Results}\label{results-1}}

\textbf{P53-mutant Alleles Differentially Alter BRAF (V600E) Lung Tumorigenesis}

Expression of BRAF\textsuperscript{V600E} in the distal epithelium of the mouse lung elicits clonal tumorigenic outgrowths of alveolar type 2 (AT2) pneumocytes (Dankort et al.~2007).Prior work has shown that \emph{Trp53\textsuperscript{fl/fl}} cooperates with BRAF(V600E) to form malignant lung tumors when Cre recombinase is activated (Dankort et al., 2007; Shai et al., 2015). Therefore, we compared the tumor burden of \emph{Braf\textsuperscript{Cat/+}} mice with either p53 loss (\emph{Trp53\textsuperscript{fl/fl}}, p53\textsuperscript{R172H} (\emph{Trp53\textsuperscript{R172H/R172H}}), or p53\textsuperscript{R245W} (\emph{Trp53\textsuperscript{R245W/R245W}}) after infection with 106 pfu of adenovirus expressing surfactant protein C specific Cre (ad5-Spc-Cre) (Figure \ref{fig:11} A). As expected, mice developed lung cancer, consistent with previous findings (Figure \ref{fig:11} B-C). However, there was a striking, highly reproducible increase in the tumor size of tumors harboring \emph{Trp53\textsuperscript{R172H/R172H}} and to a lesser extent \emph{Trp53\textsuperscript{R245W/R245W}}, compared to \emph{Trp53\textsuperscript{fl/fl}} (Figure \ref{fig:11} B-D). These data led us to hypothesize that p53\textsuperscript{R172H} and p53\textsuperscript{R245W} expression alters tumor progression uniquely from p53 deletion, and these differences may be potentially evident of gain-of-function effects imparted by mutant-53. Since lung cancer patients with mutations in \emph{TP53} often start only have one mutated \emph{p53} allele unless loss of heterozygosity (LOH) occurs. Therefore, we generated mice heterozygous for both \emph{Braf\textsuperscript{CAT}} and either \emph{Trp53\textsuperscript{fl}} , \emph{Trp53\textsuperscript{R172H}}, or \emph{Trp53\^{} R245W\^{}} and observed their survival over time (Figure \ref{fig:11} E). We noted that \emph{Trp53\textsuperscript{R172H/+}} mice had diminished median overall survival compared to \emph{Trp53\textsuperscript{fl/+}} mice (183 days and 216 days), with \emph{Trp53\textsuperscript{R245W/+}} mice falling in between the two groups (median survival 212 days) (Figure \ref{fig:11} E).

We observed a distinct separation of survival between mice harboring lung tumors that are either \emph{Trp53\textsuperscript{fl/+}}, \emph{Trp53\textsuperscript{R172H/+}}, and \emph{Trp53\textsuperscript{R245W/+}}, this separation is diminished when altered p53 alleles are homozygous (Figure \ref{fig:11} F). \emph{Trp53\textsuperscript{R172H/R172H}} mice have a median survival of 120 days while \emph{Trp53\textsuperscript{fl/fl}} mice have a median survival of 136 days. Importantly, this observed difference in survival is statistically significant (Log-rank test p\textless{} 0.001) and is likely a reflection of worse diseased, characterized by enhanced tumor burden and tumor size.

\textbf{Mutant-p53 Accumulation is Associated with Dedifferentiation and p19 Induction}
Increased malignancy in lung adenocarcinoma is often associated with altered states of tumor cell identity and dedifferentiation (Marjanovic et al., 2020; van Veen et al., 2019). Since LUADs often arise from AT2 cells, we surveyed the expression of AT2 cell identifiers, such as NKX2-1, a master regulator of AT2 transcriptional programs, and SPC, a marker of differentiated AT2 cells (Marjanovic et al., 2020). Quantification of over 50 tumors from \emph{Braf\textsuperscript{V600E/+}} mice with either \emph{Trp53\^{} flfl\^{}}, \emph{Trp53\textsuperscript{R172H/R172H}}, and \emph{Trp53\textsuperscript{R245W/R245W}} revealed that \emph{Trp53\textsuperscript{R172H/R1722H}} mice had significantly more tumors that stained negative for NKX2-1 and SPC (Figure \ref{fig:12} A an B). This finding strongly indicates that mutant-p53 tumors are dedifferentiated. Since HMGA2 expression is associated with LUAD dedifferentiation and worse patient prognosis, we performed IHC for HMGA2 (Cite). We noted an increase in HMGA2-positive tumors in both \emph{Trp53\textsuperscript{R172H/R172H}}, and \emph{Trp53\textsuperscript{R245W/R245W}} tumors compared to \emph{Trp53\textsuperscript{fl/fl}} (Figure \ref{fig:12} A,B). Thus, this data taken together reveals that Trp53R172H/R172H mice exhibit both a higher tumor burden and worse overall survival than \emph{Trp53\textsuperscript{fl/fl}}, but the tumors developed in these mice are more likely to exhibit malignant progression and dedifferentiation.

One striking feature of lung tumors in mice either heterozygous or homozygous for \emph{Trp53 \textsuperscript{R172H}} or \emph{Trp53\textsuperscript{R245W}} is how few tumors and tumor cells have detectable levels of mutant-p53 protein as measured by Immunohistochemistry (IHC). One critical aspect of normal p53 function is self-regulation through transcriptional activation of its negative regulator MDM2 (Wu et al., 1993). Thus, we predicted that homozygous p53 mutant-cells would exhibit increased mutant-p53 protein. Interestingly, in our models, we observed mutant p53 protein accumulation is restricted to malignant cells exhibited by lack of papillary architecture and nuclear atypia (Figure \ref{fig:12} C). Indeed, p53 immunostaining is correlated with increased phospho-ERK1/2 detection (Figure \ref{fig:12} D). Moreover, immunofluorescence of p53 and its positive regulator, p19/ARF, reveal cells with high p19/ARF are also high in p53 accumulation (Figure \ref{fig:12} D). This result suggests that even in homozygous mutant\_Trp53\_ mice, there is still a barrier to mutant-p53 accumulation, and our data suggests that elevated levels of p19/ARF is important for mutant-p53 accumulation, even with homozygous p53 mutations. Collectively, these data indicate that \emph{Trp53\textsuperscript{R172H}} drives more aggressive lung adenocarcinoma formation in BRAF(V600E) driven mouse models of lung cancer.

\textbf{P53\textsuperscript{R172H} and P53\textsuperscript{R245W} cell lines are refractory to BRAF/MEK inhibition}
Given that previous research has determined that p53 mutations can confer resistance to both chemotherapies and pathway-targeted therapies (Huang et al., 2011; Rusch et al., 1995), we next questioned whether our p53 alterations would be sufficient to drive therapeutic resistance in BRAF\textsuperscript{V600E}-driven cell lines exposed to pathway-targeted therapeutics commonly administered to BRAF\textsuperscript{V600E} mutant patients, such as the combination of dabrafenib (BRAFi) plus trametinib (MEKi) (Planchard et al., 2016). Cell lines were derived from \emph{Braf\textsuperscript{CAT/+}} murine tumors expressing either \emph{Trp53\textsuperscript{flfl}}, \emph{Trp53\textsuperscript{R172H/R172H}}, or \emph{Trp53\textsuperscript{R245W/R245W}}) 14 weeks after infection with ad5-Spc-Cre. Cell lines expressing either \emph{Trp53\textsuperscript{R172H/R172H}} or \emph{Trp53\textsuperscript{R245W/R245W}} were completely resistant to single agent dabrafenib and resistant to the combination of dabrafenib plus trametinib (Figure \ref{fig:13} A,B). Of note, \emph{Trp53\textsuperscript{R172H/R172H}} and \emph{Trp53\textsuperscript{R245W/R245W}} cell lines express increased ERK1/2 activity, as evidenced by increased phosphorylated ERK1/2, both with and without Dabrafenib and Trametinib (Figure \ref{fig:13} E). Moreover, \emph{Trp53\textsuperscript{R172H/R172H}} mice retain a larger tumor burden after a week 4 treatment of MAPK blockade \emph{in vivo} (Figure \ref{fig:13} E). To gain relevance to human lung cancer, we infected BRAF(V600E) mutant HCC364 cells with lentivirus expressing human cancer hotspot mutant p53\textsuperscript{R175H}. p53\textsuperscript{R175H} expressing cells were more proliferative and more refractory than parental HCC364 cells (Figure \ref{fig:13} F and G).Collectively, these \emph{in vitro} and \emph{in vivo} data suggest missense mutant p53\textsuperscript{R175H} and its murine homologue p53\textsuperscript{R172H}, as well as p53\textsuperscript{R245W} can support cancer cell viability in the face of BRAF+MEK blockade that cannot be achieved by p53-null mutations.

\textbf{P53\textsuperscript{R172H} and P53\textsuperscript{R245W} alters the transcriptome of BRAF(V600E) lung tumor cell lines}
Due to our observations that mutant p53\textsuperscript{R172H} drives more aggressive cancer phenotypes such as increased growth, dedifferentiation, and resistance to FDA-approved therapeutics, we reasoned that this would be reflected in a change in the transcriptional landscape. This is due in part, by the transcriptional trans-activation potential by mutant p53. RNA sequencing of cell lines generated from p53-mutant mouse lung tumors do reveal transcriptomic differences between both p53\textsuperscript{R172H} and p53\textsuperscript{R245W} cells compared to p53-null cell lines (Figure \ref{fig:14} 4 A). Interestingly, genes involved in cell cycle regulation, epithelial-to-mesenchymal transition (EMT), and MYC pathway are upregulated in both p53\textsuperscript{R172H} and p53\textsuperscript{R2245W} cell lines (Figure 4 \ref{fig:14} B). These results suggest that mutant-p53 is affecting transcription of genes involved in pro-growth signaling pathways.

\textbf{addiction to mutant-p53 expression}
\textbf{PDGFRA RNA data}
\textbf{Response to PDGFR stimuliation and inhibition}

\begin{figure}
\hypertarget{fig:11}{%
\centering
\includegraphics[width=0.1\textwidth,height=\textheight]{images/p53_1.png}
\caption{\textbf{Differential BRAF\textsuperscript{V600E} lung tumorigenesis imparted by p53-Hotspot mutants.} \textbf{(A)} Genetically-engineered alleles of \emph{Braf\textsuperscript{v600E}} and \textsuperscript{Trp53} before and after Cre-Recombinase. \textbf{(B)} Tumor burden and \emph{(C)}** Tumor size of BRAF(V600E) plus Trp53-wildtype, null, or mutant alleles 14 weeks post Cre-mediated recombination. \emph{(D)} Representative H\&E images 14 weeks after Cre-recombination. \textbf{(E)} Survival of mice with Heterozygous \emph{Trp53} mutant alleles. \textbf{(F)} Survival of mice with homozygous \emph{Trp53} alleles.}\label{fig:11}
}
\end{figure}

\begin{figure}
\hypertarget{fig:12}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p53_2.png}
\caption{\textbf{Missense mutant-p53 alters lung adenocarcinoma differentiation status and accumulation of mutant-p53 correlates with p19 ARF expression.} \textbf{(A)} Panel representative IHC sections stained with SPC,NKX2-1, p53 and HMGA2. \textbf{(B)} H-score of individual tumors. \textbf{(C)} Association between mutant-p53 R172H accumulation and increased malignancy and dedifferentiation. \textbf{(D)} serial section of P53-HI and pERK1/2-HI tumors compared to adjacent low. \textbf{(E)} Colocalization of p19/ARF and p53 in p53-mutant. tumors}\label{fig:12}
}
\end{figure}

\begin{figure}
\hypertarget{fig:13}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p53_3.png}
\caption{\textbf{Mutant-p53 cell lines are refractory to BRAFi+MEKi.} \textbf{(A)} Response of GEMMs to BRAFi+MEKi chow after four weeks \textbf{(B)} in vitro dose response to BRAFi single agent. \textbf{(C)} in vitro timecourse with dabrafenib (1000nm) plus trametinib (100nM). \textbf{(D)} Accumulation of dead cells in dabrafenib plus trametinib measured by CytotoxRed accumulation. \textbf{(E)} Immunoblotting of phospho-ERK1/2 in response to dabrafenib plus trametinib. \textbf{(F)} Ectopic expression of p53\textsuperscript{R17H} in human HCC364 cells. \textbf{(E)} Response of p53\textsuperscript{R175H} cells to dabrafenib plus trametinib.}\label{fig:13}
}
\end{figure}

\begin{figure}
\hypertarget{fig:14}{%
\centering
\includegraphics{images/p53_4.png}
\caption{\textbf{Mutant-p53 expression cells have distinct transcriptome compared to p53-null cells.} \textbf{(A)} Volcano plot of differentially-expressed genes. \textbf{(B)} Upregulated pathways using GSEA with Hallmark gene sets.}\label{fig:14}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p53_5.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p536.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p537.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p538.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p539.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p5310.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/p5311.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/pdgfr1.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/pdgfr2.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\hypertarget{discussion-1}{%
\subsection{Discussion}\label{discussion-1}}

While lung adenocarcinomas primarily contain missense mutations in p53, most studies using mouse models of lung adenocarcinoma utilize a p53-null (\emph{Trp53\textsuperscript{fl/fl}}) allele instead of alleles encoding hotspot missense mutations (Gao et al., 2013). This is most likely because prior mutant-p53 alleles were null for \emph{Trp53} prior to Cre-mediated recombination, leaving every cell in the mouse sensitive to leukemias, lymphomas and sarcomas due to being partially p53-deficient (Jackson et al., 2005; Olive et al., 2004; Wijnhoven et al., 2005). Our work has been made possible by newly described ``wildtype to mutant'' p53 alleles that allow for mice to be fully wildtype for p53 in all cells prior to Cre-mediated recombination(Zhang et al., 2018). This strategy is crucial, as it allows for temporally extended experiments not confounded by separate malignancies. Here, we utilize "wildtype to mutant"\emph{Trp53\textsuperscript{R172H}} and \emph{Trp53\textsuperscript{R245W}} alleles R245W to determine how they alter tumor progression in a BRAF(V600E) model of lung cancer. We show that p53\textsuperscript{R172H} and p53\textsuperscript{R245W} accelerates lung tumor progression and increases the rate of dedifferentiation beyond complete loss of p53 expression.

Although point mutations in the \emph{TP53} gene can poison the natural tumor suppressive mechanisms of p53 protein, not all point mutations are equivalent in executing these functions. Mutations in the DNA-binding domain are most common and most potent at ablating normal p53 function. Taking advantage of a conditional BRAF\textsuperscript{V600E} lung tumor model where BRAF\textsuperscript{V600E} expression in AT2 cells drives benign tumor formation, we show that either p53\textsuperscript{R172H} or p53\textsuperscript{R245W} can bypass growth arrest and influence tumor progression and therapeutic sensitivity beyond how p53 loss can. Specifically, P53\textsuperscript{R172H} reduces survival and sensitivity to dabrafenib plus trametinib \emph{in vivo}. Interestingly, a single allele of p53\textsuperscript{R172H} has a stronger tumor promoting phenotype than P53\textsuperscript{R245W}. These findings further support a hierarchy model of p53 missense mutations, where conformation mutants such as p53\textsuperscript{R175H} have a stronger dominant-negative effect on wildtype p53, while mice with a single \emph{trp53\textsuperscript{fl}} allele can still suppress tumorigenesis with one copy of wildtype but are prone to Loss of heterozygosity (Muzumdar et al., 2016; Olive et al., 2004; Zhang et al., 2018).

In BRAF-driven lung adenocarcinoma, the repressive role of wild-type p53 is initiated by oncogenic stress brought about by increasing levels of MAPK-pathway activation(Dankort et al., 2007) . Hyperactivation of the MAPK pathway triggers p19/ARF to block MDM2, leading to p53 accumulation(Christophorou et al., 2006; Feldser et al., 2010; Junttila et al., 2010). Therefore, p53 acts late in lung adenocarcinoma, in contrast to pancreatic adenocarcinoma, where p53 is triggered early likely through PDAC-specific increases in p19/ARF through either increased PI3K, STAT3, or DMP1 (Muzumdar et al., 2016). Our studies in BRAF(V600E) mutant lung adenocarcinomas reinforce this concept, as mutant-p53 does not accelerate growth in early tumors (supplemental figure). Furthermore, p53 is not detected until later stage tumor development and correlates with p19/ARF expression and increased phospho-ERK1/2 levels. Our studies indicate that mutant-p53 levels are associated with increased expression of p19/ARF. Therefore, any gain-of-function phenotype manifested by mutant-p53 must depend on accumulated p19/ARF. Thus, ablating the ARF tumor suppressor in human cancers with GOF \emph{TP53} mutants may paradoxically lead to tumor regression.

While most \emph{TP53} mutations observed in cancer result in an inability to bind DNA, they are often stable and preserve functional oligomerization and transcriptional transactivation domains(Garcia and Attardi, 2014). Thus, mutant-p53 can still interact with and affect the function of cellular processes. Supporting this rationale we observe altered transcriptomes with both p53\textsuperscript{R172H} and p53\^{}R245W cell lines compared to p53-null. Consistent with previous mutant-p53 GOF studies, we observed enriched genes associated with EMT and cell-cycle entry with p53\textsuperscript{R172H} and p53\textsuperscript{R245W}.

Many GOF phenotypes exhibited by mutant-p53 have centered around drug resistant mechanisms. Previous studies have shown that mutant-p53 can confer resistance to pathway targeted therapy and chemotherapies. Our studies highlight that p53\textsuperscript{R172H} and p53\textsuperscript{R245W} can confer resistance to BRAF-targeted therapy. Here, we show p53-R172H and p53-R245W cell lines are resistant to BRAF(V600E) inhibition and less sensitivity to the combination of BRAF+MEK inhibition. In pancreatic cancer models with LSL-p53-R172H expression, mutant-p53 regulates PDGFR-signaling to promote an invasive phenotype that is sensitive to Imatinib(Weissmueller et al., 2014). Consistent with these findings, our transcriptional analysis and immunoblotting indicate p53-R172H and p53-R245W cells express PDGF-ligands and exhibit PDGFR-pathway activation.

In summary, we have compared the tumorigenic potential of three \emph{\textsuperscript{Trp53}} alleles (\emph{Trp53\textsuperscript{fl/fl}}, \emph{Trp53\textsuperscript{R172H}}, and \emph{Trp53\textsuperscript{R245W}}) concurrently with \emph{BRAF\textsuperscript{V600E}}. \emph{Trp53\textsuperscript{R172H}} and \emph{Trp53\textsuperscript{R245W}} promoted more aggressive lung cancers exhibited by larger tumors and tumor dedifferentiation. However, only \emph{Trp53\textsuperscript{R172H}} was observed to shorten survival compared to \emph{Trp53\textsuperscript{fl/fl}}. Moreover, \emph{Trp53\textsuperscript{R172H}} and \emph{Trp53\textsuperscript{R245W}} cell lines were refractory to dabrafenib and trametinib and were more proliferative. \emph{Trp53\textsuperscript{R172H}} and \emph{Trp53\textsuperscript{R245W}} cell lines had altered transcriptomes compared to p53-null cells and were highly enriched in genes involved in cell cycle proliferation. Our work highlights the importance of studying disease-relevant \emph{TP53} alleles in pre-clinical modeling systems.

\hypertarget{chapter-four}{%
\subsection{Chapter Four}\label{chapter-four}}

\#\#Abstract

\hypertarget{introduction-2}{%
\subsection{Introduction}\label{introduction-2}}

The mammalian lung is a vast and intricate labyrinth that is responsible that is responsible for gas exchange between air and body through a meshwork of endothelial cells. The lung organ system begins with a single airway that branches in a geometric and stereotypic pattern(Metzger et al., 2008). These many thousands of branches lead to the business end of gas exchange, the lung alveoli. The alveoli are comprised of two specialized cell types: Alveolar Type 1 (AT1) cells comprise most of the surface area of the lung and are thin squamous cells that mediate oxygen exchange across cell membranes. Alveolar Type 2 (AT2) cells, however, are cuboidal that secrete surfactant proteins that aide in preventing the delicate alveoli from collapsing.

Pathology of the alveoli is a steadfast threat to an organism's life due to the importance of gas exchange. Diseases such as lung adenocarcinoma, pulmonary fibrosis, and emphysema are the most common dysfunctions of the distal lung epithelium(Desai et al., 2014). Lung adenocarcinoma is also the most common type of human lung cancers in both men and women. A large body of literature has suggested that lung adenocarcinomas arise in the alveolar and are descendants of AT2 cells (Ferone et al., 2020; Jackson et al., 2001). Therefore, there is great interest in understanding how AT2 cells develop, maintain cell identity, as well as how they renew themselves during aging or in response to injury (2020; Alanis et al., 2014, 2014; Treutlein et al., 2014).

Recent advances in profiling of single-cell transcriptomes during homeostasis conditions and diseases states have greatly improved our understanding of AT2 cell, and lung adenocarcinoma cell biology(LaFave et al., 2020; Marjanovic et al., 2020; Treutlein et al., 2014).Single-cell profiling of human lung cancers have revealed the drastic intra-tumoral heterogeneity that exists in a single tumor(Gerlinger et al., 2012; Xue et al., 2016). Since transcriptional heterogeneity can be considered a source of therapeutic resistance, there is great interest in understanding the evolution of lung adenocarcinoma and how it changes with response to pathway targeted therapy(Maynard et al., 2020).Since GEMMs are an important pre-clinical model for testing out potential therapeutic strategies for lung adenocarcinoma, it is important to understand single-cell heterogeneity in popular GEMM models.

Although GEMMs are a great way to model human lung adenocarcinomas, there are several drawbacks worth mentioning. Human lung adenocarcinomas typically develop from a single tumor-initiating cell and is a consequence of an accumulation of mutations in certain cell types and a weakened immune system across a lifetime. In GEMMs, tumorigenesis is often initiated in many cells at once and is the consequence of an engineered mutation. In contrast to human lung adenocarcinomas that have a high mutational burden, murine lung adenocarcinomas do not have a high mutational burden and instead display high copy number variation(Westcott et al., 2014). Therefore, we set out to profile the single-cell transcriptomes of various BRAF\textsuperscript{V600E}-driven lung cancer GEMMs in order to determine genotype-specific heterogeneity and response to pathway-targeted therapy.

\hypertarget{results-2}{%
\subsection{Results}\label{results-2}}

\textbf{table of genotypes/treatment/and cells collected}
Pre-sort
pre-batch correction
batch correction
cell types
tumor cells only
Feature plots of AT2 markers
feature plots of AT1 markers
Feature plots of P53 target genes
heatmap of P53 target genes
heatmap/violin plot of MAPK score

\hypertarget{discussion-2}{%
\subsection{Discussion}\label{discussion-2}}

talk about the batch effects
talk about the times
similarity to other studies with KRAS
talk about homogenous response to D+T

\begin{figure}
\hypertarget{fig:04}{%
\centering
\includegraphics[width=1\textwidth,height=\textheight]{images/scrna_1.png}
\caption{\textbf{This is the title of Figure} this is the text.**}\label{fig:04}
}
\end{figure}

\hypertarget{chapter-six-methods}{%
\subsection{Chapter Six: methods}\label{chapter-six-methods}}

\textbf{Animal Husbandry} All mice were housed in an environmentally controlled room, and all animal care and experimental procedures were approved by (and in accordance with) the Institutional Animal Care and Use Committee Office of the Huntsman Cancer Institute at the University of Utah. Genetically engineered mouse breeding and genotyping was conducted as previously described (van Veen et al., 2019). The BRAF\textsuperscript{CAT} and \emph{Trp53\textsuperscript{fl/fl}} mice were previously described (Dankort et al., 2007; Jonkers et al., 2001; Shai et al., 2015; van Veen et al., 2019). All viruses were administered in a Biosafety Level 2+ room, as is regulated by the Institutional Biosafety Committee Guidelines. Adeno-SPC-CRE virus (University of Iowa) was delivered through nasal instillation. The \emph{Trp53\textsuperscript{wmR172H}} and \emph{Trp53\textsuperscript{wmR245W}} mice and respective genotyping protocols were gifted by Gigi Lozano and her laboratory and previously published (Kim et al., 2021; Zhang et al., 2018).

\textbf{Tuba-Seq} \emph{Braf\textsuperscript{CAT/+}} and \emph{H11b\textsuperscript{LSL-CAS9/+}} mice have been described(Chiou et al., 2015; van Veen et al., 2019) and maintained on a mixed background. approximetley equal numbers ofmale and female mice were used in the study. lung tumors were initiated by intratracheal intubation of lentiviral Cre vectors as previously described(DuPage et al., 2009). Barcoded lentiviral vectors were supplied by Monte Winslow as previously described(Winters et al., 2017). Briefly, Lentiviral vectors that contain an eight nucleotide sequence that defines the CRISPR-targeted followed by a 15 nucleotide sequence that is unique only to each initiated tumor. Lentivirus was generated in HEK 293T cells in DMEM supplemented with 10\% FBS by standard lentiiral generation and titered as previously described(Winters et al., 2017). Genomic DNA was isolated from bulk lung tissue after weighing. Briefly, benchmark control cell lines were spiked-in to enable calculation of the absolute number of tumor cells from the number of barcode reads. After tissue homogenization and 12-hour proteinase-K digestion, genomic DNA isolation was extracted using standard Phenol-Chloroform and precipitated by ethanol. Suquencing libraries were prepared by PCR amlifying the sgRNA-BC sequencing in eight reactions in parallel with Q5 master mix and purified with AMPure beads and quantified with the Qubit instrument. Computational analysis was performed in house by the Monte Winslow laboratory.

\textbf{Cell Line Generation}Tumor bearing mice were euthanized and lungs dissected. Lungs were then minced and incubated in digestive enzymes Collagenase I, dispase, elastase, for 30 minutes at 37 degrees. Single-cell suspensions were then resuspended in complete DMEM medium and filtered through a 100 micron filter and centrifuged at 300 x g for 5 minutes. Single-cell suspensions were then incubated in red blood cell lysis buffer for 5 minutes at room temperature, then resuspended in complete RPMI and placed in a tissue culture incubator. Cells were then passaged once a week with differential trypsinization to purify tumors cells from viable stromal cells. p53 status and BRAF\textsuperscript{V600E} status (tdTomato+) were assessed by western blot and flow cytometry.

\textbf{Tissue Culture}Murine tumor derived cell lines and HCC-3564 cells were routinely cultured in DMEM supplemented with 10\% heat-inactivated FBS and 1\% penicillin/streptomycin. Cells were routinely tested for mycoplasma.

\textbf{Immunoblotting}Cells destined for western blot were scraped in ice cold PBS then centrifuged to pellet cells. After supernatant aspiration, cells were lysed with RIPA buffer supplemented with a protease/phosphatase inhibitor cocktail (Halt). Cell lysate protein concentration was determined with BCA assay. 40 microgram/lane was mixed with 4X SDS buffer and 10X sample buffer. Samples were run through a 4-12\% Bis-tris gel at 200V for approximately 75 minutes then transferred to a PVDF membrane using the iBlot2 transfer apparatus. Membranes were then incubated in Odyssey blocking buffer for 30 minutes before being incubated overnight with primary antibody. Membranes were then washed 3X with TBS-T and incubated in secondary antibody for 2 hours before being washed 3X and imaged on the Licor Odyssey clX.

\textbf{RNA Sequencing}Cells lines were routinely cultured for approximately 10 passages before RNA purification using Qiagen Mini kit from one million cells. Libraries were generated with Illumina TruSeq Stranded mRNA Library Prep with UDI kit and sequenced on a Nova-Seq targeting 25 million reads per sample. fastq files were processed on Galazy.org using FastQC, HISAT2, MultiQC, FeatureCounts, and DESeq2.

\textbf{Immunohistochemistry and Immunofluoresence}Immunohistochemistry was performed as previously described, with the rabbit primary antibody against P53, SPC, phospho-ERK1/2, NKX2-1, HMGA2, p19/ARF (Mollaoglu et al., 2017; van Veen et al., 2019).

\textbf{Slide Imaging and Quantification}Hematoxylin and eosin (H\&E) and IHC stained slides 1 were scanned automatically by the 3D Histech Pannoramic MIDI scanner (Thermo Fisher). Slides were imaged and analyzed using Caseviewer Software or QuantCenter analytical center provided on the 3D Histech Slide Scanner at the indicated magnification. Tumor burden was manually calculated on each lung lobe and total tumor area was compared to total lung area. Tumor diameters were measured using QuantCenter software from 3D Histech.

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.1624}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.1709}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.2308}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.1368}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.2991}}@{}}
\caption{\textbf{Critical reagents} \label{tbl:reagents}}\tabularnewline
\toprule
\begin{minipage}[b]{\linewidth}\raggedright
\textbf{Reagents List}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Reagent Type
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Name
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Source
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Catalog number
\end{minipage} \\
\midrule
\endfirsthead
\toprule
\begin{minipage}[b]{\linewidth}\raggedright
\textbf{Reagents List}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Reagent Type
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Name
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Source
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Catalog number
\end{minipage} \\
\midrule
\endhead
antibody & beta-Actin & CST & 3700T & wb: 1:10,000 \\
antibody & Phospho ERK1/2 & CST & 4370 & wb:1:2000, IHC:1:1000 \\
antibody & ERK1/2 & CST & 4696 & wb: 1:2000 \\
antibody & P53 & Leica & cm-5 & wb: 1:2000, IHC: 1:1000, IF:1:250 \\
antibody & NKX2-1 & Abcam & ab76013 & IHC: 1:1000 \\
antibody & Pro-SPC & Millipore & Cat\# AB3786 & IHC: 1:2000 \\
antibody & HMGA2 & CST & 8179S & IHC 1:2000 \\
antibody & p19/ARF & Chuck Sherr & (Bertwistle et al., 2004) & \\
Cell Line & 293T & ATCC & CRL-3216 & \\
Cell Line & HCC-364 & Bivona lab & (Lin et al., 2014) & \\
Recombinant DNA & Pmd2.5 & Addgene & \#12259 & \\
Recombinant DNA & pspax2 & Addgene & \#12260 & \\
Recombinant DNA & P53-R175H & Addgene & \#22936 & \\
Chemical Compound & dabrafenib & Selleck & S2807 & \\
Chemical Compound & PLX4720 & Plexxikon/ Research Diets & & \\
Chemical Compound & trametinib & Selleck & S2673 & \\
Other & Collagenase type 1 & Thermo Fischer Scientific & 17100017 & \\
Other & Dispase & Corning & 354235 & \\
other & Elastase & Worthington Biochem & LS006365 & \\
other & ad5spc-Cre & university of Iowa & VVC-Berns1168 & \\
\bottomrule
\end{longtable}

\hypertarget{chapter-six-future}{%
\subsection{Chapter Six: Future}\label{chapter-six-future}}

\#\#Conclusions
The broad purpose of this work was to better understand how BRAF\textsuperscript{V600E} tumors form and progress to lung adenocarcinoma.
Our strategy was (1) to determine additional tumor supressor genes in BRAF\textsuperscript{V600E} lung tumorigenesis, (2) evaluate effect different mutant-p53 alleles, and (3) profile the tumor cell heterogenity using scRNA-seq.

\#\#Future directions
immuno profiling?

\hypertarget{references}{%
\subsection{References}\label{references}}

\hypertarget{refs}{}
\begin{CSLReferences}{0}{0}
\leavevmode\vadjust pre{\hypertarget{ref-8Gf0ScB2}{}}%
(2014). \href{https://doi.org/10.1038/nature13385}{Comprehensive molecular profiling of lung adenocarcinoma.} Nature \emph{511}, 543--550.

\leavevmode\vadjust pre{\hypertarget{ref-179MWicwA}{}}%
(2020). \href{https://doi.org/10.1038/s41586-020-2496-1}{A single-cell transcriptomic atlas characterizes ageing tissues in the mouse.} Nature \emph{583}, 590--595.

\leavevmode\vadjust pre{\hypertarget{ref-JyurLVYi}{}}%
Abegglen, L.M., Caulin, A.F., Chan, A., Lee, K., Robinson, R., Campbell, M.S., Kiso, W.K., Schmitt, D.L., Waddell, P.J., Bhaskara, S., et al. (2015). \href{https://doi.org/10.1001/jama.2015.13134}{Potential Mechanisms for Cancer Resistance in Elephants and Comparative Cellular Response to DNA Damage in Humans.} JAMA \emph{314}, 1850--1860.

\leavevmode\vadjust pre{\hypertarget{ref-1HKPvGqty}{}}%
Adorno, M., Cordenonsi, M., Montagner, M., Dupont, S., Wong, C., Hann, B., Solari, A., Bobisse, S., Rondina, M.B., Guzzardo, V., et al. (2009). \href{https://doi.org/10.1016/j.cell.2009.01.039}{A Mutant-p53/Smad complex opposes p63 to empower TGFbeta-induced metastasis.} Cell \emph{137}, 87--98.

\leavevmode\vadjust pre{\hypertarget{ref-vsbCfYx4}{}}%
Alanis, D.M., Chang, D.R., Akiyama, H., Krasnow, M.A., and Chen, J. (2014). \href{https://doi.org/10.1038/ncomms4923}{Two nested developmental waves demarcate a compartment boundary in the mouse lung.} Nat Commun \emph{5}, 3923.

\leavevmode\vadjust pre{\hypertarget{ref-e3rkxhDE}{}}%
Alexandrov, L.B., Ju, Y.S., Haase, K., Van Loo, P., Martincorena, I., Nik-Zainal, S., Totoki, Y., Fujimoto, A., Nakagawa, H., Shibata, T., et al. (2016). \href{https://doi.org/10.1126/science.aag0299}{Mutational signatures associated with tobacco smoking in human cancer.} Science \emph{354}, 618--622.

\leavevmode\vadjust pre{\hypertarget{ref-6j21xB75}{}}%
Arteaga, C.L., and Engelman, J.A. (2014). \href{https://doi.org/10.1016/j.ccr.2014.02.025}{ERBB receptors: from oncogene discovery to basic science to mechanism-based cancer therapeutics.} Cancer Cell \emph{25}, 282--303.

\leavevmode\vadjust pre{\hypertarget{ref-HnQw16l7}{}}%
Bartek, J., Iggo, R., Gannon, J., and Lane, D.P. (1990). \href{https://www.ncbi.nlm.nih.gov/pubmed/1694291}{Genetic and immunochemical analysis of mutant p53 in human breast cancer cell lines.} Oncogene \emph{5}, 893--899.

\leavevmode\vadjust pre{\hypertarget{ref-18dmdoj17}{}}%
Bertwistle, D., Zindy, F., Sherr, C.J., and Roussel, M.F. (2004). \href{https://doi.org/10.1089/hyb.2004.23.293}{Monoclonal antibodies to the mouse p19(Arf) tumor suppressor protein.} Hybrid Hybridomics \emph{23}, 293--300.

\leavevmode\vadjust pre{\hypertarget{ref-1AV7mz5Fc}{}}%
Bhattacharya, S., and Workman, J.L. (2020). \href{https://doi.org/10.1186/s13072-020-00362-8}{Regulation of SETD2 stability is important for the fidelity of H3K36me3 deposition.} Epigenetics Chromatin \emph{13}, 40.

\leavevmode\vadjust pre{\hypertarget{ref-ju5z9qDz}{}}%
Bouaoun, L., Sonkin, D., Ardin, M., Hollstein, M., Byrnes, G., Zavadil, J., and Olivier, M. (2016). \href{https://doi.org/10.1002/humu.23035}{TP53 Variations in Human Cancers: New Lessons from the IARC TP53 Database and Genomics Data.} Hum Mutat \emph{37}, 865--876.

\leavevmode\vadjust pre{\hypertarget{ref-GiCMgc6K}{}}%
Bougeard, G., Sesboüé, R., Baert-Desurmont, S., Vasseur, S., Martin, C., Tinat, J., Brugières, L., Chompret, A., de Paillerets, B.B., Stoppa-Lyonnet, D., et al. (2008). \href{https://doi.org/10.1136/jmg.2008.057570}{Molecular basis of the Li-Fraumeni syndrome: an update from the French LFS families.} J Med Genet \emph{45}, 535--538.

\leavevmode\vadjust pre{\hypertarget{ref-KlPsMirp}{}}%
Boutelle, A.M., and Attardi, L.D. (2021). \href{https://doi.org/10.1016/j.tcb.2020.12.011}{p53 and Tumor Suppression: It Takes a Network.} Trends Cell Biol \emph{31}, 298--310.

\leavevmode\vadjust pre{\hypertarget{ref-Ph0Z7rTW}{}}%
Cerami, E., Gao, J., Dogrusoz, U., Gross, B.E., Sumer, S.O., Aksoy, B.A., Jacobsen, A., Byrne, C.J., Heuer, M.L., Larsson, E., et al. (2012). \href{https://doi.org/10.1158/2159-8290.cd-12-0095}{The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data.} Cancer Discov \emph{2}, 401--404.

\leavevmode\vadjust pre{\hypertarget{ref-1EwtuJJw6}{}}%
Chiou, S.-H., Winters, I.P., Wang, J., Naranjo, S., Dudgeon, C., Tamburini, F.B., Brady, J.J., Yang, D., Grüner, B.M., Chuang, C.-H., et al. (2015). \href{https://doi.org/10.1101/gad.264861.115}{Pancreatic cancer modeling using retrograde viral vector delivery and in vivo CRISPR/Cas9-mediated somatic genome editing.} Genes Dev \emph{29}, 1576--1585.

\leavevmode\vadjust pre{\hypertarget{ref-G5wVcRrJ}{}}%
Christophorou, M.A., Ringshausen, I., Finch, A.J., Swigart, L.B., and Evan, G.I. (2006). \href{https://doi.org/10.1038/nature05077}{The pathological response to DNA damage does not contribute to p53-mediated tumour suppression.} Nature \emph{443}, 214--217.

\leavevmode\vadjust pre{\hypertarget{ref-OK8s0ke5}{}}%
Cicchini, M., Buza, E.L., Sagal, K.M., Gudiel, A.A., Durham, A.C., and Feldser, D.M. (2017). \href{https://doi.org/10.1016/j.celrep.2017.01.069}{Context-Dependent Effects of Amplified MAPK Signaling during Lung Adenocarcinoma Initiation and Progression.} Cell Rep \emph{18}, 1958--1969.

\leavevmode\vadjust pre{\hypertarget{ref-wMtu748b}{}}%
Corcoran, R.B., Ebi, H., Turke, A.B., Coffee, E.M., Nishino, M., Cogdill, A.P., Brown, R.D., Della Pelle, P., Dias-Santagata, D., Hung, K.E., et al. (2012). \href{https://doi.org/10.1158/2159-8290.cd-11-0341}{EGFR-mediated re-activation of MAPK signaling contributes to insensitivity of BRAF mutant colorectal cancers to RAF inhibition with vemurafenib.} Cancer Discov \emph{2}, 227--235.

\leavevmode\vadjust pre{\hypertarget{ref-dTvhNsG9}{}}%
Dankort, D., Filenova, E., Collado, M., Serrano, M., Jones, K., and McMahon, M. (2007). \href{https://doi.org/10.1101/gad.1516407}{A new mouse model to explore the initiation, progression, and therapy of BRAFV600E-induced lung tumors.} Genes Dev \emph{21}, 379--384.

\leavevmode\vadjust pre{\hypertarget{ref-1F86sLj5s}{}}%
de Groot, P., and Munden, R.F. (2012). \href{https://doi.org/10.1016/j.rcl.2012.06.006}{Lung cancer epidemiology, risk factors, and prevention.} Radiol Clin North Am \emph{50}, 863--876.

\leavevmode\vadjust pre{\hypertarget{ref-n08HchE9}{}}%
Desai, T.J., Brownfield, D.G., and Krasnow, M.A. (2014). \href{https://doi.org/10.1038/nature12930}{Alveolar progenitor and stem cells in lung development, renewal and cancer.} Nature \emph{507}, 190--194.

\leavevmode\vadjust pre{\hypertarget{ref-WyV78aHe}{}}%
Deuker, M.M., Marsh Durban, V., Phillips, W.A., and McMahon, M. (2014). \href{https://doi.org/10.1158/2159-8290.cd-14-0856}{PI3'-kinase inhibition forestalls the onset of MEK1/2 inhibitor resistance in BRAF-mutated melanoma.} Cancer Discov \emph{5}, 143--153.

\leavevmode\vadjust pre{\hypertarget{ref-kxXMFjnc}{}}%
Donehower, L.A., Harvey, M., Slagle, B.L., McArthur, M.J., Montgomery, C.A., Butel, J.S., and Bradley, A. (1992). \href{https://doi.org/10.1038/356215a0}{Mice deficient for p53 are developmentally normal but susceptible to spontaneous tumours.} Nature \emph{356}, 215--221.

\leavevmode\vadjust pre{\hypertarget{ref-bhNe3oCb}{}}%
DuPage, M., Dooley, A.L., and Jacks, T. (2009). \href{https://doi.org/10.1038/nprot.2009.95}{Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase.} Nat Protoc \emph{4}, 1064--1072.

\leavevmode\vadjust pre{\hypertarget{ref-M2kGnbOE}{}}%
Edmunds, J.W., Mahadevan, L.C., and Clayton, A.L. (2007). \href{https://doi.org/10.1038/sj.emboj.7601967}{Dynamic histone H3 methylation during gene induction: HYPB/Setd2 mediates all H3K36 trimethylation.} EMBO J \emph{27}, 406--420.

\leavevmode\vadjust pre{\hypertarget{ref-IILnKdOc}{}}%
el-Deiry, W.S., Tokino, T., Velculescu, V.E., Levy, D.B., Parsons, R., Trent, J.M., Lin, D., Mercer, W.E., Kinzler, K.W., and Vogelstein, B. (1993). \href{https://doi.org/10.1016/0092-8674(93)90500-p}{WAF1, a potential mediator of p53 tumor suppression.} Cell \emph{75}, 817--825.

\leavevmode\vadjust pre{\hypertarget{ref-CMVwSV1C}{}}%
Feldser, D.M., Kostova, K.K., Winslow, M.M., Taylor, S.E., Cashman, C., Whittaker, C.A., Sanchez-Rivera, F.J., Resnick, R., Bronson, R., Hemann, M.T., et al. (2010). \href{https://doi.org/10.1038/nature09535}{Stage-specific sensitivity to p53 restoration during lung cancer progression.} Nature \emph{468}, 572--575.

\leavevmode\vadjust pre{\hypertarget{ref-oVtv8JOR}{}}%
Ferone, G., Lee, M.C., Sage, J., and Berns, A. (2020). \href{https://doi.org/10.1101/gad.338228.120}{Cells of origin of lung cancers: lessons from mouse studies.} Genes Dev \emph{34}, 1017--1032.

\leavevmode\vadjust pre{\hypertarget{ref-Vbw51s50}{}}%
Gaiddon, C., Lokshin, M., Ahn, J., Zhang, T., and Prives, C. (2001). \href{https://doi.org/10.1128/mcb.21.5.1874-1887.2001}{A subset of tumor-derived mutant forms of p53 down-regulate p63 and p73 through a direct interaction with the p53 core domain.} Mol Cell Biol \emph{21}, 1874--1887.

\leavevmode\vadjust pre{\hypertarget{ref-gI9Yeagx}{}}%
Gao, J., Aksoy, B.A., Dogrusoz, U., Dresdner, G., Gross, B., Sumer, S.O., Sun, Y., Jacobsen, A., Sinha, R., Larsson, E., et al. (2013). \href{https://doi.org/10.1126/scisignal.2004088}{Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal.} Sci Signal \emph{6}, pl1.

\leavevmode\vadjust pre{\hypertarget{ref-ztWGsqcf}{}}%
Garcia, P.B., and Attardi, L.D. (2014). \href{https://doi.org/10.1016/j.semcdb.2013.12.014}{Illuminating p53 function in cancer with genetically engineered mouse models.} Semin Cell Dev Biol \emph{27}, 74--85.

\leavevmode\vadjust pre{\hypertarget{ref-1EmZp72ZM}{}}%
Gerlinger, M., Rowan, A.J., Horswell, S., Math, M., Larkin, J., Endesfelder, D., Gronroos, E., Martinez, P., Matthews, N., Stewart, A., et al. (2012). \href{https://doi.org/10.1056/nejmoa1113205}{Intratumor heterogeneity and branched evolution revealed by multiregion sequencing.} N Engl J Med \emph{366}, 883--892.

\leavevmode\vadjust pre{\hypertarget{ref-hRW7rjfZ}{}}%
Ghimessy, A., Radeczky, P., Laszlo, V., Hegedus, B., Renyi-Vamos, F., Fillinger, J., Klepetko, W., Lang, C., Dome, B., and Megyesfalvi, Z. (2020). \href{https://doi.org/10.1007/s10555-020-09903-9}{Current therapy of KRAS-mutant lung cancer.} Cancer Metastasis Rev \emph{39}, 1159--1177.

\leavevmode\vadjust pre{\hypertarget{ref-Pje1Soz4}{}}%
Green, S., Trejo, C.L., and McMahon, M. (2015). \href{https://doi.org/10.1158/0008-5472.can-15-1249}{PIK3CA(H1047R) Accelerates and Enhances KRAS(G12D)-Driven Lung Tumorigenesis.} Cancer Res \emph{75}, 5378--5391.

\leavevmode\vadjust pre{\hypertarget{ref-kgaUYMuT}{}}%
Hanahan, D. (2022). \href{https://doi.org/10.1158/2159-8290.cd-21-1059}{Hallmarks of Cancer: New Dimensions.} Cancer Discov \emph{12}, 31--46.

\leavevmode\vadjust pre{\hypertarget{ref-4H4Kjv9w}{}}%
Hanahan, D., and Weinberg, R.A. (2011). \href{https://doi.org/10.1016/j.cell.2011.02.013}{Hallmarks of cancer: the next generation.} Cell \emph{144}, 646--674.

\leavevmode\vadjust pre{\hypertarget{ref-19duX0kT0}{}}%
Herbst, R.S., Heymach, J.V., and Lippman, S.M. (2008). \href{https://doi.org/10.1056/nejmra0802714}{Lung cancer.} N Engl J Med \emph{359}, 1367--1380.

\leavevmode\vadjust pre{\hypertarget{ref-PAjnX7uA}{}}%
Hoadley, K.A., Yau, C., Wolf, D.M., Cherniack, A.D., Tamborero, D., Ng, S., Leiserson, M.D.M., Niu, B., McLellan, M.D., Uzunangelov, V., et al. (2014). \href{https://doi.org/10.1016/j.cell.2014.06.049}{Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin.} Cell \emph{158}, 929--944.

\leavevmode\vadjust pre{\hypertarget{ref-17uu2v5lM}{}}%
Huang, S., Benavente, S., Armstrong, E.A., Li, C., Wheeler, D.L., and Harari, P.M. (2011). \href{https://doi.org/10.1158/0008-5472.can-11-0128}{p53 modulates acquired resistance to EGFR inhibitors and radiation.} Cancer Res \emph{71}, 7071--7079.

\leavevmode\vadjust pre{\hypertarget{ref-bLXbRmvx}{}}%
Iggo, R., Gatter, K., Bartek, J., Lane, D., and Harris, A.L. (1990). \href{https://doi.org/10.1016/0140-6736(90)90801-b}{Increased expression of mutant forms of p53 oncogene in primary lung cancer.} Lancet \emph{335}, 675--679.

\leavevmode\vadjust pre{\hypertarget{ref-cD4M6ONJ}{}}%
Jackson, E.L., Willis, N., Mercer, K., Bronson, R.T., Crowley, D., Montoya, R., Jacks, T., and Tuveson, D.A. (2001). \href{https://doi.org/10.1101/gad.943001}{Analysis of lung tumor initiation and progression using conditional expression of oncogenic K-ras.} Genes Dev \emph{15}, 3243--3248.

\leavevmode\vadjust pre{\hypertarget{ref-KKkLA3TK}{}}%
Jackson, E.L., Olive, K.P., Tuveson, D.A., Bronson, R., Crowley, D., Brown, M., and Jacks, T. (2005). \href{https://doi.org/10.1158/0008-5472.can-05-2193}{The differential effects of mutant p53 alleles on advanced murine lung cancer.} Cancer Res \emph{65}, 10280--10288.

\leavevmode\vadjust pre{\hypertarget{ref-kGt16xDw}{}}%
Jonkers, J., Meuwissen, R., van der Gulden, H., Peterse, H., van der Valk, M., and Berns, A. (2001). \href{https://doi.org/10.1038/ng747}{Synergistic tumor suppressor activity of BRCA2 and p53 in a conditional mouse model for breast cancer.} Nat Genet \emph{29}, 418--425.

\leavevmode\vadjust pre{\hypertarget{ref-Djv3zv5n}{}}%
Juan, J., Muraguchi, T., Iezza, G., Sears, R.C., and McMahon, M. (2014). \href{https://doi.org/10.1101/gad.233627.113}{Diminished WNT -\textgreater{} β-catenin -\textgreater{} c-MYC signaling is a barrier for malignant progression of BRAFV600E-induced lung tumors.} Genes Dev \emph{28}, 561--575.

\leavevmode\vadjust pre{\hypertarget{ref-okfbKJaD}{}}%
Junttila, M.R., Karnezis, A.N., Garcia, D., Madriles, F., Kortlever, R.M., Rostker, F., Brown Swigart, L., Pham, D.M., Seo, Y., Evan, G.I., et al. (2010). \href{https://doi.org/10.1038/nature09526}{Selective activation of p53-mediated tumour suppression in high-grade tumours.} Nature \emph{468}, 567--571.

\leavevmode\vadjust pre{\hypertarget{ref-12CuLfxxd}{}}%
Kandoth, C., McLellan, M.D., Vandin, F., Ye, K., Niu, B., Lu, C., Xie, M., Zhang, Q., McMichael, J.F., Wyczalkowski, M.A., et al. (2013). \href{https://doi.org/10.1038/nature12634}{Mutational landscape and significance across 12 major cancer types.} Nature \emph{502}, 333--339.

\leavevmode\vadjust pre{\hypertarget{ref-9P0mUQ2t}{}}%
Kemp, C.J., Wheldon, T., and Balmain, A. (1994). \href{https://doi.org/10.1038/ng0994-66}{p53-deficient mice are extremely susceptible to radiation-induced tumorigenesis.} Nat Genet \emph{8}, 66--69.

\leavevmode\vadjust pre{\hypertarget{ref-o8jKDrsB}{}}%
Kim, M.P., Li, X., Deng, J., Zhang, Y., Dai, B., Allton, K.L., Hughes, T.G., Siangco, C., Augustine, J.J., Kang, Y., et al. (2021). \href{https://doi.org/10.1158/2159-8290.cd-20-1228}{Oncogenic }. Cancer Discov \emph{11}, 2094--2111.

\leavevmode\vadjust pre{\hypertarget{ref-c63Ld4El}{}}%
Kinsey, C.G., Camolotto, S.A., Boespflug, A.M., Guillen, K.P., Foth, M., Truong, A., Schuman, S.S., Shea, J.E., Seipp, M.T., Yap, J.T., et al. (2019). \href{https://doi.org/10.1038/s41591-019-0367-9}{Protective autophagy elicited by RAF→MEK→ERK inhibition suggests a treatment strategy for RAS-driven cancers.} Nat Med \emph{25}, 620--627.

\leavevmode\vadjust pre{\hypertarget{ref-B0ClcVtn}{}}%
Klemke, L., Fehlau, C.F., Winkler, N., Toboll, F., Singh, S.K., Moll, U.M., and Schulz-Heddergott, R. (2021). \href{https://doi.org/10.3389/fonc.2021.642603}{The Gain-of-Function p53 R248W Mutant Promotes Migration by STAT3 Deregulation in Human Pancreatic Cancer Cells.} Front Oncol \emph{11}, 642603.

\leavevmode\vadjust pre{\hypertarget{ref-RCmUro9f}{}}%
LaFave, L.M., Kartha, V.K., Ma, S., Meli, K., Del Priore, I., Lareau, C., Naranjo, S., Westcott, P.M.K., Duarte, F.M., Sankar, V., et al. (2020). \href{https://doi.org/10.1016/j.ccell.2020.06.006}{Epigenomic State Transitions Characterize Tumor Progression in Mouse Lung Adenocarcinoma.} Cancer Cell \emph{38}, 212--228.e13.

\leavevmode\vadjust pre{\hypertarget{ref-V4ONQpBd}{}}%
Lang, G.A., Iwakuma, T., Suh, Y.-A., Liu, G., Rao, V.A., Parant, J.M., Valentin-Vega, Y.A., Terzian, T., Caldwell, L.C., Strong, L.C., et al. (2004). \href{https://doi.org/10.1016/j.cell.2004.11.006}{Gain of function of a p53 hot spot mutation in a mouse model of Li-Fraumeni syndrome.} Cell \emph{119}, 861--872.

\leavevmode\vadjust pre{\hypertarget{ref-qbfESpXa}{}}%
Laptenko, O., and Prives, C. (2006). \href{https://doi.org/10.1038/sj.cdd.4401916}{Transcriptional regulation by p53: one protein, many possibilities.} Cell Death Differ \emph{13}, 951--961.

\leavevmode\vadjust pre{\hypertarget{ref-13QSbJkMk}{}}%
Levine, A.J. (2020). \href{https://doi.org/10.1038/s41568-020-0262-1}{p53: 800 million years of evolution and 40 years of discovery.} Nat Rev Cancer \emph{20}, 471--480.

\leavevmode\vadjust pre{\hypertarget{ref-SWNWMYmm}{}}%
Levine, A.J., and Oren, M. (2009). \href{https://doi.org/10.1038/nrc2723}{The first 30 years of p53: growing ever more complex.} Nat Rev Cancer \emph{9}, 749--758.

\leavevmode\vadjust pre{\hypertarget{ref-TYj65wKz}{}}%
Li, F.P., Fraumeni, J.F., Mulvihill, J.J., Blattner, W.A., Dreyfus, M.G., Tucker, M.A., and Miller, R.W. (1988). \href{https://www.ncbi.nlm.nih.gov/pubmed/3409256}{A cancer family syndrome in twenty-four kindreds.} Cancer Res \emph{48}, 5358--5362.

\leavevmode\vadjust pre{\hypertarget{ref-WiSGMfl3}{}}%
Lin, L., Asthana, S., Chan, E., Bandyopadhyay, S., Martins, M.M., Olivas, V., Yan, J.J., Pham, L., Wang, M.M., Bollag, G., et al. (2014). \href{https://doi.org/10.1073/pnas.1320956111}{Mapping the molecular determinants of BRAF oncogene dependence in human lung cancer.} Proc Natl Acad Sci U S A \emph{111}, E748--57.

\leavevmode\vadjust pre{\hypertarget{ref-1FKCm8fuC}{}}%
Linzer, D.I., and Levine, A.J. (1979). \href{https://doi.org/10.1016/0092-8674(79)90293-9}{Characterization of a 54K dalton cellular SV40 tumor antigen present in SV40-transformed cells and uninfected embryonal carcinoma cells.} Cell \emph{17}, 43--52.

\leavevmode\vadjust pre{\hypertarget{ref-ku3fwB6c}{}}%
Malhotra, J., Malvezzi, M., Negri, E., La Vecchia, C., and Boffetta, P. (2016). \href{https://doi.org/10.1183/13993003.00359-2016}{Risk factors for lung cancer worldwide.} Eur Respir J \emph{48}, 889--902.

\leavevmode\vadjust pre{\hypertarget{ref-pjuOgKHJ}{}}%
Marjanovic, N.D., Hofree, M., Chan, J.E., Canner, D., Wu, K., Trakala, M., Hartmann, G.G., Smith, O.C., Kim, J.Y., Evans, K.V., et al. (2020). \href{https://doi.org/10.1016/j.ccell.2020.06.012}{Emergence of a High-Plasticity Cell State during Lung Cancer Evolution.} Cancer Cell \emph{38}, 229--246.e13.

\leavevmode\vadjust pre{\hypertarget{ref-SY8r3LQy}{}}%
Matsunaga, M., and Shida, K. (1971). \href{https://www.ncbi.nlm.nih.gov/pubmed/5130083}{{[}Case of progressive external ophthalmoplegia with hemiatrophia totalis{]}.} Naika \emph{28}, 784--788.

\leavevmode\vadjust pre{\hypertarget{ref-btzNHt9A}{}}%
Maynard, A., McCoach, C.E., Rotow, J.K., Harris, L., Haderk, F., Kerr, D.L., Yu, E.A., Schenk, E.L., Tan, W., Zee, A., et al. (2020). \href{https://doi.org/10.1016/j.cell.2020.07.017}{Therapy-Induced Evolution of Human Lung Cancer Revealed by Single-Cell RNA Sequencing.} Cell \emph{182}, 1232--1251.e22.

\leavevmode\vadjust pre{\hypertarget{ref-1DxvFFUzE}{}}%
McNeal, A.S., Belote, R.L., Zeng, H., Urquijo, M., Barker, K., Torres, R., Curtin, M., Shain, A.H., Andtbacka, R.H., Holmen, S., et al. (2021). \href{https://doi.org/10.7554/elife.70385}{BRAF}. Elife \emph{10}.

\leavevmode\vadjust pre{\hypertarget{ref-OuXi55DW}{}}%
Metzger, R.J., Klein, O.D., Martin, G.R., and Krasnow, M.A. (2008). \href{https://doi.org/10.1038/nature07005}{The branching programme of mouse lung development.} Nature \emph{453}, 745--750.

\leavevmode\vadjust pre{\hypertarget{ref-Rky3nm5Q}{}}%
Mollaoglu, G., Guthrie, M.R., Böhm, S., Brägelmann, J., Can, I., Ballieu, P.M., Marx, A., George, J., Heinen, C., Chalishazar, M.D., et al. (2017). \href{https://doi.org/10.1016/j.ccell.2016.12.005}{MYC Drives Progression of Small Cell Lung Cancer to a Variant Neuroendocrine Subtype with Vulnerability to Aurora Kinase Inhibition.} Cancer Cell \emph{31}, 270--285.

\leavevmode\vadjust pre{\hypertarget{ref-7H88gN81}{}}%
Mollaoglu, G., Jones, A., Wait, S.J., Mukhopadhyay, A., Jeong, S., Arya, R., Camolotto, S.A., Mosbruger, T.L., Stubben, C.J., Conley, C.J., et al. (2018). \href{https://doi.org/10.1016/j.immuni.2018.09.020}{The Lineage-Defining Transcription Factors SOX2 and NKX2-1 Determine Lung Cancer Cell Fate and Shape the Tumor Immune Microenvironment.} Immunity \emph{49}, 764--779.e9.

\leavevmode\vadjust pre{\hypertarget{ref-cWKMNUBI}{}}%
Mukhopadhyay, A., and Oliver, T.G. (2014). \href{https://doi.org/10.4161/23723548.2014.969651}{Mighty mouse breakthroughs: a Sox2-driven model for squamous cell lung cancer.} Mol Cell Oncol \emph{2}, e969651.

\leavevmode\vadjust pre{\hypertarget{ref-7Yb2p7xx}{}}%
Muzumdar, M.D., Dorans, K.J., Chung, K.M., Robbins, R., Tammela, T., Gocheva, V., Li, C.M.-C., and Jacks, T. (2016). \href{https://doi.org/10.1038/ncomms12685}{Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers.} Nat Commun \emph{7}, 12685.

\leavevmode\vadjust pre{\hypertarget{ref-wDnAOh0T}{}}%
Nichols, L., Saunders, R., and Knollmann, F.D. (2012). \href{https://doi.org/10.5858/arpa.2011-0521-oa}{Causes of death of patients with lung cancer.} Arch Pathol Lab Med \emph{136}, 1552--1557.

\leavevmode\vadjust pre{\hypertarget{ref-Co9gQ0V6}{}}%
Nikitin, A.Y., Alcaraz, A., Anver, M.R., Bronson, R.T., Cardiff, R.D., Dixon, D., Fraire, A.E., Gabrielson, E.W., Gunning, W.T., Haines, D.C., et al. (2004). \href{https://doi.org/10.1158/0008-5472.can-03-3376}{Classification of proliferative pulmonary lesions of the mouse: recommendations of the mouse models of human cancers consortium.} Cancer Res \emph{64}, 2307--2316.

\leavevmode\vadjust pre{\hypertarget{ref-MLY9vR0E}{}}%
Olive, K.P., Tuveson, D.A., Ruhe, Z.C., Yin, B., Willis, N.A., Bronson, R.T., Crowley, D., and Jacks, T. (2004). \href{https://doi.org/10.1016/j.cell.2004.11.004}{Mutant p53 gain of function in two mouse models of Li-Fraumeni syndrome.} Cell \emph{119}, 847--860.

\leavevmode\vadjust pre{\hypertarget{ref-1Aj9gKIb1}{}}%
Olivier, M., Hollstein, M., and Hainaut, P. (2010). \href{https://doi.org/10.1101/cshperspect.a001008}{TP53 mutations in human cancers: origins, consequences, and clinical use.} Cold Spring Harb Perspect Biol \emph{2}, a001008.

\leavevmode\vadjust pre{\hypertarget{ref-WVwMP2mo}{}}%
Orth, M., Metzger, P., Gerum, S., Mayerle, J., Schneider, G., Belka, C., Schnurr, M., and Lauber, K. (2019). \href{https://doi.org/10.1186/s13014-019-1345-6}{Pancreatic ductal adenocarcinoma: biological hallmarks, current status, and future perspectives of combined modality treatment approaches.} Radiat Oncol \emph{14}, 141.

\leavevmode\vadjust pre{\hypertarget{ref-E3yjcNvW}{}}%
Perry, M.E., Piette, J., Zawadzki, J.A., Harvey, D., and Levine, A.J. (1993). \href{https://doi.org/10.1073/pnas.90.24.11623}{The mdm-2 gene is induced in response to UV light in a p53-dependent manner.} Proc Natl Acad Sci U S A \emph{90}, 11623--11627.

\leavevmode\vadjust pre{\hypertarget{ref-BpRBkUVl}{}}%
Planchard, D., Besse, B., Groen, H.J.M., Souquet, P.-J., Quoix, E., Baik, C.S., Barlesi, F., Kim, T.M., Mazieres, J., Novello, S., et al. (2016). \href{https://doi.org/10.1016/s1470-2045(16)30146-2}{Dabrafenib plus trametinib in patients with previously treated BRAF(V600E)-mutant metastatic non-small cell lung cancer: an open-label, multicentre phase 2 trial.} Lancet Oncol \emph{17}, 984--993.

\leavevmode\vadjust pre{\hypertarget{ref-4S1jzORt}{}}%
Planchard, D., Besse, B., Groen, H.J.M., Hashemi, S.M.S., Mazieres, J., Kim, T.M., Quoix, E., Souquet, P.-J., Barlesi, F., Baik, C., et al. (2021). \href{https://doi.org/10.1016/j.jtho.2021.08.011}{Phase 2 Study of Dabrafenib Plus Trametinib in Patients With BRAF V600E-Mutant Metastatic NSCLC: Updated 5-Year Survival Rates and Genomic Analysis.} J Thorac Oncol \emph{17}, 103--115.

\leavevmode\vadjust pre{\hypertarget{ref-kQzAqqtP}{}}%
Politi, K., Zakowski, M.F., Fan, P.-D., Schonfeld, E.A., Pao, W., and Varmus, H.E. (2006). \href{https://doi.org/10.1101/gad.1417406}{Lung adenocarcinomas induced in mice by mutant EGF receptors found in human lung cancers respond to a tyrosine kinase inhibitor or to down-regulation of the receptors.} Genes Dev \emph{20}, 1496--1510.

\leavevmode\vadjust pre{\hypertarget{ref-1fsz5jxh}{}}%
Powell, S.M., Zilz, N., Beazer-Barclay, Y., Bryan, T.M., Hamilton, S.R., Thibodeau, S.N., Vogelstein, B., and Kinzler, K.W. (1992). \href{https://doi.org/10.1038/359235a0}{APC mutations occur early during colorectal tumorigenesis.} Nature \emph{359}, 235--237.

\leavevmode\vadjust pre{\hypertarget{ref-1CvrKB1k3}{}}%
Reita, D., Pabst, L., Pencreach, E., Guérin, E., Dano, L., Rimelen, V., Voegeli, A.-C., Vallat, L., Mascaux, C., and Beau-Faller, M. (2022). \href{https://doi.org/10.3390/cancers14051321}{Direct Targeting }. Cancers (Basel) \emph{14}.

\leavevmode\vadjust pre{\hypertarget{ref-CgOGrXoS}{}}%
Robert, C., Karaszewska, B., Schachter, J., Rutkowski, P., Mackiewicz, A., Stroiakovski, D., Lichinitser, M., Dummer, R., Grange, F., Mortier, L., et al. (2014). \href{https://doi.org/10.1056/nejmoa1412690}{Improved overall survival in melanoma with combined dabrafenib and trametinib.} N Engl J Med \emph{372}, 30--39.

\leavevmode\vadjust pre{\hypertarget{ref-sufLyk4G}{}}%
Rogers, Z.N., McFarland, C.D., Winters, I.P., Naranjo, S., Chuang, C.-H., Petrov, D., and Winslow, M.M. (2017). \href{https://doi.org/10.1038/nmeth.4297}{A quantitative and multiplexed approach to uncover the fitness landscape of tumor suppression in vivo.} Nat Methods \emph{14}, 737--742.

\leavevmode\vadjust pre{\hypertarget{ref-17E8qJFQ0}{}}%
Rogers, Z.N., McFarland, C.D., Winters, I.P., Seoane, J.A., Brady, J.J., Yoon, S., Curtis, C., Petrov, D.A., and Winslow, M.M. (2018). \href{https://doi.org/10.1038/s41588-018-0083-2}{Mapping the in vivo fitness landscape of lung adenocarcinoma tumor suppression in mice.} Nat Genet \emph{50}, 483--486.

\leavevmode\vadjust pre{\hypertarget{ref-PX0Ki15S}{}}%
Rusch, V., Klimstra, D., Venkatraman, E., Oliver, J., Martini, N., Gralla, R., Kris, M., and Dmitrovsky, E. (1995). \href{https://www.ncbi.nlm.nih.gov/pubmed/7585548}{Aberrant p53 expression predicts clinical resistance to cisplatin-based chemotherapy in locally advanced non-small cell lung cancer.} Cancer Res \emph{55}, 5038--5042.

\leavevmode\vadjust pre{\hypertarget{ref-UfKKrJ6l}{}}%
Sainz de Aja, J., and Kim, C.F. (2020). \href{https://doi.org/10.1016/j.cell.2019.12.020}{May the (Mechanical) Force Be with AT2.} Cell \emph{180}, 20--22.

\leavevmode\vadjust pre{\hypertarget{ref-Ntv29NRu}{}}%
Shai, A., Dankort, D., Juan, J., Green, S., and McMahon, M. (2015). \href{https://doi.org/10.1158/0008-5472.can-14-3701}{TP53 Silencing Bypasses Growth Arrest of BRAFV600E-Induced Lung Tumor Cells in a Two-Switch Model of Lung Tumorigenesis.} Cancer Res \emph{75}, 3167--3180.

\leavevmode\vadjust pre{\hypertarget{ref-gY2NonCA}{}}%
Siegel, R.L., Miller, K.D., Fuchs, H.E., and Jemal, A. (2021). \href{https://doi.org/10.3322/caac.21654}{Cancer Statistics, 2021}. CA A Cancer J. Clin. \emph{71}, 7--33.

\leavevmode\vadjust pre{\hypertarget{ref-4BJVCLfM}{}}%
Strong, L.C. (1978). \href{https://doi.org/10.1002/jso.2930100410}{Three mice, two blind.} J Surg Oncol \emph{10}, 353--360.

\leavevmode\vadjust pre{\hypertarget{ref-dAZ32h3a}{}}%
Sutherland, K.D., Song, J.-Y., Kwon, M.C., Proost, N., Zevenhoven, J., and Berns, A. (2014). \href{https://doi.org/10.1073/pnas.1319963111}{Multiple cells-of-origin of mutant K-Ras-induced mouse lung adenocarcinoma.} Proc Natl Acad Sci U S A \emph{111}, 4952--4957.

\leavevmode\vadjust pre{\hypertarget{ref-fXe5uEAl}{}}%
Tabbò, F., Pisano, C., Mazieres, J., Mezquita, L., Nadal, E., Planchard, D., Pradines, A., Santamaria, D., Swalduz, A., Ambrogio, C., et al. (2021). \href{https://doi.org/10.1016/j.ctrv.2021.102335}{How far we have come targeting BRAF-mutant non-small cell lung cancer (NSCLC).} Cancer Treat Rev \emph{103}, 102335.

\leavevmode\vadjust pre{\hypertarget{ref-NLPwvKCP}{}}%
Treutlein, B., Brownfield, D.G., Wu, A.R., Neff, N.F., Mantalas, G.L., Espinoza, F.H., Desai, T.J., Krasnow, M.A., and Quake, S.R. (2014). \href{https://doi.org/10.1038/nature13173}{Reconstructing lineage hierarchies of the distal lung epithelium using single-cell RNA-seq.} Nature \emph{509}, 371--375.

\leavevmode\vadjust pre{\hypertarget{ref-jKCnCnG4}{}}%
Truong, A., Yoo, J.H., Scherzer, M.T., Sanchez, J.M.S., Dale, K.J., Kinsey, C.G., Richards, J.R., Shin, D., Ghazi, P.C., Onken, M.D., et al. (2020). \href{https://doi.org/10.1158/1078-0432.ccr-20-1675}{Chloroquine Sensitizes }. Clin Cancer Res \emph{26}, 6374--6386.

\leavevmode\vadjust pre{\hypertarget{ref-1GkFEmggg}{}}%
van Veen, J.E., Scherzer, M., Boshuizen, J., Chu, M., Liu, A., Landman, A., Green, S., Trejo, C., and McMahon, M. (2019). \href{https://doi.org/10.7554/elife.43668}{Mutationally-activated PI3'-kinase-α promotes de-differentiation of lung tumors initiated by the BRAF}. Elife \emph{8}.

\leavevmode\vadjust pre{\hypertarget{ref-14fESvHZW}{}}%
Vogelstein, B., Lane, D., and Levine, A.J. (2000). \href{https://doi.org/10.1038/35042675}{Surfing the p53 network.} Nature \emph{408}, 307--310.

\leavevmode\vadjust pre{\hypertarget{ref-S6qbb6kC}{}}%
Walter, D.M., Venancio, O.S., Buza, E.L., Tobias, J.W., Deshpande, C., Gudiel, A.A., Kim-Kiselak, C., Cicchini, M., Yates, T.J., and Feldser, D.M. (2017). \href{https://doi.org/10.1158/0008-5472.can-16-2159}{Systematic }. Cancer Res \emph{77}, 1719--1729.

\leavevmode\vadjust pre{\hypertarget{ref-11g54rA8N}{}}%
Warren, G.W., and Cummings, K.M. (2013). \href{https://doi.org/10.14694/edbook_am.2013.33.359}{Tobacco and lung cancer: risks, trends, and outcomes in patients with cancer.} Am Soc Clin Oncol Educ Book 359--364.

\leavevmode\vadjust pre{\hypertarget{ref-jHVcJWec}{}}%
Weissmueller, S., Manchado, E., Saborowski, M., Morris, J.P., Wagenblast, E., Davis, C.A., Moon, S.-H., Pfister, N.T., Tschaharganeh, D.F., Kitzing, T., et al. (2014). \href{https://doi.org/10.1016/j.cell.2014.01.066}{Mutant p53 drives pancreatic cancer metastasis through cell-autonomous PDGF receptor β signaling.} Cell \emph{157}, 382--394.

\leavevmode\vadjust pre{\hypertarget{ref-zOzqGxQM}{}}%
Westcott, P.M.K., Halliwill, K.D., To, M.D., Rashid, M., Rust, A.G., Keane, T.M., Delrosario, R., Jen, K.-Y., Gurley, K.E., Kemp, C.J., et al. (2014). \href{https://doi.org/10.1038/nature13898}{The mutational landscapes of genetic and chemical models of Kras-driven lung cancer.} Nature \emph{517}, 489--492.

\leavevmode\vadjust pre{\hypertarget{ref-CVOIe1Dg}{}}%
Wijnhoven, S.W.P., Zwart, E., Speksnijder, E.N., Beems, R.B., Olive, K.P., Tuveson, D.A., Jonkers, J., Schaap, M.M., van den Berg, J., Jacks, T., et al. (2005). \href{https://doi.org/10.1158/0008-5472.can-05-1650}{Mice expressing a mammary gland-specific R270H mutation in the p53 tumor suppressor gene mimic human breast cancer development.} Cancer Res \emph{65}, 8166--8173.

\leavevmode\vadjust pre{\hypertarget{ref-2gOIsLdX}{}}%
Winters, I.P., Chiou, S.-H., Paulk, N.K., McFarland, C.D., Lalgudi, P.V., Ma, R.K., Lisowski, L., Connolly, A.J., Petrov, D.A., Kay, M.A., et al. (2017). \href{https://doi.org/10.1038/s41467-017-01519-y}{Multiplexed in vivo homology-directed repair and tumor barcoding enables parallel quantification of Kras variant oncogenicity.} Nat Commun \emph{8}, 2053.

\leavevmode\vadjust pre{\hypertarget{ref-tR3WrnP9}{}}%
Woodard, G.A., Jones, K.D., and Jablons, D.M. \href{https://doi.org/10.1007/978-3-319-40389-2_3}{Lung Cancer Staging and Prognosis.} Cancer Treat Res \emph{170}, 47--75.

\leavevmode\vadjust pre{\hypertarget{ref-Mb1Q4pdk}{}}%
Wu, X., Bayle, J.H., Olson, D., and Levine, A.J. (1993). \href{https://doi.org/10.1101/gad.7.7a.1126}{The p53-mdm-2 autoregulatory feedback loop.} Genes Dev \emph{7}, 1126--1132.

\leavevmode\vadjust pre{\hypertarget{ref-FfxdQ0tV}{}}%
Xue, Y., Hou, S., Ji, H., and Han, X. (2016). \href{https://doi.org/10.1007/s13238-016-0330-1}{Evolution from genetics to phenotype: reinterpretation of NSCLC plasticity, heterogeneity, and drug resistance.} Protein Cell \emph{8}, 178--190.

\leavevmode\vadjust pre{\hypertarget{ref-3xXZMpos}{}}%
Yoshida, K., Gowers, K.H.C., Lee-Six, H., Chandrasekharan, D.P., Coorens, T., Maughan, E.F., Beal, K., Menzies, A., Millar, F.R., Anderson, E., et al. (2020). \href{https://doi.org/10.1038/s41586-020-1961-1}{Tobacco smoking and somatic mutations in human bronchial epithelium.} Nature \emph{578}, 266--272.

\leavevmode\vadjust pre{\hypertarget{ref-kitgYOkt}{}}%
Zhang, Y., Xiong, S., Liu, B., Pant, V., Celii, F., Chau, G., Elizondo-Fraire, A.C., Yang, P., You, M.J., El-Naggar, A.K., et al. (2018). \href{https://doi.org/10.1038/s41467-018-06146-9}{Somatic Trp53 mutations differentially drive breast cancer and evolution of metastases.} Nat Commun \emph{9}, 3953.

\leavevmode\vadjust pre{\hypertarget{ref-bZo8j50s}{}}%
Zheng, Q., Peacock, D.M., and Shokat, K.M. (2022). \href{https://doi.org/10.1021/acs.jmedchem.2c00099}{Drugging the Next Undruggable KRAS Allele-Gly12Asp.} J Med Chem \emph{65}, 3119--3122.

\leavevmode\vadjust pre{\hypertarget{ref-N2yNjRyX}{}}%
Zhu, J., Woods, D., McMahon, M., and Bishop, J.M. (1998). \href{https://doi.org/10.1101/gad.12.19.2997}{Senescence of human fibroblasts induced by oncogenic Raf.} Genes Dev \emph{12}, 2997--3007.

\end{CSLReferences}
