## Chapter Two: Quantitative tumor burden and small library screen using TUBA-SEQ 
{.page_break_before}

# Abstract


# Introduction

Next generation sequencing has allowed for the unbiased identification of genes that are mutated in human cancer [@pmid:25079552; @pmid:25109877; @pmid:24132290].
The wealth of knowledge gained from such large-scale sequencing projects has revealed actionable mutationas that can be targeted with small molecule inhibitors, such as dabrafenib and trametinib.
However, Identificaton of possible targets is not enough, as many cancers with similarly mutated oncogenes, such as BRAF^V600E^, respond differentially to BRAF^V600E^-targeed therapy.
For example,  mealanoma patients with BRAF^V600E^ mutations respond well to BRAF^V600E^-targeted therapy whereas colorectal cancer patients with BRAF^V600E^ mutations do not respond[@pmid:22448344].
The disconnect between putative driver oncogenes identified in human cancers and response to targeted therapy is a problem that can be solved with rigorous pre-clinical experimentation, such as in GEMMs. 

Genomic sequencing of human tumors has also been crucial insight into how cancers form and what signaling nodes might be important for progressive stages of cancer evolution.
For example, pancreatic cancers have a very stereotypical histiological progression that is brought about by known mutational activation.
Specifically, mutations in _KRAS_ initiate tumorigenesis and loss of _TP53_ function and _SMAD3_ are associated with cancer progression[@pmid:31395068].
However, this is more difficult to do in lung adenocarcinoma because of the high mutational burden often due to the consumption of cigarettes[@pmid:27811275; @pmid:31996850].


Indeed, in highly mutated cancers such as melanoma and lung cancer, recurrent mutations are not necessarily predictive of biological importance.
To better understand the importance of candidate tumor suppressors or oncogenes, researchers exploit tumor cell lines and GEMMs to identify bona-fide genetic drivers of cancer.
However, these systems are suboptimal, as they are either not physiologically relevant (tissue culture) or are not scalable to determine the effect of many mutations (GEMMs).
Moreover, techniques to measure cooperation of genes to drive lung cancer are qualitative and are not sufficiently quantitative to allow for large-scale studies.
Typically, lung adenocarcinoma is quantified by lung weight or histological analyses such as tumor burden (Percentage of lung that is tumor), tumor size, and immunohistochemistry (IHC) of tumor markers.
Although important and informative, these analyses are not high-throughput and rely on many mice to acheive statistical confidence. Furthermore, there is intrinsic bias when calculating tumor size and tumor burden, as tissue-sectioning results in tumor sections that do not accurately portray true tumor size. Therefore, more advanced techniques are required for large-scale analysis of mouse tumors that are both quantitative and high throughput.

Recently, Monte Winslow and his lab at Stanford University has developed **Tu**mor- **Ba**rcode **Seq**uencing (Tuba-Seq), a rapid, multiplexed, quantitative assay for analyzing the contribution of tumor-suppressor loss _in vivo_.
This technique works by delivering a uniquely barcoded lentiviral endoing Cre recombinase, to activate Cre-mediated oncogenes, and a guide RNA, to inactivate a single tumor suppressor gene.
Therefore, each tumor will be driven by the same oncogene, but will have different tumor suppressors inactivated.
After a period of time (6-24 weeks) tissue is harvested for whole-lung DNA extraction and targeted DNA-sequencing is performed to quantify relative tumor size to spike-in controls[@pmid:28530655].
Therefore, contribution of a single tumor suppressor can be quantified in single animal. 

BRAF^V600E^ has previously been shown to be a potent driver of lung tumorigenesis[@pmid:17299132].
However, tumor fails to progress past adenoma-like tumors.
Given the theory that BRAF-^V600E^ induces growth arrest after an initial burst of neoplastic growth, it would be of interest to know if all tumors grow to a predetermined and absolute size, or if there is a spectrum of tumor sizes that are determined by extrinsic factors such as proximity to blood vessels or certain stromal cells.
Therfore, we used Tuba-Seq combined with CRISPR/CAS9 genome editing to explore the contribution of 11 tumor suppressor genes and 5 controls in the context of our BRAF ^V600E^- driven lung tumor models.

# Results
Adenoviral vectors and lentiviral vectors introduced in the lungs of GEMMs offer temporal control and can be titrated to acheive an optimal level of tumor numerosity. 
However, only lentiviral vectors can be used to barcode tumors because they integrate into the genome while adenoviruses do not[@pmid:19561589; @pmid:28530655].
Together with CRISPR/CAS9 gene editing, lentiviruses have been used to analyzed many genetic interactions of oncogenic _Kras_ driven lung adenocarcinoma[@pmid:29233960;@pmid:28530655].
To enable simultaneous actvation of BRAF^V600E^ and loss of tumor supressor genes, we bred _Braf^CAT^_; _H11b^LSL-CAS9/+^_ mice. 
To  determine the size of BRAF ^V600E^ tumors and the contribution of 11 tumor suppressor genes (_Trp53, Rb1, Cdkn2a, Apc, Rbm10, Stk11, Setd2, Arid1a, Atm, Keap1, Smad4)_ on BRAF ^V600E^ lung tumorigenesis, we obtained tittered barcoded-lentiviral plasmids that encode Cre-recombinase as well as a sgRNA that inactivates one of eleven tumor suppressors common in lung adenocarcinoma.
All genes tested are putative tumor suppressor genes commonly lost in human lung cancer patients. Each lentiviral solution contained ~1000 individually barcoded lentiviruses.
We infected both _LSL-Cas9_ and _CAS9_ deficient mice in order to determine the contribution of genetic manipulation and/or deletion of the targeted gene.
Mice meant to be analyzed after 8 weeks received a viral titer of 1x10^5 while mice mean to be analyzed after 16 weeks received a viral titer of 1x10^4. After euthanasia, lungs were collected for genomic DNA extraction, and barcode amplification followed by preperation of sequencing libraries.
Before genomic extraction and library preparation, a known quantity of known-barcoded cells was added in order to benchmark sequencing reads to a known cell number(figure).

We determined the mean size of a BRAF^V600E^ driven tumor to be 21,248 thousand cells (figure). 
Interestingly, there is a widespread distribution of tumor sizes with the largest BRAF^V600E^ tumor to be ~300,000 cells along with many small tumors (figure).

Gross histologic images of experimental lungs are show an increase in tumor burden with both time (Figure xxx) and CAS9 status (figure xxx). 
Therefore, based on macroscopic inspection, at least one tumor suppressor appeared to cooperate with BRAF ^V600E^ to increase tumor burden.
In order to determine the fitness landscape of any one particular tumor suppressor, we calculated the relative number of tumor cells in a given tumor using the Tuba-Seq genomics pipeline.
After 8 weeks post tumor initiation there no statistically significant contribution of tumor suppressor loss (Figure). 
SETD2 loss was trending towards statistical significance. 
However, 16 weeks post initiation revealed both SETD2 loss and APC loss cooperated with BRAF ^V600E^ to form larger tumors (Figure x). 
APC loss was a suspected hit as we have previously shown that activation of Wnt-beta-catenin signaling promoted BRAF^V600E^ tumorigenesis. 
As APC loss is predicted to increase the activity of Beta-catenin signaling, this result was unsurprising to us. 
However, loss of SETD2 was a novel finding in the context of BRAF ^V600E^ lung tumorigenesis.

Given our findings of SETD2 loss increased tumor burden concurrent with BRAF^V600E^ we next validated SETD2 in a new cohort of mice with single-guide RNAs targeting SETD2. 
Indeed, loss of SETD2 increased tumor burden from 5% to over 30% in some animals.
Since SETD2 antibodies are unreliable in both western blots and immunohistochemistry we used H3K36me3 as a surrogate marker for SETD2 activity by immunohistochemistry. 
Whereas 90% of nuclei in BRAF ^V600E^ only tumors were H3K36me3, more than 60% tumors of mice infected with SETD2 CRISPR virus were negative for H3K36me3.
Furthermore, when classified by H3K36me3-positive or -negative, there is even stronger evidence that SETD2 loss leads to larger tumor formation.

# Discussion
The mutational landscape of lung adenocarcinoma is complex due lifelong bombardment on DNA by carcinogen ingested into the distal lung epithelium[23770567].
While understanding which altered genes play causal roles in tumor formation and progression is required for personalized medicine, the noisy mutation profiles in lung adenocarcinoma makes it hard to do so.
However, it is clear that there is a central role for the MAPK pathway during the initiation and progression of lung adenocarcinoma [@pmid:28228261].
Oncogenic BRAF^V600E^ is a major driver of lung adenocarcinoma.
However, mouse models with activation of BRAF ^V600E^ only lead to benign tumorigenesis[@pmid:26001956; @pmid:17299132].
Although it is known that activation of WNT and PI3k signaling, as well as lost of _Trp53_ and _Cdkn2a_ genes allow BRAF^V600E^ benign lung tumors to progress to malignant disease, it is a time-consuming and laborious process to effects of individual tummor suppressors in GEMMs.
Therefore, we sought out to systematically test a variety of known tumor suppressors in lung adenocarcinoma.
Furthermore,until now, we did not know the absolute number of cells in a BRAF^V600E^ adenoma, only its area, which is confounded by tissue sectioning as well as stromal cell contamination.
Prior to Tuba-Seq, it was very difficult to accurately determine the size of BRAF^V600E^ adenomas.
Based on population doublings, as well as cross sections of BRAF ^V600E^ tumors in histological sections, it is estimated that BRAF^V600E^-driven lung adenomas grow about 15-20 populations doublings and reaching a size of 1mm^3^[@pmid:24589553]. 
With Tuba-Seq, we accurately discerned the absolute size of all BRAF^V600E^ lung adenomas, regardless of size.
Interestingly, our results have revealed BRAF^V600E^-driven tumors widely differ in absloute tumor cell number. Our results support the hypothesis that each tumor-initiating cell has its own unique barriers for tumor outgrowth. 
This heterogenous reponse to BRAF^V600E^ activation likely reflects intrinisic differences between individual cell states. 
Indeed, tumorigenic potential can vary greatly due to different cell types (AT2 vs Club cell vs AT1), or can be influenced by subtle differences within a cell type, such proximity to supporting stromal cells such as fibroblast or immune cells. 
Since we used a ubiquitous promoter to drive BRAF^V600E^ and CAS9 expression, it is likely that not all sequence reads come from adenomas or tumors, but rather from normal cell turnover during tissue homeostasis. 
Therefore, in out quantifcations and analysis, we focused on tumors that were greater than 5,000 cells. 

Combining Tuba-Seq with CRISPR/CAS9 gene editing of eleven putative tumor supressor genes with four inert control sgRNAs, we identified both APC and SETD2 loss as a cooperating genetic event that allows BRAF^V600E^ tumors to grow larger. 
Therefore, both APC and SETD2 are bona fide tumor suppressors in lung adenocarinoma. 
APC loss is known to activate the Wnt-signaling pathway and is a well establish tumor supressor in colon cancer[@pmid:1528264].
Furthermore, our lab has previously published a requirement for WNT-signaling for progression of BRAF^V600E^ lung tumors[@pmid:24589553].
Specifically, inhibition of WNT signaling by a Porcupine inhibitor prevent tumor onset, while constitutive activation of the pathway drive tumor progression to high grade. 
Therefore, it is not surprising that APC loss cooperates with BRAF^V600E^ to induce tumor growth.

We have also identified _SETD2_ loss as a potent tumor supressor(Figure 2).
_SETD2_ is mutated in other solid malignancies on cBioPortal, such as Kidney and bladder cancers[@pmid:22588877]. 
SETD2 is a histone methyltransferase that deposits trimethylation marks on H3K36[@pmid:33023640]. 
H3K36me^3^ is used by chromatin modifiers to mark actively trascribed genes[@pmid:18157086]. 
mining of TCGA mRNA expression and survival analysis further revealed that _SETD2_ mRNA is lower in lung adenocarcinoma and is associated with poorer overall survival(Figure 2). 
Interestingly, _SETD2_ alterations are also significantly associated with BRAF^V600E^ mutations (Figure 2).
The unique association of _BRAF^V600E^_ mutation and _SETD2_ loss may reflect an interesting synergistic effect between MAPK signaling and loss of chromatin regulation. 
One hypothesis is that SETD2 loss drives spurious transcriptional activation.  
Similarily, Recent work by Monte Winslow and also David Feldser's labs have indentified SETD2 loss as a cooperating event in KRAS-G12D lung tumorigenesis[@pmid:28202515]. 
However, the mechanism by which _SETD2_ loss drives tumor progression in either BRAF^V600E^ or KRAS-G12D remain elusive. 

Although we have identified novel tumor suppressors in BRAF^V600E^ lung tumorigenesis using Tuba-Seq, we have failed to confirm previously identified tumor suppressors _CdkN2a_ and _Trp53_ as cooperating events.
One hypothesis is that the sgRNAs targeting _Trp53_ and _Cdkn2a_ are inefficient. However, previous Tuba-Seq experiments did confirm _Trp53_ editing and even demonstrated tumorigeneic cooperation with KRAS-G12D[@pmid:29610476].
Another hypothesis is that the effect size of _trp53_ loss is so small that many mice and tumors are needed to detect a robust cooperation. 
Indeed, p53 loss is not that to suddenly make tumors immediately proliferate. 
Rather, p53 loss merely allows for tumor cells to sample various transcriptional programs that can be co-opted for increased fitness in the face of stress.
Along similar lines, perhaps 16-weeks is too little time to allow for _Trp53_ or _Cdkn2a_ deficient tumors to proliferate. 
Likely, it is a combination of small effect size, ineficient gene editing, and insufficient time that prevented us from identifying _Trp53_ as a tumor supressor in BRAF6V600E^ driven lung tumorigenesis using Tuba-Seq combined with CRISPR/CAS9.  

Since GEMMs lack the mutational burden exhibited by human lung adenocarcinomas, they are good models to study gene-gene interactions in tumorigenesis. 
Our data provides further insight into what biological mechanisms are important for BRAF^V600E^ lung tumorigenesis.
Additional studies of SETD2's role in lung adenocarcinoma will further reveal how it alters the evolution of lung adenocarcinomas. 
Furthermore, given its striking association with _BRAF^V600E^_ mutations, it will be important to know how _SETD2_ alters response to BRAF^V600E^-targeted therapy. 
Full understanding of the consequences and outcomes of genetic cooperation between BRAF^V600E^ and tumor supressors or oncogenes will help clinicians better understand tumor evolution and paths to drug resistance. 
Utltimetly, aiding to the refinement of cancer biology knowledge and precision oncology.



![**Figure 2.1. Overview of Tuba-Seq experimentation** (A) Lenti-sgRNA-Cre plasmid library with collection of targeted tumor supressors. (B) _Braf^CAT/+^_ and _Braf^CAT/+^;H11b^LSL-Cas9/+^_ were infected with either 10^4^ (16 week _Braf^CAT/+^; H11b^LSL-Cas9/+^_) or 10^5^ (8 weeks _Braf^CAT/+^; H11b^LSL-Cas9/+^_ and 16 weeks _Braf^CAT/+^; H11b^LSL-Cas9/+^_) (B) Representative gross images of experimental mouse lungs.](images/tuba1.png){#fig:04 .white width="100%"}

![**Figure 2.2. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline.** Tumor size corresponds to dot size. Mean tumor number for 16 weeks _Braf^CAT/+^; H11b^+/+^_ is 21,248 cells. Mean tumor number for 8 weeks _Braf^CAT/+^; H11b^LSL-Cas9/+^_ is 9,095 cells. Mean tumor number for 16 week _Braf^CAT/+^; H11b^LSL-Cas9/+^_ is 29,867 cells.  ](images/tuba2.png){#fig:05 .white width="100%"}

![**Figure 2.3. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline seperated by sgID.** (A) 16 weeks _Braf^CAT/+^; H11b^+/+^_(negative control) has no apparent cooperative genetic events. (B) 8 weeks _Braf^CAT/+^; H11b^LSL-Cas9/+^_ identifed Apc loss as a cooperating genetic event based on increase in tumor cell number (C) 16 week _Braf^CAT/+^; H11b^LSL-Cas9/+^_ identified Apc and Setd2 loss as cooperating events.](images/tuba3.png){#fig:06 .white width="100%"}  
  
![**Figure 2.4. Validation of _Setd2_ loss as cooperating genetic event in BRAF^V600E^ lung cancer.** (A) Representative images of lung tumor histology of 16 weeks _Braf^CAT/+^; H11b^LSL-Cas9/+^_ mice infected with 2.5x10^4^ PFU Lenti-sgSetd2-Cre or Lenti-sgNT1-Cre. (B) Quantification of tumor size based on histology, n= 30 tumors. * = _p-value<.05.](images/tuba4.png){#fig:07 .white width="100%"}

![**Figure 2.5. H3K36me^3^ is lost in larger tumors in _Set2d_-edited lung tumors.**(A) Representative images of H3K36me3 (surrogate for _setd2_ loss) depicting either "high" or "low" staining indicative of protein expression. (B) Quantification of H3K36me^3^ positive tumors in either _sgSetd2_ or _sgNT1_ targeted mice. *= _p-value<.05_ for negative tumors. (C) Quantification of tumor size paired with H3K36me^3^ negative tumors. Tumor size and H3K36me^3^ positivty were simuletaneously measured using QuantCenter and binned in either "low" or "High" and then average tumor size was determined for each group. *= _p-value<.05_.](images/tuba5.png){#fig:08 .white width="100%"}

![**Figure 2.6. _SETD2_ loss is common in human cancers, an indicator of poor prognosis and frequently occurs with _BRAF^V600E^_ mutations.**(A) Survival analysis with "Survival" R package indicates a trend towards poorer overall survival with low _SETD2_ mRNA in TGCA Lung adenocarcinoma dataset. (B) _SETD2_ mRNA is attenuated in Lung adenocarcinoma compared to normal lung tissue (C) _SETD2_ mutations significantly co-occur with BRAF^V600E^ mutations and are present in 40% of _BRAF^V600E^) mutant Lung Adenocarcinoma.](images/tuba6.png){#fig:09 .white width="100%"}

