<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Michael" />
  <meta name="dcterms.date" content="2022-04-08" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Dissertation</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Dissertation" />
  <meta name="citation_title" content="Dissertation" />
  <meta property="og:title" content="Dissertation" />
  <meta property="twitter:title" content="Dissertation" />
  <meta name="dc.date" content="2022-04-08" />
  <meta name="citation_publication_date" content="2022-04-08" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Michael" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="twitter:creator" content="@johndoe" />
  <link rel="canonical" href="https://scherzerthesis.github.io/thesis/" />
  <meta property="og:url" content="https://scherzerthesis.github.io/thesis/" />
  <meta property="twitter:url" content="https://scherzerthesis.github.io/thesis/" />
  <meta name="citation_fulltext_html_url" content="https://scherzerthesis.github.io/thesis/" />
  <meta name="citation_pdf_url" content="https://scherzerthesis.github.io/thesis/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://scherzerthesis.github.io/thesis/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://scherzerthesis.github.io/thesis/v/7a02dfb8a9274d8e9859aa2cf500dd7720d847a5/" />
  <meta name="manubot_html_url_versioned" content="https://scherzerthesis.github.io/thesis/v/7a02dfb8a9274d8e9859aa2cf500dd7720d847a5/" />
  <meta name="manubot_pdf_url_versioned" content="https://scherzerthesis.github.io/thesis/v/7a02dfb8a9274d8e9859aa2cf500dd7720d847a5/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Dissertation</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://scherzerthesis.github.io/thesis/v/7a02dfb8a9274d8e9859aa2cf500dd7720d847a5/">permalink</a>)
was automatically generated
from <a href="https://github.com/scherzerthesis/thesis/tree/7a02dfb8a9274d8e9859aa2cf500dd7720d847a5">scherzerthesis/thesis@7a02dfb</a>
on April 8, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><strong>Michael</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/johndoe">johndoe</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/johndoe">johndoe</a><br>
<small>
Department of Something, University of Whatever
· Funded by Grant XXXXXXXX
</small></li>
</ul>
<h2 class="page_break_before" id="chapter-one-introduction">Chapter One: Introduction</h2>
<p><strong>Abstract</strong>
Lung cancer remains a threat to human health throughout the world. Due to the accumulation of mutations from tobacco or otherwise, lung cancers have a vast mutational spectrum that affects many genes. These characteristics make lung cancer a challenge in understanding how the disease forms. Furthermore, mutational tumor heterogeneity is linked to drug resistance and a poor prognosis for patients. Interestingly, MAPK pathway activation appears to be a prerequisite for lung adenocarcinoma formation. In chapter 1, using genetically engineered mouse models of BRAF<sup>V600E</sup> lung tumorigenesis, we analyzed the contribution of commonly mutated genes to tumor progression. It was found that, while BRAF<sup>V600E</sup> drives benign tumor formation, loss of SETD2 function drives tumor progression to larger tumors. Thus, we identified SETD2 as a common alteration in BRAF<sup>V600E</sup> lung cancer and is a bone fide tumor suppressor.</p>
<p>Genetic sequencing of human lung adenocarcinomas has also revealed the widespread inactivation of p53 as a common alteration in late-stage lung adenocarcinoma. Normal p53 function restricts tumor plasticity, and thus losing p53 function enables tumor cells to adopt more malignant cell identities. However, not all p53 mutations are created equal. In chapter 2, we characterized the effect of three p53 mutant alleles (P53<sup>fl/fl</sup>, P53<sup>R172H</sup>, and p53<sup>R245W</sup>) on BRAF<sup>V600E</sup> lung tumorigenesis. It was found that p53<sup>R172H</sup> and p53<sup>R245W</sup> induce more widespread tumor dedifferentiation, thus leading to worst tumor outcomes compared to p53-loss. Cell lines derived from the tumors allowed for the dissection of signaling pathway dynamics, which led us to observe higher MAPK pathway activation in p53<sup>R172H</sup> and p53<sup>R245W</sup> cell lines. We also found that p53<sup>R172H</sup> and p53<sup>R245W</sup> cell lines were more refractory to MAPK pathway inhibition with small molecule inhibitors. These altered phenotypes seen in p53<sup>R172H</sup> and p53<sup>R245W</sup> are likely a reflection of their altered transcriptomic profiles.</p>
<p>Although genetically-engineered mouse models are foundational in understanding how lung adenocarcinomas develop, progress, and metastasize, little is known about the transcriptional heterogeneity of genetically-engineered mice. In Chapter 3, we profiled the single-cell transcriptomes of BRAF<sup>V600E</sup> lung tumors. We identified distinct clusters of tumor cell types and determined the transcriptional response to MAPK pathway-targeted therapy.</p>
<p>By quantifying the tumor-suppressive landscape and the heterogeneous transcriptional landscape of BRAF<sup>V600E</sup> cells, our work has contributed valuable knowledge to lung adenocarcinoma biology.</p>
<p><strong>NSCLC</strong> Lung cancer is a global hazard to human health. The historical consumption of cigarette smoke has caused lung cancer to be the most common and deadly form of cancer. Fortunately, smoking rates have decreased in the past 30 years, with lung cancer rates also declining. However, even if cigarette consumption is eradicated, lung cancer would still significantly affect human health. Indeed, approximately 25% of all lung cancer cases arise in never smokers <span class="citation" data-cites="11g54rA8N">(Warren and Cummings, 2013)</span>. Secondary causes of lung cancer include Radon exposure, high-heat cooking, family history, and air pollution <span class="citation" data-cites="ku3fwB6c">(Malhotra et al., 2016)</span>. An individual’s risk of lung cancer also increases with age due to decreases in the efficiency of tumor-suppressive mechanisms such as tumor immunity and apoptosis<span class="citation" data-cites="1F86sLj5s">(de Groot and Munden, 2012)</span>.</p>
<p>Lung cancer can be broadly classified into two major histological categories based on how the malignant cells look under a microscope: Non-small Cell Lung Cancer (NSCLC) and Small Cell Lung Cancer (SCLC). Within NSCLC, tumors are further characterized based on histological structures, anatomical location, cell-of-origin, and genetic drivers<span class="citation" data-cites="19duX0kT0">(Herbst et al., 2008)</span>. For example, lung adenocarcinoma is the most common subtype and is characterized by glandular and papillary structures, while squamous cell lung carcinoma is characterized by keratin-pearls<span class="citation" data-cites="Co9gQ0V6">(Nikitin et al., 2004)</span>. Moreover, adenocarcinomas arise from alveolar-type-2 (AT2) cells in the distal lung and alveoli and are typified by genomic alterations throughout the MAPK pathway, such as EGFR, KRAS, BRAF, PI3K <span class="citation" data-cites="oVtv8JOR cD4M6ONJ">(Ferone et al., 2020; Jackson et al., 2001)</span>. On the other hand, squamous cells carcinomas likely arise from basal cells and typically harbor SOX2 gene amplification <span class="citation" data-cites="cWKMNUBI 7H88gN81">(Mollaoglu et al., 2018; Mukhopadhyay and Oliver, 2014)</span>.</p>
<p>The 5-year survival rate for patients with lung cancer was 25% in 2020 but varied depending on the stage of disease at the time of diagnosis <span class="citation" data-cites="gY2NonCA">(Siegel et al., 2021)</span>. Like other solid cancers, lung cancers are most deadly after primary tumors have metastasized to other organ systems like the liver, kidney, bone, or nervous system <span class="citation" data-cites="SY8r3LQy">(Matsunaga and Shida, 1971)</span>. Lung cancer patients with no local or distant metastasis typically have a 5-year survival percentage of approximately 40% compared to approximately 5% with distant metastasis<span class="citation" data-cites="tR3WrnP9 iZbc8sIr">(Duggan et al., 2016; Woodard et al.)</span>. Ultimately, the cause of death of many lung cancer patients includes primary lung tumor burden, infection, pulmonary embolism or hemorrhage, or complications of metastatic dissemination<span class="citation" data-cites="wDnAOh0T">(Nichols et al., 2012)</span>. Although patient survival has improved due to improved targeted and immune therapies, the still poor prognosis reflects a need to understand better the molecular mechanisms underlying lung adenocarcinoma initiation, progression, maintenance, and response to targeted- or immune therapy.</p>
<p>The current standard of care for Lung adenocarcinoma patients is often a combination of several conventional chemotherapeutics, including Cisplatin, Carboplatin, Paclitaxel, or Pemetrexed<span class="citation" data-cites="hRW7rjfZ">(Ghimessy et al., 2020)</span>. Fortunately, patients who have been identified to have EGFR, ALK, ROS, TRK, or BRAF genetic alternations are often offered pathway-targeted therapies<span class="citation" data-cites="BpRBkUVl 6j21xB75">(Arteaga and Engelman, 2014; Planchard et al., 2016)</span>. As of late, many patients receive immunotherapy first, followed by pathway-targeted therapy after progression on immunotherapy. Furthermore, in the last seven years, exciting therapeutic options for <em>KRAS</em> mutant lung cancer have been developed and tested, such as a combination of MEK-inhibition and autophagy-inhibition and direct inhibitors of <em>KRAS</em> G12c and <em>KRAS</em> G12D <span class="citation" data-cites="c63Ld4El 1CvrKB1k3 bZo8j50s">(Kinsey et al., 2019; Reita et al., 2022; Zheng et al., 2022)</span></p>
<p><strong>MAPK signaling</strong> standard activation from the top of the MAPK pathway begins with extracellular ligands (EGF, NGF) binding to Receptor-Tyrosine Kinases (RTKs), leading to loading of RAS with GTP, which activates RAS. Many RAS mutations lead to either more efficient loading of GTP or less efficient GTP hydrolysis, which both serve to activate RAS. GTP-loaded RAS recruits and activates RAF kinases (ARAF, BRAF, CRAF) which then phosphorylates and activates MEK1/2 kinases, which then phosphorylates and activates ERK1/2. Ultimately, this leads to the activation of a suite of cellular processes required for proliferation, differentiation, migration, and cell survival. Therefore, activating mutation in this pathway, from RTKs to kinases, keeps the pathway in a constitutively active state. For example, the most common mutation in <em>BRAF</em> is V600E. This valine to glutamic acid substitution is located in the kinase domain and mimics phosphorylation, allowing BRAF(V600E) to become independent of upstream regulations like RAS. This active state drives many of the cancer hallmarks required for tumor formation. It is important to note that oncogenic activation of the MAPK pathway differs from normal ligand-mediated pathway activation. For example, mitogens, such as EGF only transiently activate the pathway even with constant ligand stimulation. Various biological processes integrate this immediate and intense spike in pathway activation in the cell, resulting in specific target genes’ transcription. In contrast, oncogenic signaling thru mutational activation of BRAF(V600E) results in constant kinase signaling that is integrated differently than normal pathway stimulation. Thus, an oncogene-specific transcriptional program is activated.</p>
<p>The importance of this pathway is reflected in the strong anti-tumor responses that are seen when small molecule inhibitors are used to target various molecules involved in the MAPK pathway. For example, Dabrafenib and Trametinib, inhibitors of BRAF and MEK kinases, respectfully, are used in patients that harbor BRAF(V600E) melanomas and lung cancers<span class="citation" data-cites="4S1jzORt CgOGrXoS">(Planchard et al., 2021; Robert et al., 2014)</span>. Although there is strong preclinical and clinical evidence that BRAF+MEK blockade can lead to tumor regression, not all patients respond to treatment<span class="citation" data-cites="fXe5uEAl">(Tabbò et al., 2021)</span>. Therefore, multiple pathways need to be targeted to see a complete response, such as PI3Kor autophagy <span class="citation" data-cites="WyV78aHe jKCnCnG4 c63Ld4El">(Deuker et al., 2014; Kinsey et al., 2019; Truong et al., 2020)</span>.</p>
<p>Great effort has elucidated the proto-oncogenes that drive cancer formation when mutated, such as <em>MYC</em>, <em>RAS</em>, <em>PI3K</em>, and others. Nevertheless, we still do not fully understand the mechanistic details that determine how genes cooperate to drive cancer.</p>
<p><strong>Genetically engineered mouse models</strong> Genetically-Engineered mouse models (GEMMs) of human cancer have allowed for the reproducible analysis of cancer biology driven by specific mutations frequently found in cancer patients. These models are helpful in understanding the molecular mechanisms that drive tumor formation and testing tumor responses to targeted or immune therapies. Early mouse models were not engineered but instead selected inbred mouse strains that were natural hosts of tumor transplantation<span class="citation" data-cites="4BJVCLfM">(Strong, 1978)</span>. However, these models suffer from long latencies, incomplete penetrance, and are often not genetically similar to human cancers. Therefore, carcinogen-induced models such as UV, Urethane, radiation, cigarette consumption, or MNU treatment models were developed to resemble carcinogen-induced human cancers<span class="citation" data-cites="9P0mUQ2t zOzqGxQM">(Kemp et al., 1994; Westcott et al., 2014)</span>. After techniques were developed to genetically manipulate embryonic stem cells came the ability to engineer mice with specific mutations that could drive tumor formation. Since genes involved in cancer are also crucial for normal mouse development, conditional gene manipulation was developed to allow for temporal control.</p>
<p>The most commonly used conditional gene expression system relies on inducible transcription factors, such as the tetracycline-dependent regulatory system, where the treatment of mice with doxycycline toggles gene transcription levels of transgenes. More recently, Cre-lox recombination strategies have been utilized to get more cell-type-specific control. In this system, genes of interest are constructed to contain LoxP-sites that flank a particular genetic element. Upon delivery of Cre-recombinase, the DNA sequence in between LoxP sites is excised. With this approach, one can knock out a gene by engineering LoxP sites around critical exons or induce gene transcription by flanking a strong stop signal such as a polyadenylation sequence upstream of a gene-of-interest sequence such as KRAS(G12D)<span class="citation" data-cites="cD4M6ONJ">(Jackson et al., 2001)</span>.</p>
<p>Genetically engineered mouse models of human cancer are important preclinical models because they resemble the physiological environment of tumor growth in which tumors arise as progeny from a single initiating cell. These tumors can approximate the genetic alterations, transcriptional landscape, histology, and responsive disposition or lack, therefore, seen in human cancers(cite). However, these models often take months to develop tumors, and compound genetic alterations take time to develop. Later in this thesis, we will utilize TUBA-SEQ to both quantify tumor burden and cooperation of BRAF(V600E) with other common alterations in lung cancer.</p>
<p>Due to the robust connection between lung adenocarcinomas and mutational activation of genes involved in the MAPK-pathway, such as <em>EGFR</em>, <em>KRAS</em>, and <em>BRAF</em>, many labs have generated genetically-engineered mouse models (GEMMs) that harbor conditionally-activated cancer-specific mutant alleles of either EGFR (L858R), KRAS (G12D), or BRAF(V600E).</p>
<p>Our lab has previously developed a conditional mouse model of BRAF (V600E) human lung cancers<span class="citation" data-cites="dTvhNsG9 Ntv29NRu 1GkFEmggg">(Dankort et al., 2007; Shai et al., 2015; van Veen et al., 2019)</span>. In the <em>Braf(CAT)</em> model, normal BRAF is expressed from a conditional allele prior to Cre-mediated recombination. LoxP sites flank human <em>BRAF</em> cDNA encoding normal <em>BRAF</em> exons 15-18. Downstream of the LoxP sites is the mutant exon 15 that encodes the murine equivalent of the T1799A mutation that gives rise to the BRAF (V600E) oncoprotein. Downstream of the mutant exon, there is a P2A element and a CAAX-tagged <em>TdTomato</em>. Therefore, after Cre-mediated recombination, the BRAF (V600E) oncoprotein and fluorophore TdTomato are expressed at normal levels<span class="citation" data-cites="1GkFEmggg">(van Veen et al., 2019)</span>.</p>
<p><strong>BRAF (V600E) Tumorigenesis and Cooperating Events for Progression</strong> Mutational activation of the MAPK pathway forms lung tumors in mice. Specifically, expression of BRAF (V600E) in the Surfactant-Protein C expression cells (alveolar type 2 pneumocytes) elicits benign tumor formation that fails to progress to lung adenocarcinoma. It is hypothesized that the cell cycle arrest is dependent on the tumor suppressors P53 and CDKN2A. Loss of either P53 or CDKN2A allows tumors to progress to malignant and deadly lung adenocarcinoma. Also, activating WNT signaling and PI3K signaling also allows BRAF (V600E) adenomas to progress to adenocarcinoma <span class="citation" data-cites="1GkFEmggg Pje1Soz4 Djv3zv5n">(Green et al., 2015; Juan et al., 2014; van Veen et al., 2019)</span>. These observations are in concordance with the "multiple-hit" hypothesis that states that cancers do not arise from single mutagenic events but are the consequence of sequential assaults on the genome that activate more than one oncogenic pathway. These multiple genetic hits allow cells to adopt traits to form cancer, which Bob Weinberg and Douglas Hanahan coined as the "Hallmarks of Cancer" <span class="citation" data-cites="4H4Kjv9w kgaUYMuT">(Hanahan, 2022; Hanahan and Weinberg, 2011)</span>. The traits include suppression of apoptosis, active proliferation, altered metabolism, changes in cell identity, and forming blood vessels to supply tumor cells with adequate resources for growth<span class="citation" data-cites="kgaUYMuT">(Hanahan, 2022)</span>. BRAF(V600E) expression alone is sufficient to activate many of the hallmarks of cancer, but additional pathways need to be activated or suppressed to develop into malignant tumors. Great effort has elucidated the proto-oncogenes that drive cancer formation when mutated, such as <em>MYC</em>, <em>RAS</em>, <em>PI3K</em>, and others. Yet we still do not fully understand the mechanistic details that determine how genes cooperate to override tumor suppressive mechanisms and drive cancer progression.</p>
<p><strong>Talk here about the prevalence of mutations in the egfr-ras-RAF-mek-erk pathway</strong></p>
<p><strong>Identification of this pathway as central to luad tumorigenesis</strong></p>
<p><strong>Development of gemms to study BRAF<sup>V600E</sup></strong></p>
<p><strong>Proliferation arrest by this model</strong></p>
<p><strong>How does p53 loss lead to sustained wnt-signalling? Do dominant-negative recapitulate?</strong></p>
<p><strong>P53- mediated tumor suppression</strong>
Cancers pose an evolutionary challenge for organisms regardless of their life span. Therefore, suppression of the outgrowth of neoplastic cells is built into many organisms. One such mechanism is the engagement of the p53 pathway. At the time of writing this dissertation, there are approximately 105,000 manuscripts on PUBMED that mention P53. P53 is well studied because of its prominent role in tumor suppression and because it is the most frequently altered gene in human cancer.</p>
<p>The p53 protein was initially discovered as a protein that bound large T antigene of the SV40 virus in infected cells <span class="citation" data-cites="1FKCm8fuC">(Linzer and Levine, 1979)</span>.P53 is a transcriptional factor with a DNA-binding domain that has a DNA-binding domain recognizes explicitly two decameric half-sites. P53 also contains two N-terminal transactivation domains and a C-terminal oligomerization domain critical for P53-target gene activation<span class="citation" data-cites="qbfESpXa">(Laptenko and Prives, 2006)</span>. Central to the ability of P53 to induce transcriptional activation is tetramerization. Thus, if one or more P53 protein in the complex is compromised in transactivation or DNA binding, transcriptional activity is compromised. Therefore, if one copy of P53 is mutated, the organism’s ability to suppress tumor formation is compromised.</p>
<p>The importance of p53 is best demonstrated in people and in mice that have just one mutated copy of P53. Humans with the familial- inherited Li-Fraumeni syndrome, in which they have one or more mutations in p53, are almost certain to develop cancer early on and throughout their life<span class="citation" data-cites="TYj65wKz">(Li et al., 1988)</span>. Similarly, mice lacking two function P53 genes are prone to leukemias and lymphomas and have a shortened lifespan due to cancer incidence<span class="citation" data-cites="kxXMFjnc">(Donehower et al., 1992)</span>. On the other hand, organisms such as the African and Asian elephants, which have an abnormally low level of cancer incidence, have been shown to have higher multiple copies of <em>TP53</em> in their genomes<span class="citation" data-cites="JyurLVYi">(Abegglen et al., 2015)</span>.</p>
<p>The most well-known cellular functions of P53 include its ability to induce cell-cycle arrest in response to DNA damage<span class="citation" data-cites="IILnKdOc">(el-Deiry et al., 1993)</span>. Many stress signals, including oncogene activation, have been shown to stimulate a reversible or irreversible cell cycle arrest<span class="citation" data-cites="N2yNjRyX 1DxvFFUzE">(McNeal et al., 2021; Zhu et al., 1998)</span>. However, the strength of P53 induced cell cycle arrest or apoptosis is likely cell type- and cellular stress-specific and is an active area of p53 research.</p>
<p>Once a cell encounters a stressor such as DNA damage or oncogene activation, P53 is stabilized and accumulates in the nucleus to activate a suite of target genes that can lead to several cellular phenotypes, such as cell-cycle arrest or apoptosis.</p>
<p>P53 was initially thought to be an oncogene as P53 is accumulated in lung cancer, which is not common in normal tissues<span class="citation" data-cites="bLXbRmvx HnQw16l7">(Bartek et al., 1990; Iggo et al., 1990)</span>. Moreover, ectopic expression of a P53 cDNA was found to aid in transforming primary cells induced by RAS. However, early studies erroneously used mutated P53 instead of wild-type, leading to the misclassification of P53 as an oncogene. We now know P53 suppresses cell growth and transformation<span class="citation" data-cites="SWNWMYmm">(Levine and Oren, 2009)</span>.</p>
<p>Oftentimes, P53 is mutated in the DNA-binding domain at sites commonly referred to as "hotspots" due to their extraordinary frequency. These hotspot mutations poison the ability of p53 to bind DNA, therefore, blocking its function. Interestingly, since one mutated P53 can disrupt the entire P53 tetramer complex, a single mutation can exert dominant-negative effects by inhibiting the normal tumor-suppressive functions of P53. Although one mutant P53 allele is enough to compromise transcriptional activity, there is still selective pressure to lose the other wild-type copy. Loss of hemizygosity implies there is still residual tumor suppression in the presence of a wild-type P53 allele<span class="citation" data-cites="7Yb2p7xx">(Muzumdar et al., 2016)</span>.</p>
<p>Early on in P53 studies, there were reports of specific P53 mutations having a "gain-of-function" (GOF) effect. For example, Li-Fraumeni patients with specific missense mutations in P53 would develop tumors earlier than Li-Fraumeni patients with loss-of-function (LOF) mutations <span class="citation" data-cites="GiCMgc6K">(Bougeard et al., 2008)</span>. Furthermore, experimental cell biology studies would express mutant-p53 in P53<sup>null</sup> cells and demonstrate enhanced tumorigenic potential <span class="citation" data-cites="13QSbJkMk">(Levine, 2020)</span>. Further evidence for GOF mutant-P53 in mice indicated that missense mutant P53 induced different cancer types and enhanced metastasis than LOF P53 <span class="citation" data-cites="MLY9vR0E">(Olive et al., 2004)</span>. Furthermore, many groups have shown that mutant-p53 can alter signal transduction affecting chemoresistance and metabolism.</p>
<p>The cellular mechanisms that GOF P53 is involved in are also well-characterized but context-dependent. For example, in Pancreatic cancers driven by KRAS(G12D), Mutant P3 interacts with CREB to induce FOXA1 transcription, which enhances Beta-Catenin signaling to augment liver metastasis <span class="citation" data-cites="o8jKDrsB">(Kim et al., 2021)</span>.</p>
<h2 id="chapter-two-quantitative-tumor-burden-and-small-library-screen-using-tuba-seq">Chapter Two: Quantitative tumor burden and small library screen using TUBA-SEQ</h2>
<p>{.page_break_before}</p>
<p>##Abstract
Large-scale sequencing of cancers has revealed that lung adenocarcinomas have a high mutation rate. Although it is clear that the evolution of lung adenocarcinoma is centered around mutational activation of the EGFR-RAS-RAF-MEK-ERK pathway, little is known about secondary mutations that drive disease progression. Since lung cancers are riddled with mutations, it is challenging to glean which mutations support cancer growth and which are merely bystanders. Therefore, pre-clinical mouse models that are syngenetic are essential to determining the oncogenicity of a particular gene. Here, we use Tuba-Sequencing combined with CRISPR/Cas9 gene editing to simultaneously determine the contribution of the loss of one of eleven tumor suppressor genes in BRAF<sup>V600E</sup> mouse lung tumorigenesis. We show that loss of either <em>Apc</em> or <em>Setd2</em> leads to enhanced tumor growth. We validated SETD2 loss as a cooperating mutation with single-guide RNAs. Functionality, SETD2 is a histone lysine methyltransferase that marks active gene bodies by depositing me<sup>3</sup> marks on H3K36. In BRAF<sup>V600E</sup> lung tumors, <em>Setd2</em> loss decreases H3K36me<sup>3</sup> and H3K36me<sup>3</sup> was correlated with larger tumors. <em>SETD2</em> RNA is expressed less in human lung cancers than in normal lungs and can indicate worse overall survival. Moreover, <em>SETD2</em> loss is common in human <em>BRAF<sup>V600E</sup></em> lung adenocarcinomas. Together, our results highlight the potential importance of <em>SETD2</em> in the progression of BRAF<sup>V600E</sup> lung adenocarcinoma.</p>
<h1 id="introduction">Introduction</h1>
<p>Next-generation sequencing has allowed for the unbiased identification of mutated genes in human cancer <span class="citation" data-cites="8Gf0ScB2 PAjnX7uA 12CuLfxxd">(2014; Hoadley et al., 2014; Kandoth et al., 2013)</span>.
The wealth of knowledge gained from such large-scale sequencing projects has revealed actionable mutations that can be targeted with small molecule inhibitors, such as dabrafenib and trametinib.
However, identifying possible targets is not enough, as many cancers with similarly mutated oncogenes, such as BRAF<sup>V600E</sup>, respond differentially to BRAF<sup>V600E</sup>-targeted therapy.
For example, melanoma patients with BRAF<sup>V600E</sup> mutations respond well to BRAF<sup>V600E</sup>-targeted therapy, whereas colorectal cancer patients with BRAF<sup>V600E</sup> mutations do not respond<span class="citation" data-cites="wMtu748b">(Corcoran et al., 2012)</span>.
The disconnect between putative driver oncogenes identified in human cancers and response to targeted therapy is a problem that can be solved with rigorous pre-clinical experimentation, such as in GEMMs.</p>
<p>Genomic sequencing of human tumors has also been a crucial insight into how cancers form and what signaling nodes might be necessary for progressive stages of cancer evolution.
For example, pancreatic cancers have a stereotypical histological progression brought about by known mutational activation.
Specifically, mutations in <em>KRAS</em> initiate tumorigenesis, and loss of <em>TP53</em> function and <em>SMAD3</em> are associated with cancer progression<span class="citation" data-cites="WVwMP2mo">(Orth et al., 2019)</span>.
However, this is more difficult to do in lung adenocarcinoma because of the high mutational burden often due to the consumption of cigarettes<span class="citation" data-cites="e3rkxhDE 3xXZMpos">(Alexandrov et al., 2016; Yoshida et al., 2020)</span>.</p>
<p>Indeed, in highly mutated cancers such as melanoma and lung cancer, recurrent mutations are not necessarily predictive of biological importance.
To better understand the importance of candidate tumor suppressors or oncogenes, researchers exploit tumor cell lines and GEMMs to identify bona-fide genetic drivers of cancer.
However, these systems are suboptimal, as they are either not physiologically relevant (tissue culture) or are not scalable to determine the effect of many mutations (GEMMs).
Moreover, techniques to measure the cooperation of genes to drive lung cancer are qualitative and are not sufficiently quantitative to allow for large-scale studies.
Typically, lung adenocarcinoma is quantified by lung weight or histological analyses such as tumor burden (Percentage of the lung that is tumor), tumor size, and immunohistochemistry (IHC) of tumor markers.
Although essential and informative, these analyses are not high-throughput and rely on many mice to achieve statistical confidence. Furthermore, there is intrinsic bias when calculating tumor size and tumor burden, as tissue-sectioning results in tumor sections that do not accurately portray actual tumor size. Therefore, more advanced techniques are required for large-scale analysis of mouse tumors that are both quantitative and high throughput.</p>
<p>Recently, Monte Winslow and his lab at Stanford University have developed <strong>Tu</strong>mor- <strong>Ba</strong>rcode <strong>Seq</strong>uencing (Tuba-Seq), a rapid, multiplexed, quantitative assay for analyzing the contribution of tumor-suppressor loss <em>in vivo</em>.
This technique delivers a uniquely barcoded lentiviral encoding Cre recombinase to activate Cre-mediated oncogenes and a guide RNA to inactivate a single tumor suppressor gene.
Therefore, each tumor will be driven by the same oncogene but with different tumor suppressors inactivated.
After some time (6-24 weeks), tissue is harvested for whole-lung DNA extraction, and targeted DNA-sequencing is performed to quantify relative tumor size to spike-in controls<span class="citation" data-cites="sufLyk4G">(Rogers et al., 2017)</span>.
Therefore, the contribution of a single tumor suppressor can be quantified in a single animal.</p>
<p>BRAF<sup>V600E</sup> has previously been shown to be a potent driver of lung tumorigenesis<span class="citation" data-cites="dTvhNsG9">(Dankort et al., 2007)</span>.
However, tumors fail to progress past adenoma-like tumors.
Given the theory that BRAF-<sup>V600E</sup> induces growth arrest after an initial burst of neoplastic growth, it would be of interest to know if all tumors grow to a predetermined and absolute size or if there is a spectrum of tumor sizes that are determined by extrinsic factors such as proximity to blood vessels or specific stromal cells.
Therefore, we used Tuba-Seq combined with CRISPR/CAS9 genome editing to explore the contribution of 11 tumor suppressor genes and five controls in our BRAF <sup>V600E</sup>- driven lung tumor models.</p>
<h1 id="results">Results</h1>
<p>Adenoviral vectors and lentiviral vectors introduced in the lungs of GEMMs offer temporal control and can be titrated to achieve an optimal level of tumor numerosity.
However, only lentiviral vectors can be used to barcode tumors because they integrate into the genome while adenoviruses do not<span class="citation" data-cites="bhNe3oCb sufLyk4G">(DuPage et al., 2009; Rogers et al., 2017)</span>.
With CRISPR/CAS9 gene editing, lentiviruses have been used to analyze many genetic interactions of oncogenic <em>Kras</em> driven lung adenocarcinoma<span class="citation" data-cites="2gOIsLdX sufLyk4G">(Rogers et al., 2017; Winters et al., 2017)</span>.
To enable simultaneous activation of BRAF<sup>V600E</sup> and loss of tumor suppressor genes, we bred <em>Braf<sup>CAT</sup></em>; <em>H11b<sup>LSL-CAS9/+</sup></em> mice.
To determine the size of BRAF <sup>V600E</sup> tumors and the contribution of 11 tumor suppressor genes (<em>Trp53, Rb1, Cdkn2a, Apc, Rbm10, Stk11, Setd2, Arid1a, Atm, Keap1, Smad4)</em> on BRAF <sup>V600E</sup> lung tumorigenesis, we obtained tittered barcoded-lentiviral plasmids that encode Cre-recombinase as well as a sgRNA that inactivates one of eleven tumor suppressors common in lung adenocarcinoma(Figure 2.1).
All genes tested are putative tumor suppressor genes commonly lost in human lung cancer patients. Each lentiviral solution contained ~1000 individually barcoded lentiviruses.
We infected both <em>LSL-Cas9</em> and <em>CAS9</em> deficient mice in order to determine the contribution of genetic manipulation on tumor size (Figure 2.1).
Mice meant to be analyzed after eight weeks received a viral titer of 1x10^5, while mice meant to be analyzed after 16 weeks received a viral titer of 1x10^4.
Gross, histologic images of experimental lungs show an increase in tumor burden with both time (Figure xxx) and CAS9 status (Figure 2.1).
Therefore, based on macroscopic inspection, at least one tumor suppressor appeared to cooperate with BRAF <sup>V600E</sup> to increase tumor burden.
After euthanasia, lungs were collected for genomic DNA extraction and barcode amplification, followed by the preparation of sequencing libraries.
Before genomic extraction and library preparation, a known quantity of known-barcoded cells was added to benchmark sequencing reads to a known cell number.</p>
<p>We determined the mean size of a BRAF<sup>V600E</sup> driven tumor to be 21,248 thousand cells (Figure 2.2).
Interestingly, there is a widespread distribution of tumor sizes, with the most giant BRAF<sup>V600E</sup> tumor to be ~300,000 cells, along with many small tumors (Figure).</p>
<p>In order to determine the fitness landscape of any one particular tumor suppressor, we calculated the relative number of tumor cells in a given tumor using the Tuba-Seq genomics pipeline. In CAS9 deficient mice, there was no statiscically significant cooperation with any single-guide RNAs (Figure 2.3A).
After eight weeks post tumor initiation, APC loss provided a growth advantage(Figure 2.3 B).
However, 16 weeks post-initiation revealed that both SETD2 loss and APC loss cooperated with BRAF <sup>V600E</sup> to form larger tumors (Figure 2.3 C).
APC loss was a suspected hit as we have previously shown that activation of Wnt-beta-catenin signaling promoted BRAF<sup>V600E</sup> tumorigenesis.
As APC loss is predicted to increase Beta-catenin signaling activity, this result was unsurprising to us.
However, loss of SETD2 was a novel finding in the context of BRAF <sup>V600E</sup> lung tumorigenesis.</p>
<p>Given our findings of SETD2 loss increased tumor burden concurrent with BRAF<sup>V600E</sup>, we next validated SETD2 in a new cohort of mice with single-guide RNAs targeting SETD2 (Figure 2.4).
Indeed, loss of SETD2 increased tumor size based on gross histological analysis(Figure 2.4A). Addiionally, tumor size increased from in reponse to CRISPR/Cas9-mediated gene editing of <em>Setd2</em>(Figure 2.4B).
Since SETD2 antibodies are unreliable in both western blots and immunohistochemistry, we used H3K36me3 as a surrogate marker for SETD2 activity by immunohistochemistry(Figure 2.5A).
Whereas 90% of nuclei in BRAF <sup>V600E</sup> only tumors were H3K36me3, more than 60% of tumors of mice infected with SETD2 CRISPR virus were negative for H3K36me3(Figure 2.5B).
Furthermore, when classified by H3K36me3-positive or -negative, there is even more substantial evidence that SETD2 loss leads to larger tumor formation(Figure 2.5C). Specifically, tumors that have lost H3H36me<sup>3</sup>, were 5-times larger than tumors that have retained H3K36me<sup>3</sup>. Therefore, we conclude that <em>Setd2</em> loss drives growth of BRAF<sup>V600E</sup>-driven lung tumors.</p>
<p>Clearly, there is a strong selective advantage given to tumors that have lost <em>Setd2</em> function. We next wondered if human lung cancer patients have altered <em>SETD2</em> levels. Indeed, Low <em>SETD2</em> levels correlate with poor overall survival and <em>SETD2</em> is expressed less in lung tumors compared to normal lung tissue(Figure 2.6A,B). To further demonstrate the relationship between BRAF<sup>V600E</sup> and <em>SETD2</em> loss, we mined patient data avaiable from Project Genie<span class="citation" data-cites="1CW3EbE5O">(2017)</span>. Interestingly, we found an association between <em>SETD2</em> genomic alterations and <em><sup>BRAF</sup>V600^</em> mutational activation(Figure 2.6C). Specifically, of the 220 <em>BRAF<sup>V600</sup></em> mutations (include V600E and V600K), 40% had concurrent alterations in <em>SETD2</em>. These observations indicate <em>SETD2</em> loss is a common feature of <em>BRAF<sup>V600</sup></em>-driven human lung adenocarcinomas and predicts poor overall survival in non-small cell lung cancer.</p>
<h1 id="discussion">Discussion</h1>
<p>The mutational landscape of lung adenocarcinoma is complex due to the lifelong bombardment of DNA by carcinogens ingested into the distal lung epithelium[23770567].
While understanding which altered genes play causal roles in tumor formation and progression is required for personalized medicine, the noisy mutation profiles in lung adenocarcinoma make it hard to do so.
However, there is a central role for the MAPK pathway during the initiation and progression of lung adenocarcinoma <span class="citation" data-cites="OK8s0ke5">(Cicchini et al., 2017)</span>.
Oncogenic BRAF<sup>V600E</sup> is a significant driver of lung adenocarcinoma.
However, mouse models with activation of BRAF <sup>V600E</sup> only lead to benign tumorigenesis<span class="citation" data-cites="Ntv29NRu dTvhNsG9">(Dankort et al., 2007; Shai et al., 2015)</span>.
Although it is known that activation of WNT and PI3k signaling and loss of <em>Trp53</em> and <em>Cdkn2a</em> genes allow BRAF<sup>V600E</sup> benign lung tumors to progress to malignant disease, it is a time-consuming and laborious process to effects individual tumor suppressors, in GEMMs.
Therefore, we sought to systematically test a variety of known tumor suppressors in lung adenocarcinoma.
Furthermore, until now, we did not know the absolute number of cells in a BRAF<sup>V600E</sup> adenoma, only its area, which is confounded by tissue sectioning and stromal cell contamination.
Prior to Tuba-Seq, it was challenging to accurately determine the size of BRAF<sup>V600E</sup> adenomas.
Based on population doublings and cross-sections of BRAF <sup>V600E</sup> tumors in histological sections, it is estimated that BRAF<sup>V600E</sup>-driven lung adenomas grow about 15-20 populations doublings and reach a size of 1mm<sup>3</sup><span class="citation" data-cites="Djv3zv5n">(Juan et al., 2014)</span>.
With Tuba-Seq, we accurately discerned the absolute size of all BRAF<sup>V600E</sup> lung adenomas, regardless of size.
Interestingly, our results have revealed that BRAF<sup>V600E</sup>-driven tumors widely differ in absolute tumor cell number. Our results support the hypothesis that each tumor-initiating cell has unique barriers to tumor outgrowth.
This heterogeneous response to BRAF<sup>V600E</sup> activation likely reflects intrinsic differences between individual cell states.
Indeed, tumorigenic potential can vary significantly due to different cell types (AT2 vs. Club cell vs. AT1) or be influenced by subtle differences within a cell type, such as proximity to supporting stromal cells such as fibroblast or immune cells.
Since we used a ubiquitous promoter to drive BRAF<sup>V600E</sup> and CAS9 expression, not all sequence likely reads come from adenomas or tumors but from average cell turnover during tissue homeostasis.
Therefore, in our quantification and analysis, we focused on tumors greater than 5,000 cells.</p>
<p>We combined Tuba-Seq with CRISPR/CAS9 gene editing of eleven putative tumor suppressor genes with four inert control sgRNAs. We identified APC and SETD2 loss as a cooperating genetic event that allows BRAF<sup>V600E</sup> tumors to grow larger.
Therefore, APC and SETD2 are bona fide tumor suppressors in lung adenocarcinoma.
APC loss is known to activate the Wnt-signaling pathway and is a well-established tumor suppressor in colon cancer<span class="citation" data-cites="1fsz5jxh">(Powell et al., 1992)</span>.
Furthermore, our lab has previously published a requirement for WNT-signaling for the progression of BRAF<sup>V600E</sup> lung tumors<span class="citation" data-cites="Djv3zv5n">(Juan et al., 2014)</span>.
Specifically, inhibition of WNT signaling by a Porcupine inhibitor prevents tumor onset, while constitutive activation of the pathway drives tumor progression to high grade.
Therefore, it is not surprising that APC loss cooperates with BRAF<sup>V600E</sup> to induce tumor growth.</p>
<p>We have also identified <em>SETD2</em> loss as a potent tumor suppressor(Figure 2).
<em>SETD2</em> is mutated in other solid malignancies on cBioPortal, such as Kidney and bladder cancers<span class="citation" data-cites="Ph0Z7rTW">(Cerami et al., 2012)</span>.
SETD2 is a histone methyltransferase that deposits trimethylation marks on H3K36<span class="citation" data-cites="1AV7mz5Fc">(Bhattacharya and Workman, 2020)</span>.
Chromatin modifiers use H3K36me<sup>3</sup> to mark actively transcribed genes<span class="citation" data-cites="M2kGnbOE">(Edmunds et al., 2007)</span>.
Mining of TCGA mRNA expression and survival analysis further revealed that <em>SETD2</em> mRNA is lower in lung adenocarcinoma and is associated with poorer overall survival(Figure 2).
Interestingly, <em>SETD2</em> alterations are also significantly associated with BRAF<sup>V600E</sup> mutations (Figure 2).
The unique association of <em>BRAF<sup>V600E</sup></em> mutation and <em>SETD2</em> loss may reflect an interesting synergistic effect between MAPK signaling and loss of chromatin regulation.
One hypothesis is that SETD2 loss drives spurious transcriptional activation.<br />
Similarly, recent work by Monte Winslow and David Felder’s labs have identified SETD2 loss as a cooperating event in KRAS-G12D lung tumorigenesis<span class="citation" data-cites="S6qbb6kC">(Walter et al., 2017)</span>.
However, the mechanism by which <em>SETD2</em> loss drives tumor progression in either BRAF<sup>V600E</sup> or KRAS-G12D remains elusive.</p>
<p>Although we have identified novel tumor suppressors in BRAF<sup>V600E</sup> lung tumorigenesis using Tuba-Seq, we have failed to confirm previously identified tumor suppressors <em>CdkN2a</em> and <em>Trp53</em> as cooperating events.
One hypothesis is that the sgRNAs targeting <em>Trp53</em> and <em>Cdkn2a</em> are inefficient. However, previous Tuba-Seq experiments did confirm <em>Trp53</em> editing and even demonstrated tumorigenic cooperation with KRAS-G12D<span class="citation" data-cites="17E8qJFQ0">(Rogers et al., 2018)</span>.
Another hypothesis is that the effect size of <em>trp53</em> loss is so tiny that many mice and tumors are needed to detect robust cooperation.
Indeed, p53 loss is not that to suddenly make tumors immediately proliferate.
Instead, p53 loss merely allows for tumor cells to sample various transcriptional programs that can be co-opted for increased fitness in the face of stress.
Along similar lines, perhaps 16-weeks is too little time to allow for <em>Trp53</em> or <em>Cdkn2a</em> deficient tumors to proliferate.
Likely, it is a combination of small effect size, inefficient gene editing, and insufficient time that prevented us from identifying <em>Trp53</em> as a tumor suppressor in BRAF6V600E^ driven lung tumorigenesis using Tuba-Seq combined with CRISPR/CAS9.</p>
<p>Since GEMMs lack the mutational burden exhibited by human lung adenocarcinomas, they are good models to study gene-gene interactions in tumorigenesis.
Our data provide further insight into the essential biological mechanisms for BRAF<sup>V600E</sup> lung tumorigenesis.
Additional studies of SETD2’s role in lung adenocarcinoma will further reveal how it alters the evolution of lung adenocarcinomas.
Furthermore, given its striking association with <em>BRAF<sup>V600E</sup></em> mutations, it will be essential to know how <em>SETD2</em> alters the response to BRAF<sup>V600E</sup>-targeted therapy.
A complete understanding of the consequences and outcomes of genetic cooperation between BRAF<sup>V600E</sup> and tumor suppressors or oncogenes will help clinicians better understand tumor evolution and paths to drug resistance.</p>
<p>##Methods</p>
<p><strong>Animal Husbandry</strong> All mice were housed in an environmentally controlled room, and all animal care and experimental procedures were approved by (and in accordance with) the Institutional Animal Care and Use Committee Office of the Huntsman Cancer Institute at the University of Utah. Genetically engineered mouse breeding and genotyping was conducted as previously described <span class="citation" data-cites="1GkFEmggg">(van Veen et al., 2019)</span>. The <em>BRAF<sup>CAT</sup></em> and mice were previously described <span class="citation" data-cites="kGt16xDw 1GkFEmggg dTvhNsG9 Ntv29NRu">(Dankort et al., 2007; Jonkers et al., 2001; Shai et al., 2015; van Veen et al., 2019)</span>. <em>H11b<sup>LSL-Cas9</sup></em> mice were previously described<span class="citation" data-cites="1EwtuJJw6">(Chiou et al., 2015)</span> All viruses were administered in a Biosafety Level 2+ room, as is regulated by the Institutional Biosafety Committee Guidelines. Lentivirus (Stanford University) was delivered through intratracheal intubation.</p>
<p><strong>Tuba-Seq</strong> <em>Braf<sup>CAT/+</sup></em> and <em>H11b<sup>LSL-CAS9/+</sup></em> mice have been described<span class="citation" data-cites="1GkFEmggg 1EwtuJJw6">(Chiou et al., 2015; van Veen et al., 2019)</span> and maintained on a mixed background. approximetley equal numbers ofmale and female mice were used in the study. lung tumors were initiated by intratracheal intubation of lentiviral Cre vectors as previously described<span class="citation" data-cites="bhNe3oCb">(DuPage et al., 2009)</span>. Barcoded lentiviral vectors were supplied by Monte Winslow as previously described<span class="citation" data-cites="2gOIsLdX">(Winters et al., 2017)</span>. Briefly, Lentiviral vectors that contain an eight nucleotide sequence that defines the CRISPR-targeted followed by a 15 nucleotide sequence that is unique only to each initiated tumor. Lentivirus was generated in HEK 293T cells in DMEM supplemented with 10% FBS by standard lentiiral generation and titered as previously described<span class="citation" data-cites="2gOIsLdX">(Winters et al., 2017)</span>. Genomic DNA was isolated from bulk lung tissue after weighing. Briefly, benchmark control cell lines were spiked-in to enable calculation of the absolute number of tumor cells from the number of barcode reads. After tissue homogenization and 12-hour proteinase-K digestion, genomic DNA isolation was extracted using standard Phenol-Chloroform and precipitated by ethanol. Suquencing libraries were prepared by PCR amlifying the sgRNA-BC sequencing in eight reactions in parallel with Q5 master mix and purified with AMPure beads and quantified with the Qubit instrument. Computational analysis was performed in house by the Monte Winslow laboratory.</p>
<p><strong>Histology and Quantification of lung tumor burden</strong>
Lungs were perfused with neutral-buffered formalin, removed, and kept in neutral-buffered formalin for 16-24 hours and then stored in 70% ethanol prior to embedding. 5 micrometer sections were cut using the HCI histology core. sections were then stained with hematoxylin and eosin (H&amp;E). Stained slides were scanned using the 3D-histech Panoramic MIDI slide scanner. Tumor size was quantifie using 3D Histech QuantCenter. Immunohistochemistry for H3K26me<sup>3</sup> was performed as previously described at a concentration of 1:1000 with rabbit mAb #4909 from Cell Signaling Technologies. <span class="citation" data-cites="Rky3nm5Q 1GkFEmggg">(Mollaoglu et al., 2017; van Veen et al., 2019)</span>.</p>
<p><strong>Human lung cancer data</strong> cBio Portal and Project Genie were used to calculate co-ocurring mutations with <em>BRAF<sup>V600</sup></em> mutations. KMplotter was used for survival analysis and TNMplot was used to calculate <em>SETD2</em> expression in tumor tissue vesus normal lung tissue<span class="citation" data-cites="68clBYF4 EpOVYdFF">(Bartha and Győrffy, 2021; Lánczky and Győrffy, 2021)</span>.</p>
<div id="fig:04" class="fignos">
<figure>
<img src="images/tuba1.png" class="white" style="width:100.0%" alt="Figure 1: Figure 2.1. Overview of Tuba-Seq experimentation (A) Lenti-sgRNA-Cre plasmid library with collection of targeted tumor supressors. (B) BrafCAT/+ and BrafCAT/+;H11bLSL-Cas9/+ were infected with either 104 (16 week BrafCAT/+; H11bLSL-Cas9/+) or 105 (8 weeks BrafCAT/+; H11bLSL-Cas9/+ and 16 weeks BrafCAT/+; H11bLSL-Cas9/+) (B) Representative gross images of experimental mouse lungs." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Figure 2.1. Overview of Tuba-Seq experimentation</strong> (A) Lenti-sgRNA-Cre plasmid library with collection of targeted tumor supressors. (B) <em>Braf<sup>CAT/+</sup></em> and <em>Braf<sup>CAT/+</sup>;H11b<sup>LSL-Cas9/+</sup></em> were infected with either 10<sup>4</sup> (16 week <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em>) or 10<sup>5</sup> (8 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em> and 16 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em>) (B) Representative gross images of experimental mouse lungs.</figcaption>
</figure>
</div>
<div id="fig:05" class="fignos">
<figure>
<img src="images/tuba2.png" class="white" style="width:100.0%" alt="Figure 2: Figure 2.2. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline. Tumor size corresponds to dot size. Mean tumor number for 16 weeks BrafCAT/+; H11b+/+ is 21,248 cells. Mean tumor number for 8 weeks BrafCAT/+; H11bLSL-Cas9/+ is 9,095 cells. Mean tumor number for 16 week BrafCAT/+; H11bLSL-Cas9/+ is 29,867 cells." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Figure 2.2. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline.</strong> Tumor size corresponds to dot size. Mean tumor number for 16 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>+/+</sup></em> is 21,248 cells. Mean tumor number for 8 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em> is 9,095 cells. Mean tumor number for 16 week <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em> is 29,867 cells.</figcaption>
</figure>
</div>
<div id="fig:06" class="fignos">
<figure>
<img src="images/tuba3.png" class="white" style="width:100.0%" alt="Figure 3: Figure 2.3. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline seperated by sgID. (A) 16 weeks BrafCAT/+; H11b+/+(negative control) has no apparent cooperative genetic events. (B) 8 weeks BrafCAT/+; H11bLSL-Cas9/+ identifed Apc loss as a cooperating genetic event based on increase in tumor cell number (C) 16 week BrafCAT/+; H11bLSL-Cas9/+ identified Apc and Setd2 loss as cooperating events." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Figure 2.3. Jitter plot of calculated tumor cell number based on Tuba-Seq pipeline seperated by sgID.</strong> (A) 16 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>+/+</sup></em>(negative control) has no apparent cooperative genetic events. (B) 8 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em> identifed Apc loss as a cooperating genetic event based on increase in tumor cell number (C) 16 week <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em> identified Apc and Setd2 loss as cooperating events.</figcaption>
</figure>
</div>
<div id="fig:07" class="fignos">
<figure>
<img src="images/tuba4.png" class="white" style="width:100.0%" alt="Figure 4: Figure 2.4. Validation of Setd2 loss as cooperating genetic event in BRAFV600E lung cancer. (A) Representative images of lung tumor histology of 16 weeks BrafCAT/+; H11bLSL-Cas9/+ mice infected with 2.5x104 PFU Lenti-sgSetd2-Cre or Lenti-sgNT1-Cre. (B) Quantification of tumor size based on histology, n= 30 tumors. * = _p-value&lt;.05." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Figure 2.4. Validation of <em>Setd2</em> loss as cooperating genetic event in BRAF<sup>V600E</sup> lung cancer.</strong> (A) Representative images of lung tumor histology of 16 weeks <em>Braf<sup>CAT/+</sup>; H11b<sup>LSL-Cas9/+</sup></em> mice infected with 2.5x10<sup>4</sup> PFU Lenti-sgSetd2-Cre or Lenti-sgNT1-Cre. (B) Quantification of tumor size based on histology, n= 30 tumors. * = _p-value&lt;.05.</figcaption>
</figure>
</div>
<div id="fig:08" class="fignos">
<figure>
<img src="images/tuba5.png" class="white" style="width:100.0%" alt="Figure 5: Figure 2.5. H3K36me3 is lost in larger tumors in Set2d-edited lung tumors.(A) Representative images of H3K36me3 (surrogate for setd2 loss) depicting either “high” or “low” staining indicative of protein expression. (B) Quantification of H3K36me3 positive tumors in either sgSetd2 or sgNT1 targeted mice. = p-value&lt;.05 for negative tumors. (C) Quantification of tumor size paired with H3K36me3 negative tumors. Tumor size and H3K36me3 positivty were simuletaneously measured using QuantCenter and binned in either “low” or “High” and then average tumor size was determined for each group. = p-value&lt;.05." />
<figcaption aria-hidden="true"><span>Figure 5:</span> <strong>Figure 2.5. H3K36me<sup>3</sup> is lost in larger tumors in <em>Set2d</em>-edited lung tumors.</strong>(A) Representative images of H3K36me3 (surrogate for <em>setd2</em> loss) depicting either “high” or “low” staining indicative of protein expression. (B) Quantification of H3K36me<sup>3</sup> positive tumors in either <em>sgSetd2</em> or <em>sgNT1</em> targeted mice. <em>= <em>p-value&lt;.05</em> for negative tumors. (C) Quantification of tumor size paired with H3K36me<sup>3</sup> negative tumors. Tumor size and H3K36me<sup>3</sup> positivty were simuletaneously measured using QuantCenter and binned in either “low” or “High” and then average tumor size was determined for each group. </em>= <em>p-value&lt;.05</em>.</figcaption>
</figure>
</div>
<div id="fig:09" class="fignos">
<figure>
<img src="images/tuba6.png" class="white" style="width:100.0%" alt="Figure 6: Figure 2.6. SETD2 loss is common in human cancers, an indicator of poor prognosis and frequently occurs with BRAFV600E mutations.(A) Survival analysis with “KMplotter”” indicates a trend towards poorer overall survival with low SETD2 mRNA in TGCA Lung adenocarcinoma dataset. (B) SETD2 mRNA is attenuated in Lung adenocarcinoma compared to normal lung tissue (C) SETD2 mutations significantly co-occur with BRAFV600E mutations and are present in 40% of _BRAFV600E) mutant Lung Adenocarcinoma." />
<figcaption aria-hidden="true"><span>Figure 6:</span> <strong>Figure 2.6. <em>SETD2</em> loss is common in human cancers, an indicator of poor prognosis and frequently occurs with <em>BRAF<sup>V600E</sup></em> mutations.</strong>(A) Survival analysis with “KMplotter”” indicates a trend towards poorer overall survival with low <em>SETD2</em> mRNA in TGCA Lung adenocarcinoma dataset. (B) <em>SETD2</em> mRNA is attenuated in Lung adenocarcinoma compared to normal lung tissue (C) <em>SETD2</em> mutations significantly co-occur with BRAF<sup>V600E</sup> mutations and are present in 40% of _BRAF<sup>V600E</sup>) mutant Lung Adenocarcinoma.</figcaption>
</figure>
</div>
<h2 class="page_break_before" id="chapter-three">Chapter Three</h2>
<h2 id="abstract">Abstract</h2>
<p><sup>TP53</sup> missense BRAF<sup>V600E</sup> mutations both frequently occur in lung adenocarcinoma. While genetically engineered mouse models (GEMMs) of lung adenocarcinoma are invaluable for understanding the initiation, progression, and response to therapeutics, little is known about how different hotspot point mutations in p53 affect disease progression and therapeutic responses. To this end, we use GEMMs, tumor-derived cell lines, and human lung cancer cell lines to test how murine homologs of p53 hotspot mutants R172H and R245W compare to p53-deletion. We find that conformation mutation p53<sup>R172H</sup> drives more aggressive lung adenocarcinomas and dedifferentiation. We also observe p53<sup>R172H</sup> and p53<sup>R245W</sup> accumulation in only high-grade tumors that have activated ARF expression. In vitro and in vivo studies demonstrate enhanced resistance to BRAF+MEK blockade with p53<sup>R172H</sup> and p53<sup>R245W</sup>, a standard treatment regimen for BRAF<sup>V600E</sup> mutant patients. Mechanistically, mutant-p53 alters the transcriptome of cancer cells and enhances entry into the cell cycle by a yet to be determine mechanism.</p>
<h2 id="introduction-1">Introduction</h2>
<p>Lung adenocarcinoma (LUAD) has remained the deadliest form of cancer since 1987 (Siegel, Miller et al. 2021). It is characterized by alterations in the MAP kinase (MAPK) pathway, including activating mutations in the oncoproteins EGFR, KRAS (G12C, G12D, G12V) or BRAF<sup>V600E</sup> <span class="citation" data-cites="8Gf0ScB2 kQzAqqtP cD4M6ONJ dTvhNsG9">(2014; Dankort et al., 2007; Jackson et al., 2001; Politi et al., 2006)</span>. The recognized predominant cell of origin for LUAD, lung alveolar type-2 pneumocyte cells (AT2), are exquisitely responsive to the transformative properties of constitutive activation of the MAPK pathway <span class="citation" data-cites="dAZ32h3a UfKKrJ6l">(Sainz de Aja and Kim, 2020; Sutherland et al., 2014)</span>. Indeed, genetically engineered mouse models (GEMMs) of mutationally activated EGFR, KRAS, or BRAF form lung tumors that strongly resemble key characteristics of human LUAD. However, tumors formed in all three models (KRAS:G12D, EGFR: Deletion 19 or L858R, or BRAF<sup>V600E</sup>) develop only histologically low- to mid-grade cancer and are growth-arrested <span class="citation" data-cites="cD4M6ONJ kQzAqqtP dTvhNsG9">(Dankort et al., 2007; Jackson et al., 2001; Politi et al., 2006)</span>. Prior research has determined that oncogene-induced tumor growth arrest occurs through activation of several tumor suppression mechanisms, most notably, p53-mediated tumor suppression <span class="citation" data-cites="Ntv29NRu">(Shai et al., 2015)</span>.</p>
<p>Alterations in the <em>TP53</em> tumor suppressor gene are common in all cancer types, including LUAD <span class="citation" data-cites="ju5z9qDz">(Bouaoun et al., 2016)</span>. P53 encodes a transcription factor that responds to various cell stresses, such as DNA damage and oncogene activation. In response to stress, p53 is stabilized and induces anti-proliferative and genome-mending effects through trans-activation of a diverse set of P53-target genes <span class="citation" data-cites="KlPsMirp 14fESvHZW">(Boutelle and Attardi, 2021; Vogelstein et al., 2000)</span>. p53 mutations in cancer frequently occur in the DNA binding domain, where a single amino acid change results in dysfunctional DNA-binding and fails transcriptional activation <span class="citation" data-cites="1Aj9gKIb1">(Olivier et al., 2010)</span>. Therefore, mutated p53 defunct in p53 transcriptional regulation results in failed tumor suppression and cancer outgrowth. Furthermore, defective p53 signaling will permit future genomic insults to remain unchecked, thus fostering an environment primed for malignant potential. Since mutant-p53 lacks transcriptional activity through its inability to bind DNA, it cannot induce expression of its negative regulators, ARF, MDM2, and MDMX <span class="citation" data-cites="E3yjcNvW Mb1Q4pdk">(Perry et al., 1993; Wu et al., 1993)</span>. Lack of negative feedback results in mutant-p53 stabilization. Accumulation of mutant p53 has been reported to elicit gain-of-function p53 activity. Thus, any gain-of-function activity in mutant-p53 cannot be studied with p53-genetic deletion. Many research groups have characterized gain-of-function phenotypes of mutant-p53 in cancer. For instance, mice expressing mutant-p53 have altered disease phenotype compared to p53-null models <span class="citation" data-cites="MLY9vR0E V4ONQpBd">(Lang et al., 2004; Olive et al., 2004)</span>. Specifically in lung cancer model systems, mutant-p53 has modulated SMAD3, NF<span class="math inline">\(\kappa\)</span>B, p63, TGF<span class="math inline">\(\beta\)</span>, and others to drive various hallmarks of cancer invasion, chemoresistance, metabolism, and propagation of a stem cell fate <span class="citation" data-cites="B0ClcVtn Vbw51s50 1HKPvGqty">(Adorno et al., 2009; Gaiddon et al., 2001; Klemke et al., 2021)</span>. Although the mechanisms that lead to these altered processes are controversial, it is thought that the main effect is through mutant-p53 binding and sequestering known p53-associated proteins. This often leads to the activation of transcription factors such as NF-Y <span class="citation" data-cites="o8jKDrsB">(Kim et al., 2021)</span>.</p>
<p>In BRAF<sup>V600E</sup> GEMMs, when p53 function is abolished by <em>Trp53</em> gene deletion or by <em>Cdkn2a</em> deletion, lung tumors can escape growth arrest and undergo malignant transformation <span class="citation" data-cites="dTvhNsG9 Ntv29NRu">(Dankort et al., 2007; Shai et al., 2015)</span>. Furthermore, p53 pathway dysregulation allows a cell to sample diverse phenotypic landscapes that allow for enhanced cell fitness and tumor outgrowth <span class="citation" data-cites="pjuOgKHJ">(Marjanovic et al., 2020)</span>. However, BRAF<sup>V600E</sup> lung tumorigenesis and progression to malignant lung adenocarcinoma have only been studied in the context of p53 genetic deletion, not p53 mutation, which is more frequently seen in human lung cancer. Specifically, p53-R172H and p53-R245W are murine homologs of typical hotspots in human cancer R175H and R245W, respectively.</p>
<p>Here, we characterize the differential impact of p53(null), as well as p53 hotspot mutations p53<sup>R712H</sup> (R175H in humans) and p53<sup>R245W</sup> (R248W in humans) on BRAF<sup>V600E</sup> lung tumorigenesis. We observed significantly differential effects of p53-R172H on BRAF<sup>V600E</sup>-driven lung tumorigenesis that were not observed when p53 is deleted, such as enhanced tumor size and decreased survival. Our findings highlight functional differences in mutant-p53 biology, as well as the importance and the utility of clinically relevant mutant-p53 mouse models.</p>
<h2 id="results-1">Results</h2>
<p><strong>P53-mutant Alleles Differentially Alter BRAF (V600E) Lung Tumorigenesis</strong></p>
<p>Expression of BRAF<sup>V600E</sup> in the distal epithelium of the mouse lung elicits clonal tumorigenic outgrowths of alveolar type 2 (AT2) pneumocytes <span class="citation" data-cites="dTvhNsG9">(Dankort et al., 2007)</span>. Prior work has shown that <em>Trp53<sup>fl/fl</sup></em> cooperates with BRAF(V600E) to form malignant lung tumors when Cre recombinase is activated <span class="citation" data-cites="dTvhNsG9 Ntv29NRu">(Dankort et al., 2007; Shai et al., 2015)</span>. Therefore, we compared the tumor burden of <em>Braf<sup>Cat/+</sup></em> mice with either p53 loss (<em>Trp53<sup>fl/fl</sup></em>, p53<sup>R172H</sup> (<em>Trp53<sup>R172H/R172H</sup></em>), or p53<sup>R245W</sup> (<em>Trp53<sup>R245W/R245W</sup></em>) after infection with 106 pfu of adenovirus expressing surfactant protein C specific Cre (ad5-Spc-Cre) (Figure 3.1). As expected, mice developed lung cancer, consistent with previous findings (Figure 3.2B-C). However, there was a striking, highly reproducible increase in the tumor size of tumors harboring <em>Trp53<sup>R172H/R172H</sup></em> and to a lesser extent <em>Trp53<sup>R245W/R245W</sup></em>, compared to <em>Trp53<sup>fl/fl</sup></em> (Figure 3.2A and B). These data led us to hypothesize that p53<sup>R172H</sup> and p53<sup>R245W</sup> expression alters tumor progression uniquely from p53 deletion, and these differences may be potentially evident of gain-of-function effects imparted by mutant-53. We next looked at early time points to determine if p53 mutation affects the growth and proliferation of early stage lung tumors. However, we did not find any differences in tumor size between the three different p53 alleles(Figure 3.3A and B).</p>
<p>Since lung cancer patients with mutations in <em>TP53</em> often start only have one mutated <em>p53</em> allele unless the loss of heterozygosity (LOH) occurs. Therefore, we generated mice heterozygous for both <em>Braf<sup>CAT</sup></em> and either <em>Trp53<sup>fl</sup></em>, <em>Trp53<sup>R172H</sup></em>, or <em>Trp53^ R245W^</em> and observed their survival over time (Figure 3.4A). We noted that <em>Trp53<sup>R172H/+</sup></em> mice had diminished median overall survival compared to <em>Trp53<sup>fl/+</sup></em> mice (183 days and 216 days), with <em>Trp53<sup>R245W/+</sup></em> mice falling in between the two groups (median survival 212 days) (Figure 3.4A).</p>
<p>We observed a distinct separation of survival between mice harboring lung tumors that are either <em>Trp53<sup>fl/+</sup></em>, <em>Trp53<sup>R172H/+</sup></em>, and <em>Trp53<sup>R245W/+</sup></em>. This separation is diminished when altered p53 alleles are homozygous (Figure 3.4B). <em>Trp53<sup>R172H/R172H</sup></em> mice have a median survival of 120 days, while <em>Trp53<sup>fl/fl</sup></em> mice have a median survival of 136 days. Importantly, this observed difference in survival is statistically significant (Log-rank test p&lt; 0.001) and is likely to reflect the worse diseased, characterized by enhanced tumor burden and size.</p>
<p><strong>Mutant-p53 Accumulation is Associated with Dedifferentiation and p19 Induction</strong>
Increased malignancy in lung adenocarcinoma is often associated with altered states of tumor cell identity and dedifferentiation <span class="citation" data-cites="1GkFEmggg pjuOgKHJ">(Marjanovic et al., 2020; van Veen et al., 2019)</span>. Since LUADs often arise from AT2 cells, we surveyed the expression of AT2 cell identifiers, such as NKX2-1, a master regulator of AT2 transcriptional programs, and SPC, a marker of differentiated AT2 cells <span class="citation" data-cites="pjuOgKHJ">(Marjanovic et al., 2020)</span>. Quantification of over 50 tumors from <em>Braf<sup>V600E/+</sup></em> mice with either <em>Trp53^ flfl^</em>, <em>Trp53<sup>R172H/R172H</sup></em>, and <em>Trp53<sup>R245W/R245W</sup></em> revealed that <em>Trp53<sup>R172H/R1722H</sup></em> mice had significantly more tumors that stained negative for NKX2-1 and SPC (Figure 3.5 A and B). This finding strongly indicates that mutant-p53 tumors are dedifferentiated. Since HMGA2 expression is associated with LUAD dedifferentiation and worse patient prognosis, we performed IHC for HMGA2<span class="citation" data-cites="hYWjwiQA 1C8aTrJLL to1qRTTV">(Di Cello et al., 2008; Gao et al., 2017; Winslow et al., 2011)</span>. We noted an increase in HMGA2-positive tumors in <em>Trp53<sup>R172H/R172H</sup></em>, and <em>Trp53<sup>R245W/R245W</sup></em> tumors compared to <em>Trp53<sup>fl/fl</sup></em> (Figure 3.5A and B). Thus, this data reveals that Trp53R172H/R172H mice exhibit a higher tumor burden and worse overall survival than <em>Trp53<sup>fl/fl</sup></em>. However, the tumors developed in these mice are more likely to exhibit malignant progression and dedifferentiation.</p>
<p>One striking feature of lung tumors in mice, either heterozygous or homozygous for <em>Trp53 <sup>R172H</sup></em> or <em>Trp53<sup>R245W</sup></em> is how few tumors and tumor cells have detectable levels of mutant-p53 protein as measured by Immunohistochemistry. One critical aspect of normal p53 function is self-regulation through transcriptional activation of its negative regulator MDM2 <span class="citation" data-cites="Mb1Q4pdk">(Wu et al., 1993)</span>. Thus, we predicted that homozygous p53 mutant cells would exhibit increased mutant-p53 protein. In our models, we observed that mutant p53 protein accumulation is restricted to malignant cells exhibited by a lack of papillary architecture and nuclear atypia (Figure 3.6A). Indeed, p53 immunostaining is correlated with increased phosphorylated-ERK1/2 detection (Figure 3.6B).
Moreover, immunofluorescence of p53 and its positive regulator, p19/ARF, reveal that cells with high p19/ARF are also high in p53 accumulation (Figure 3.6C). This result suggests that even in homozygous mutant_Trp53_ mice, there is still a barrier to mutant-p53 accumulation. Our data suggest that elevated levels of p19/ARF are necessary for mutant-p53 accumulation, even with homozygous p53 mutations. Collectively, these data indicate that <em>Trp53<sup>R172H</sup></em> drives more aggressive lung adenocarcinoma formation in BRAF(V600E) driven mouse models of lung cancer.</p>
<p><strong>P53<sup>R172H</sup> and P53<sup>R245W</sup> cell lines are refractory to BRAF/MEK inhibition</strong>
Given that previous research has determined that p53 mutations can confer resistance to both chemotherapies and pathway-targeted therapies <span class="citation" data-cites="PX0Ki15S 17uu2v5lM">(Huang et al., 2011; Rusch et al., 1995)</span>, we next questioned whether our p53 alterations would be sufficient to drive therapeutic resistance in BRAF<sup>V600E</sup>-driven genetically engineered mice<span class="citation" data-cites="BpRBkUVl">(Planchard et al., 2016)</span>. Interstingly, <em>Trp53<sup>R172H/R172H</sup></em> mice retain a more considerable tumor burden after a week four treatment of MAPK blockade <em>in vivo</em> (Figure 3.7A).</p>
<p>To further investigate the effects of p53-mutants on <em>BRAF<sup>V600E</sup></em> driven lung cancers,we derived cell lines from <em>Braf<sup>CAT/+</sup></em> murine tumors expressing either <em>Trp53<sup>flfl</sup></em>, <em>Trp53<sup>R172H/R172H</sup></em>, or <em>Trp53<sup>R245W/R245W</sup></em>) 14 weeks after infection with ad5-Spc-Cre. Cell lines expressing either <em>Trp53<sup>R172H/R172H</sup></em> or <em>Trp53<sup>R245W/R245W</sup></em> were resistant to single-agent dabrafenib (Figure 3.7B). Of note, <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> cell lines express increased ERK1/2 activity, as evidenced by increased phosphorylated ERK1/2, both with and without MAPK-blockade (Figure 3.7C). Furthermore, <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> were more resistant to the combination of dabrafenib plus trametinib, than <em>Trp53<sup>fl/fl</sup></em> cell lines(Figure 3.8A-C). Not only are resistant cells still proliferative in the face of MAPK-blockade, they are resistant to cell death imparted by MAPK-inhibtion(Figure 3.8C).</p>
<p>Next, we reasoned if mutant-p53 expression is the cause differential cancer phenotypes such as increase proliferation and drug resistance, then eliminating mutant-p53 expression would have a growth inhibitory effect and sensitize cells to MAPK-pathway inhibition. We cloned a short-hairpin RNA plasmid that induces GFP expression and knockdown of mutant-p53 with supplementation of doxycycline(Figure 3.9A). Despite sucessful p53 knockdown, observed no change in <em>in vitro</em> proliferation or <em>in vivo</em> tumor outgrowth(Figure 3.9B and C). Therefore, mutant-p53 cells are not addicted to mutant-p53 expression.</p>
<p>To gain relevance to human lung cancer, we infected BRAF(V600E) mutant HCC364 cells with lentivirus expressing human cancer hotspot mutant p53<sup>R175H</sup>. p53<sup>R175H</sup> expressing cells were more proliferative and refractory than parental HCC364 cells (Figure 3.10A and B). Collectively, these <em>in vitro</em> and <em>in vivo</em> data suggest that missense mutant p53<sup>R175H</sup> and its murine homolog p53<sup>R172H</sup>, as well as p53<sup>R245W</sup> can support cancer cell viability in the face of BRAF+MEK blockade that p53-null mutations cannot achieve.</p>
<p><strong>P53<sup>R172H</sup> and P53<sup>R245W</sup> alter the transcriptome of BRAF(V600E) lung tumor cell lines</strong>
Due to our observations that mutant p53<sup>R172H</sup> drives more aggressive cancer phenotypes such as increased growth, dedifferentiation, and resistance to FDA-approved therapeutics, we reasoned that this would be reflected in a change in the transcriptional landscape. This is due in part to the transcriptional trans-activation potential by mutant p53. RNA sequencing of cell lines generated from p53-mutant mouse lung tumors does reveal transcriptomic differences between both p53<sup>R172H</sup> and p53<sup>R245W</sup> cells compared to p53-null cell lines (Figure 3.11A and B). Interestingly, genes involved in cell cycle regulation and MYC pathway are upregulated in both p53<sup>R172H</sup> and p53<sup>R2245W</sup> cell lines (Figure 3.11B). These results suggest that mutant-p53 affects the transcription of genes involved in pro-growth signaling pathways. We also observed transcriptional differences between <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> cell lines(Figure <span class="citation" data-cites="3.12A">(<strong>3.12A?</strong>)</span> and B). Thus, we hypothesize that <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> have similarities in their function, but they also have some clear differences.</p>
<p><strong>PDGFR expression is increased in mutant-p53 cells</strong>
Our <em>in vitro</em> RNA analysis revealed an upregulation of the platelet-derived growth factor recepter(PDGFR) in <em>Trp53<sup>R172H/R172H</sup></em> cell lines(Figure 3.13A). Since PDGFR is a known regulator of the MAPK pathway, we sought to validate its expression and determine its important in <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> cell lines. Indeed PDGFRB is more highly expressed and can be activated by recombinant PDGF stimulation(Figure 3.14B). Although <em>Trp53<sup>fl/fl</sup></em> cells do not have any detectable levels of PDGFR or Phospohorylated-PDFRB, they still upregulated MAPK-pathway activation in response to ligand stimulation(Figure 3.14B). Furthermore, expression of PDGFRB is evident in malignant <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> tumors, but not <em>Trp53<sup>fl/fl</sup></em> tumors(Figure 3.14C). We next sought to determine the sensitivity to PDGFR-signaling blockade with Imatinib, a known PDGFRB inhibitor. Neither PDGFR stimulation or PDGFR inhibition had any effect on the proliferation of mutant-p53 cell lines(Figure 3.15A and B). Therefore, PDGFR is a validated differentially expressed gene in <em>Trp53<sup>R172H/R172H</sup></em> and <em>Trp53<sup>R245W/R245W</sup></em> cell lines, but is not important for its growth.</p>
<h2 id="discussion-1">Discussion</h2>
<p>While lung adenocarcinomas primarily contain missense mutations in p53, most studies using mouse models of lung adenocarcinoma utilize a p53-null (<em>Trp53<sup>fl/fl</sup></em>) allele instead of alleles encoding hotspot missense mutations <span class="citation" data-cites="gI9Yeagx">(Gao et al., 2013)</span>. This is most likely because prior mutant-p53 alleles were null for <em>Trp53</em> prior to Cre-mediated recombination, leaving every cell in the mouse sensitive to leukemias, lymphomas, and sarcomas due to being partially p53-deficient <span class="citation" data-cites="KKkLA3TK CVOIe1Dg MLY9vR0E">(Jackson et al., 2005; Olive et al., 2004; Wijnhoven et al., 2005)</span>. Our work has been made possible by newly described “wild-type to mutant” p53 alleles that allow for mice to be fully wild-type for p53 in all cells prior to Cre-mediated recombination<span class="citation" data-cites="kitgYOkt">(Zhang et al., 2018)</span>. This strategy is crucial as it allows for temporally extended experiments not confounded by separate malignancies. Here, we utilize " wild-type to mutant"<em>Trp53<sup>R172H</sup></em> and <em>Trp53<sup>R245W</sup></em> alleles R245W to determine how they alter tumor progression in a BRAF(V600E) model of lung cancer. We show that p53<sup>R172H</sup> and p53<sup>R245W</sup> accelerate lung tumor progression and increase the rate of dedifferentiation beyond the complete loss of p53 expression.</p>
<p>Although point mutations in the <em>TP53</em> gene can poison the natural tumor suppressive mechanisms of p53 protein, not all point mutations are equivalent in executing these functions. Mutations in the DNA-binding domain are most common and potent at ablating normal p53 function. Taking advantage of a conditional BRAF<sup>V600E</sup> lung tumor model where BRAF<sup>V600E</sup> expression in AT2 cells drives benign tumor formation, we show that either p53<sup>R172H</sup> or p53<sup>R245W</sup> can bypass growth arrest and influence tumor progression and therapeutic sensitivity beyond how p53 loss can. Specifically, P53<sup>R172H</sup> reduces survival and sensitivity to dabrafenib plus trametinib <em>in vivo</em>. Interestingly, a single allele of p53<sup>R172H</sup> has a more robust tumor-promoting phenotype than P53<sup>R245W</sup>. These findings further support a hierarchy model of p53 missense mutations, where conformation mutants such as p53<sup>R175H</sup> have a more substantial dominant-negative effect on wild-type p53, while mice with a single <em>trp53<sup>fl</sup></em> allele can still suppress tumorigenesis with one copy of wild-type but are prone to loss of heterozygosity <span class="citation" data-cites="kitgYOkt 7Yb2p7xx MLY9vR0E">(Muzumdar et al., 2016; Olive et al., 2004; Zhang et al., 2018)</span>.</p>
<p>In BRAF-driven lung adenocarcinoma, the repressive role of wild-type p53 is initiated by oncogenic stress brought about by increasing levels of MAPK-pathway activation<span class="citation" data-cites="dTvhNsG9">(Dankort et al., 2007)</span> . Hyperactivation of the MAPK pathway triggers p19/ARF to block MDM2, leading to p53 accumulation<span class="citation" data-cites="G5wVcRrJ CMVwSV1C okfbKJaD">(Christophorou et al., 2006; Feldser et al., 2010; Junttila et al., 2010)</span>. Therefore, p53 acts late in lung adenocarcinoma, unlike pancreatic adenocarcinoma, where p53 is triggered early, likely through PDAC-specific increases in p19/ARF through increased PI3K, STAT3, or DMP1 <span class="citation" data-cites="7Yb2p7xx">(Muzumdar et al., 2016)</span>. Our studies in BRAF(V600E) mutant lung adenocarcinomas reinforce this concept, as mutant-p53 does not accelerate growth in early tumors. Furthermore, p53 is not detected until later stage tumor development and correlates with p19/ARF expression and increased phosphorylated-ERK1/2 levels. Our studies indicate that mutant-p53 levels are associated with increased expression of p19/ARF. Therefore, any gain-of-function phenotype manifested by mutant-p53 must depend on accumulated p19/ARF. Thus, ablating the ARF tumor suppressor in human cancers with GOF <em>TP53</em> mutants may paradoxically lead to tumor regression.</p>
<p>While most <em>TP53</em> mutations in cancer result in an inability to bind DNA, they are often stable and preserve functional oligomerization and transcriptional transactivation domains<span class="citation" data-cites="ztWGsqcf">(Garcia and Attardi, 2014)</span>. Thus, mutant-p53 can still interact with and affect the function of cellular processes. Supporting this rationale, we observe altered transcriptomes with both p53<sup>R172H</sup> and p53^R245W cell lines compared to p53-null. Consistent with previous mutant-p53 GOF studies, we observed enriched genes associated with EMT and cell-cycle entry with p53<sup>R172H</sup> and p53<sup>R245W</sup>.</p>
<p>Many GOF phenotypes exhibited by mutant-p53 have centered around drug-resistant mechanisms. Previous studies have shown that mutant-p53 can confer resistance to pathway-targeted therapy and chemotherapies. Our studies highlight that p53<sup>R172H</sup> and p53<sup>R245W</sup> can confer resistance to BRAF-targeted therapy. Here, we show p53-R172H and p53-R245W cell lines are resistant to BRAF(V600E) inhibition and have less sensitivity to the combination of BRAF+MEK inhibition. In pancreatic cancer models with LSL-p53-R172H expression, mutant-p53 regulates PDGFR-signaling to promote an invasive phenotype sensitive to Imatinib<span class="citation" data-cites="jHVcJWec">(Weissmueller et al., 2014)</span>. Consistent with these findings, our transcriptional analysis and immunoblotting indicate p53-R172H, and p53-R245W cells express PDGF-ligands and exhibit PDGFR-pathway activation.</p>
<p>In summary, we have compared the tumorigenic potential of three <em><sup>Trp53</sup></em> alleles (<em>Trp53<sup>fl/fl</sup></em>, <em>Trp53<sup>R172H</sup></em>, and <em>Trp53<sup>R245W</sup></em>) concurrently with <em>BRAF<sup>V600E</sup></em>. <em>Trp53<sup>R172H</sup></em> and <em>Trp53<sup>R245W</sup></em> promoted more aggressive lung cancers exhibited by larger tumors and tumor dedifferentiation. However, only <em>Trp53<sup>R172H</sup></em> was observed to shorten survival compared to <em>Trp53<sup>fl/fl</sup></em>. Moreover, <em>Trp53<sup>R172H</sup></em> and <em>Trp53<sup>R245W</sup></em> cell lines were refractory to dabrafenib and trametinib and were more proliferative. <em>Trp53<sup>R172H</sup></em> and <em>Trp53<sup>R245W</sup></em> cell lines had altered transcriptomes compared to p53-null cells and were highly enriched in genes involved in cell cycle proliferation. Our work highlights the importance of studying disease-relevant <em>TP53</em> alleles in pre-clinical modeling systems.</p>
<p>##Methods</p>
<p><strong>Animal Husbandry</strong> All mice were housed in an environmentally controlled room, and all animal care and experimental procedures were approved by (and in accordance with) the Institutional Animal Care and Use Committee Office of the Huntsman Cancer Institute at the University of Utah. Genetically engineered mouse breeding and genotyping was conducted as previously described <span class="citation" data-cites="1GkFEmggg">(van Veen et al., 2019)</span>. The BRAF<sup>CAT</sup> and <em>Trp53<sup>fl/fl</sup></em> mice were previously described <span class="citation" data-cites="kGt16xDw 1GkFEmggg dTvhNsG9 Ntv29NRu">(Dankort et al., 2007; Jonkers et al., 2001; Shai et al., 2015; van Veen et al., 2019)</span>. All viruses were administered in a Biosafety Level 2+ room, as is regulated by the Institutional Biosafety Committee Guidelines. Adeno-SPC-CRE virus (University of Iowa) was delivered through nasal instillation. The <em>Trp53<sup>wmR172H</sup></em> and <em>Trp53<sup>wmR245W</sup></em> mice and respective genotyping protocols were gifted by Gigi Lozano and her laboratory and previously published <span class="citation" data-cites="kitgYOkt o8jKDrsB">(Kim et al., 2021; Zhang et al., 2018)</span>.</p>
<p><strong>Cell Line Generation</strong>Tumor bearing mice were euthanized and lungs dissected. Lungs were then minced and incubated in digestive enzymes Collagenase I, dispase, elastase, for 30 minutes at 37 degrees. Single-cell suspensions were then resuspended in complete DMEM medium and filtered through a 100 micron filter and centrifuged at 300 x g for 5 minutes. Single-cell suspensions were then incubated in red blood cell lysis buffer for 5 minutes at room temperature, then resuspended in complete RPMI and placed in a tissue culture incubator. Cells were then passaged once a week with differential trypsinization to purify tumors cells from viable stromal cells. p53 status and BRAF<sup>V600E</sup> status (tdTomato+) were assessed by western blot and flow cytometry.</p>
<p><strong>Tissue Culture</strong>Murine tumor derived cell lines and HCC-3564 cells were routinely cultured in DMEM supplemented with 10% heat-inactivated FBS and 1% penicillin/streptomycin. Cells were routinely tested for mycoplasma.</p>
<p><strong>Slide Imaging and Quantification</strong>Hematoxylin and eosin (H&amp;E) and IHC stained slides 1 were scanned automatically by the 3D Histech Pannoramic MIDI scanner (Thermo Fisher). Slides were imaged and analyzed using Caseviewer Software or QuantCenter analytical center provided on the 3D Histech Slide Scanner at the indicated magnification. Tumor burden was manually calculated on each lung lobe and total tumor area was compared to total lung area. Tumor diameters were measured using QuantCenter software from 3D Histech.</p>
<p><strong>Immunoblotting</strong>Cells destined for western blot were scraped in ice cold PBS then centrifuged to pellet cells. After supernatant aspiration, cells were lysed with RIPA buffer supplemented with a protease/phosphatase inhibitor cocktail (Halt). Cell lysate protein concentration was determined with BCA assay. 40 microgram/lane was mixed with 4X SDS buffer and 10X sample buffer. Samples were run through a 4-12% Bis-tris gel at 200V for approximately 75 minutes then transferred to a PVDF membrane using the iBlot2 transfer apparatus. Membranes were then incubated in Odyssey blocking buffer for 30 minutes before being incubated overnight with primary antibody. Membranes were then washed 3X with TBS-T and incubated in secondary antibody for 2 hours before being washed 3X and imaged on the Licor Odyssey clX.</p>
<p><strong>RNA Sequencing</strong>Cells lines were routinely cultured for approximately 10 passages before RNA purification using Qiagen Mini kit from one million cells. Libraries were generated with Illumina TruSeq Stranded mRNA Library Prep with UDI kit and sequenced on a Nova-Seq targeting 25 million reads per sample. fastq files were processed on Galazy.org using FastQC, HISAT2, MultiQC, FeatureCounts, and DESeq2.</p>
<div id="fig:3.1" class="fignos">
<figure>
<img src="images/p53_1.png" class="white" style="width:100.0%" alt="Figure 7: Figure 3.1. Schematic of Braf and Trp53 alleles (A) Priot to Cre-mediated recombination, normal Braf and Trp53 are expressed from their endogenous loci. (B) After Cre-mediated recombination, the indicated mutant proteins are expressed, in addition to a TdTomato fluorescent protein to mark expression of BRAFV600E-positive cells." />
<figcaption aria-hidden="true"><span>Figure 7:</span> <strong>Figure 3.1. Schematic of <em>Braf</em> and <em>Trp53</em> alleles</strong> (A) Priot to Cre-mediated recombination, normal <em>Braf</em> and <em>Trp53</em> are expressed from their endogenous loci. (B) After Cre-mediated recombination, the indicated mutant proteins are expressed, in addition to a TdTomato fluorescent protein to mark expression of BRAF<sup>V600E</sup>-positive cells.</figcaption>
</figure>
</div>
<div id="fig:3.2" class="fignos">
<figure>
<img src="images/p53_2.png" class="white" style="width:100.0%" alt="Figure 8: Figure 3.2. Mutant-p53 alleles differentially alter BRAFV600E driven lung tumorigenesis. (A) Tumor burden analysis 14-weeks post Cre-recombinase. (B) Tumor size 14 weeks post Cre-Recombinase. (C) Representative histological images 14 weeks post Cre-Recombinase. *** = p-value&lt;.01" />
<figcaption aria-hidden="true"><span>Figure 8:</span> <strong>Figure 3.2. Mutant-p53 alleles differentially alter BRAF<sup>V600E</sup> driven lung tumorigenesis.</strong> (A) Tumor burden analysis 14-weeks post Cre-recombinase. (B) Tumor size 14 weeks post Cre-Recombinase. (C) Representative histological images 14 weeks post Cre-Recombinase. *** = <em>p-value&lt;.01</em></figcaption>
</figure>
</div>
<div id="fig:3.3" class="fignos">
<figure>
<img src="images/p5313.png" class="white" style="width:100.0%" alt="Figure 9: Figure 3.3. Mutant-p53 does not alter the growth of early stage tumors** (A) Tumor burden calculation after 6 weeks of tumor growth (B) Representative images of H&amp;E stained lung sections." />
<figcaption aria-hidden="true"><span>Figure 9:</span> Figure 3.3. Mutant-p53 does not alter the growth of early stage tumors** (A) Tumor burden calculation after 6 weeks of tumor growth (B) Representative images of H&amp;E stained lung sections.</figcaption>
</figure>
</div>
<div id="fig:3.4" class="fignos">
<figure>
<img src="images/p53_3.png" class="white" style="width:100.0%" alt="Figure 10: Figure 3.4. Mutant-p53 alleles differentially affect the survival of mice due to progressive lung adenocarcinoma. (A) Kaplan-Meier plot of BrafCAT/+ mice with heterozygous p53-mutant alleles. (B) Kaplan-Meier plot of BrafCAT/+ mice with homozygous p53-mutant alleles.* = p-value&lt;.05" />
<figcaption aria-hidden="true"><span>Figure 10:</span> <strong>Figure 3.4. Mutant-p53 alleles differentially affect the survival of mice due to progressive lung adenocarcinoma.</strong> (A) Kaplan-Meier plot of <em>Braf<sup>CAT/+</sup></em> mice with heterozygous p53-mutant alleles. (B) Kaplan-Meier plot of <em>Braf<sup>CAT/+</sup></em> mice with homozygous p53-mutant alleles.* = <em>p-value&lt;.05</em></figcaption>
</figure>
</div>
<div id="fig:3.5" class="fignos">
<figure>
<img src="images/p53_4.png" class="white" style="width:100.0%" alt="Figure 11: Figure 3.5. Missense mutant-p53 alters lung adenocarcinoma differentiation status. (A)Representative IHC images of tumors stained with antibodies targeting p53, NKX2-1, SPC, and HMGA2. (B) Quantification of immunostaing in (A) based on H-Score. *** = p-value&lt;.01 ; **** = p-value&lt;.001." />
<figcaption aria-hidden="true"><span>Figure 11:</span> <strong>Figure 3.5. Missense mutant-p53 alters lung adenocarcinoma differentiation status.</strong> (A)Representative IHC images of tumors stained with antibodies targeting p53, NKX2-1, SPC, and HMGA2. (B) Quantification of immunostaing in (A) based on H-Score. *** = <em>p-value&lt;.01</em> ; **** = <em>p-value&lt;.001</em>.</figcaption>
</figure>
</div>
<div id="fig:3.6" class="fignos">
<figure>
<img src="images/p53_5.png" class="white" style="width:100.0%" alt="Figure 12: Figure 3.6. p53-accumulation occurs in high grade tumors and correlates with p19/ARF expression. (A) Representative images of mutant-p53 accumulation in high grade tumors. the tumor area encased by the dashed line exhibit nuclear atypia and enlarged nuclei.(B) Mutant-p53 immunostaining correlates with phospho-ERK1/2 accumulation. (C) p53 immunostaining correlates with loss of SPC expression. (D) mutant-p53 is co-expressed with p19/ARF in p53-mutant tumors." />
<figcaption aria-hidden="true"><span>Figure 12:</span> <strong>Figure 3.6. p53-accumulation occurs in high grade tumors and correlates with p19/ARF expression.</strong> (A) Representative images of mutant-p53 accumulation in high grade tumors. the tumor area encased by the dashed line exhibit nuclear atypia and enlarged nuclei.(B) Mutant-p53 immunostaining correlates with phospho-ERK1/2 accumulation. (C) p53 immunostaining correlates with loss of SPC expression. (D) mutant-p53 is co-expressed with p19/ARF in p53-mutant tumors.</figcaption>
</figure>
</div>
<div id="fig:3.7" class="fignos">
<figure>
<img src="images/p536.png" class="white" style="width:100.0%" alt="Figure 13: Figure 3.7. Mouse tumors and cell lines that expression P53-R172H and p53-R245W are refractory to MAPK blockade. (A) Tumor burden analysis of GEMMs treated with MAPKi chow for four weeks. (B) in vitro dose response to dabrafenib for 48 hours. (C) Western blots of time-course of cell lines treated with dabrafenib plus trametinib." />
<figcaption aria-hidden="true"><span>Figure 13:</span> <strong>Figure 3.7. Mouse tumors and cell lines that expression P53-R172H and p53-R245W are refractory to MAPK blockade.</strong> (A) Tumor burden analysis of GEMMs treated with MAPKi chow for four weeks. (B) <em>in vitro</em> dose response to dabrafenib for 48 hours. (C) Western blots of time-course of cell lines treated with dabrafenib plus trametinib.</figcaption>
</figure>
</div>
<div id="fig:3.8" class="fignos">
<figure>
<img src="images/p537.png" class="white" style="width:100.0%" alt="Figure 14: Figure 3.8. p53-R172H and p53-R245W cell lines are insensitive to MAPKi-induced cell death. (A-C) Incucyte analysis of cell lines treated with the combination of dabrafinib plus trametinib for 96 hours. (D) Accumulation of dead cells in response to dabrafinib plus trametinib. Dead cells were quantified by CytoTox Red Dye." />
<figcaption aria-hidden="true"><span>Figure 14:</span> <strong>Figure 3.8. p53-R172H and p53-R245W cell lines are insensitive to MAPKi-induced cell death.</strong> (A-C) Incucyte analysis of cell lines treated with the combination of dabrafinib plus trametinib for 96 hours. (D) Accumulation of dead cells in response to dabrafinib plus trametinib. Dead cells were quantified by CytoTox Red Dye.</figcaption>
</figure>
</div>
<div id="fig:3.9" class="fignos">
<figure>
<img src="images/shp53.png" class="white" style="width:100.0%" alt="Figure 15: Figure 3.9. Mutant-p53 knockdown has no effect on cell proliferation or tumor growth. (A) Western blot of increasing Doxycycline concentration to silence mutant-p53 using a short-hairpin RNA (B) Cell proliferation assay with 4ug/mL doxycycline concentration. (C) In vivo response to mutant-p53 knockdown." />
<figcaption aria-hidden="true"><span>Figure 15:</span> <strong>Figure 3.9. Mutant-p53 knockdown has no effect on cell proliferation or tumor growth.</strong> (A) Western blot of increasing Doxycycline concentration to silence mutant-p53 using a short-hairpin RNA (B) Cell proliferation assay with 4ug/mL doxycycline concentration. (C) <em>In vivo</em> response to mutant-p53 knockdown.</figcaption>
</figure>
</div>
<div id="fig:3.10" class="fignos">
<figure>
<img src="images/p538.png" class="white" style="width:100.0%" alt="Figure 16: Figure 3.10. HCC-364 cells that express p53-R175H are refractory to MAPKi. (A) Western blot showing successful ectopic expression of p53-R175H. (B) Incucyte analysis of HCC364 parental and HCC364+p53-R175H in response to dabrafenib plus trametenib." />
<figcaption aria-hidden="true"><span>Figure 16:</span> <strong>Figure 3.10. HCC-364 cells that express p53-R175H are refractory to MAPKi. (A) Western blot showing successful ectopic expression of p53-R175H.</strong> (B) Incucyte analysis of HCC364 parental and HCC364+p53-R175H in response to dabrafenib plus trametenib.</figcaption>
</figure>
</div>
<div id="fig:3.11" class="fignos">
<figure>
<img src="images/p539.png" class="white" style="width:100.0%" alt="Figure 17: Figure 3.11. P53-R172H and p53-R245W cell lines have altered transcriptome compared to p53-null cell lines. (A) Volcano plot showing up- and down- regulated genes compared to p53-null cells. (B) Pathway analysis of Differentially expressed genes of p53-R172H and p53-R245W cell lines compared to p53-null cell lines." />
<figcaption aria-hidden="true"><span>Figure 17:</span> <strong>Figure 3.11. P53-R172H and p53-R245W cell lines have altered transcriptome compared to p53-null cell lines.</strong> (A) Volcano plot showing up- and down- regulated genes compared to p53-null cells. (B) Pathway analysis of Differentially expressed genes of p53-R172H and p53-R245W cell lines compared to p53-null cell lines.</figcaption>
</figure>
</div>
<div id="fig:3.12" class="fignos">
<figure>
<img src="images/p5310.png" class="white" style="width:100.0%" alt="Figure 18: Figure 3.12. Comparison of differentially-expressed genes between p53-R172H and p53-R245W cell lines. (A) Volcano plot of differntially expressed gene between p53-R172H and p53-R245W tumor-derived cell lines. (B) Pathway analysis of differentially expressed genes between p53-R172H and p53-R245W tumor-derived cell lines." />
<figcaption aria-hidden="true"><span>Figure 18:</span> <strong>Figure 3.12. Comparison of differentially-expressed genes between p53-R172H and p53-R245W cell lines.</strong> (A) Volcano plot of differntially expressed gene between p53-R172H and p53-R245W tumor-derived cell lines. (B) Pathway analysis of differentially expressed genes between p53-R172H and p53-R245W tumor-derived cell lines.</figcaption>
</figure>
</div>
<div id="fig:3.13" class="fignos">
<figure>
<img src="images/pdgfr_1.png" class="white" style="width:100.0%" alt="Figure 19: Figure 3.13. PDGFRB is upregulated and expressed in mutant-p53 tumors and cell lines. (A) Log-normalized counts of Pdgfrb RNA reads from p53-mutant cell lines. (B) Western blot analysis of basal and PDGF-BB stimulated cells. (C) IHC of PDGFRB in p53-mutant GEMM tumors." />
<figcaption aria-hidden="true"><span>Figure 19:</span> <strong>Figure 3.13. PDGFRB is upregulated and expressed in mutant-p53 tumors and cell lines.</strong> (A) Log-normalized counts of <em>Pdgfrb</em> RNA reads from p53-mutant cell lines. (B) Western blot analysis of basal and PDGF-BB stimulated cells. (C) IHC of PDGFRB in p53-mutant GEMM tumors.</figcaption>
</figure>
</div>
<div id="fig:3.14" class="fignos">
<figure>
<img src="images/pdgfr_2.png" class="white" style="width:100.0%" alt="Figure 20: Figure 3.14. PDGF-stimulation or PDGF patway blockade has no effect on cell proliferation. (A) Growth response to PDGF stimulation. (B) Growth response to Imatinib treatment." />
<figcaption aria-hidden="true"><span>Figure 20:</span> <strong>Figure 3.14. PDGF-stimulation or PDGF patway blockade has no effect on cell proliferation. (A) Growth response to PDGF stimulation.</strong> (B) Growth response to Imatinib treatment.</figcaption>
</figure>
</div>
<h2 class="page_break_before" id="chapter-four">Chapter Four</h2>
<p>##Abstract
Malignant lung adenocarcinomas tumors are initiated from a single neoplastic cell. Nevertheless, there is substantial cellular heterogeneity in a fully malignant tumor. Since human lung adenocarcinomas are heavily mutated, it is difficult to know whether underlying cellular heterogeneity is a consequence of the underlying mutational heterogeneity. Mouse models of lung adenocarcinoma accurately represent genomic and histological aspects of human lung adenocarcinoma. However, genetically-engineered mouse model tumor heterogeneity remains poorly understood. Here, we profile the single-cell transcriptomes of genetically-engineered mouse models driven by the BRAF<sup>V600E</sup> oncoprotein with five separate cooperating genetic events ( <em>Pi3k<sup>H1047R</sup></em>, <em>CdkN2a<sup>fl/fl</sup></em>, <em>Trp53<sup>fl/fl</sup></em>, <em>Trp53<sup>R172H/R162H</sup></em>, and <em>Trp53<sup>R245W/R245W</sup></em>). There are stereotypically distinct clusters of cells throughout all genotypes of BRAF<sup>V600E</sup>-driven tumors. We also identified known tumor cell identities in human and mouse lung adenocarcinoma.</p>
<h2 id="introduction-2">Introduction</h2>
<p>The mammalian lung is a vast and intricate labyrinth responsible for gas exchange between air and body through a meshwork of endothelial cells. The lung organ system begins with a single airway that branches in a geometric and stereotypic pattern<span class="citation" data-cites="OuXi55DW">(Metzger et al., 2008)</span>. These thousands of branches lead to the business end of gas exchange, the lung alveoli. The alveoli comprise two specialized cell types: Alveolar Type 1 (AT1) cells comprise most of the lung’s surface area and are thin squamous cells that mediate oxygen exchange across cell membranes. However, alveolar Type 2 (AT2) cells are cuboidal that secrete surfactant proteins that aid in preventing the delicate alveoli from collapsing.</p>
<p>Pathology of the alveoli is a steadfast threat to an organism’s life due to the importance of gas exchange. Diseases such as lung adenocarcinoma, pulmonary fibrosis, and emphysema are the most common dysfunctions of the distal lung epithelium<span class="citation" data-cites="n08HchE9">(Desai et al., 2014)</span>. Lung adenocarcinoma is also the most common type of human lung cancer in men and women. A large body of literature has suggested that lung adenocarcinomas arise in the alveolar and are descendants of AT2 cells <span class="citation" data-cites="oVtv8JOR cD4M6ONJ">(Ferone et al., 2020; Jackson et al., 2001)</span>. Therefore, there is great interest in understanding how AT2 cells develop and maintain cell identity and how they renew themselves during aging or in response to injury <span class="citation" data-cites="NLPwvKCP vsbCfYx4 vsbCfYx4 179MWicwA">(2020; Alanis et al., 2014, 2014; Treutlein et al., 2014)</span>.</p>
<p>Recent advances in profiling single-cell transcriptomes (scRNA Seq) during homeostasis conditions and disease states have greatly improved our understanding of AT2 cell and lung adenocarcinoma cell biology<span class="citation" data-cites="pjuOgKHJ RCmUro9f NLPwvKCP">(LaFave et al., 2020; Marjanovic et al., 2020; Treutlein et al., 2014)</span>. Single-cell profiling of human lung cancers has revealed the extreme intra-tumoral heterogeneity in a single tumor<span class="citation" data-cites="1EmZp72ZM FfxdQ0tV">(Gerlinger et al., 2012; Xue et al., 2016)</span>. Since transcriptional heterogeneity can be considered a source of therapeutic resistance, there is great interest in understanding the evolution of lung adenocarcinoma and how it changes with response to pathway targeted therapy<span class="citation" data-cites="btzNHt9A">(Maynard et al., 2020)</span>. Since GEMMs are an important pre-clinical model for testing potential therapeutic strategies for lung adenocarcinoma, it is essential to understand single-cell heterogeneity in popular GEMM models.</p>
<p>Although GEMMs are a great way to model human lung adenocarcinomas, several drawbacks are worth mentioning. Human lung adenocarcinomas typically develop from a single tumor-initiating cell. They result from an accumulation of mutations in specific cell types and a weakened immune system across a lifetime. In GEMMs, tumorigenesis is often initiated in many cells at once and is the consequence of an engineered mutation. In contrast to human lung adenocarcinomas, which have a high mutational burden, murine lung adenocarcinomas do not have a high mutational burden and instead display high copy number variation<span class="citation" data-cites="zOzqGxQM">(Westcott et al., 2014)</span>. Therefore, we set out to profile the single-cell transcriptomes of various BRAF<sup>V600E</sup>-driven lung cancer GEMMs to determine genotype-specific heterogeneity and response to pathway-targeted therapy.</p>
<p>##Methods</p>
<h2 id="results-2">Results</h2>
<p>We profiled the single-cell transcriptome of six different BRAF<sup>V600E</sup> mouse lung tumor models, including BRAF<sup>V600E</sup> on its own, BRAF<sup>V600E</sup> plus either mutant <em>Pik3ca<sup>H1047R/+</sup></em>, <em>Cdkn2a<sup>fl/fl</sup></em>, <em>Trp53<sup>fl/fl</sup></em>, <em>Trp53<sup>R172H/R172H</sup></em>, or <em>Trp53<sup>R245W/R245W</sup></em>. We also treated each genotype with dabrafenib (100mg/kg) plus trametinib(2mg/kg) once daily for four days before tissue harvesting and 10X genomics library preparation. Importantly, we kept dabrafenib and trametinib throughout all processing stages to prevent altered transcription in response to drug cessation. Using fluorescently-activated cell sorting (FACS), we purified tumor cells based on Td-Tomato expression eight weeks after tumor initiation using adenoviral Spc-Cre. We were able to identify 71,110 cells that passed standard 10X genomics and Seuratbased quality control and filtering (Figure 4.1).</p>
<p>Although we sought to purify tumor cells based on TdTomato-positivity, we unintentionally collected and analyzed stromal cells, such as immune, mesenchymal, and endothelial cells, which are also present in the murine distal lung epithelium (Figure 4.2). We next determined tumor cells by previously identified tumor cell transcriptional programs in KRAS(G12D) mouse lung cancer<span class="citation" data-cites="pjuOgKHJ">(Marjanovic et al., 2020)</span>. In our dataset, we successfully identified AT2-like, AT1/AT2 bipotent progenitor, Cycling, mixed, and Hepatic-Gastric cellular programs (Figure 4.3). We used these programs to purify tumor cells for downstream differential gene expression analysis. Reclustering of tumor cells revealed significant tumor cell heterogeneity(Figure 4.4). Next, we sought to determine cluster-specific markers using DESeq2 analysis (Figure 4.4B). Thus, we have successfully documented the tumor heterogeneity in various models of BRAF<sup>V600E</sup> lung tumorigenesis.</p>
<p>We next sought to determine the effect of dabrafenib plus trametinib on transcriptional heterogeneity (Figure 4.5). There is a MAPK specific gene expression program is decreased in response to MAPKi(Figure 4.5A). All genotypes respond similarly to MAPK-pathway inhibition(Figure 4.5B). Therefore, we hypothesize that although transcriptional heterogeneity exists, there is a uniform response to MAPK-blockade.</p>
<p>Since there is great interest in p53-mediated tumor suppression, we wondered if we could detect a p53-gene signature in our data set. Indeed, we detected the reduction of transcripts often associated with p53 transcriptional activation in response to p53-mutation, such as downregulation of <em>Cdkn1a</em>, <em>Zmat3</em>, and <em>CCng1</em>(Figure 4.6A). Furthermore, P53-target gene expression is maintained only in p53 wild-type samples, including <em>Cdkn2a</em> loss (Figure 4.6B). Therefore we have successfully identified a p53-gene signature in _BRAF<sup>V600E</sup> expressing lung tumors using scRNA sequencing.</p>
<p>We were also interested in genotype-specific differences. Therefore, we performed DESEQ2 between vehicle-treated mice of each genotype (Figure 4.7). Although we did find differentially expressed genes between genotypes, there appear to be significant batch effects, as samples cluster based on their 10X genomic chip runs. Therefore, more careful analysis is needed to determine subtle genotype-specific gene expression changes. For instance, we could utilize a variety of batch-correcting pipelines to negate batch effects.</p>
<h2 id="discussion-2">Discussion</h2>
<p>Here, we used scRNA-seq to determine transcriptional heterogeneity of BRAF<sup>V600E</sup> driven lung cancer and how it changes in response to concomitant genetic alterations. We modeled lung adenocarcinoma that mimick human oncogenic transformation where MAPK pathway activation drives tumor formation, and inactivation of the p53 pathway allows for tumor progression<span class="citation" data-cites="xy4r2sTu">(Campbell et al., 2016)</span>. We also determined how cell state changes in response to MAPK blockade, a standard therapeutic option for patients with BRAF<sup>V600E</sup>-mutant lung adenocarcinomas.</p>
<p>We determined that there is transcriptional heterogeneity and that it is reproducible across genotypes. We also confirmed previously published cell states in similar models, such as KRAS G12D<span class="citation" data-cites="pjuOgKHJ">(Marjanovic et al., 2020)</span>. Unfortunately, we did not observe robust dedifferentiation signals such as loss of NKX2-1 and a gastric cell state adoption. However, we did collect and identify highly-plastic cell states, such as ITGA2+ and CLDN4+ cells. Our failure to collect and analyze pre-metastatic cell states such as EMT-like cells is likely because we collected cells only after 8-weeks, which is likely, not long enough for these cell states to evolve during lung tumorigenesis. Therefore, although it might require specific genomic alterations like p53 loss, the adoption of highly malignant cell states is an outcome of tumor evolution. One hypothesis is that p53 loss causes genomic instability and thus copy number gain, but similar studies failed to detect robust changes in copy number variation<span class="citation" data-cites="pjuOgKHJ">(Marjanovic et al., 2020)</span>.</p>
<p>We also failed to detect robust differences between genotypes known to cause malignant transformation, such as p53 loss, p53 missense mutation, CDKN2A loss, or mutant PI3K<sup>H1047R</sup> <span class="citation" data-cites="1GkFEmggg 1HVB1AoBP dTvhNsG9">(Dankort et al., 2007; Trejo et al., 2013; van Veen et al., 2019)</span>. This is likely due to a combination of factors, such as time of collection and library capture/sequencing depth. One straightforward hypothesis is that we did not allow complete transcriptomic changes to take effect imparted by tumor suppressor or oncogene mutation. Therefore, a time-course analysis that spans early time points (4-6 weeks) and late time points (&gt;16 weeks) will likely reveal more informative transcriptional heterogeneity. Indeed, our previous studies have indicated no tumor growth advantage at early time points as late as six weeks post tumor initiation. One other possibility that could explain our lack of genotype-specific gene expression programs is that we did not capture enough unique transcripts to reveal the full extent of tumor heterogeneity. This is likely because there are known limitations to 10X genomic library preparation. Specifically, there are gene-dropout effects where the droplet-based technique captures only the most highly expressed genes. Indeed, 10X genomics technologies have been shown not to detect genes with lower expression<span class="citation" data-cites="2gUfVvoy">(Wang et al., 2021)</span>. However, 10X genomics allows for profiling more cells than similar scRNA-seq techniques. Genes with lower expression can be as important as highly expressed genes for a cell’s identity. Transcription factors that determine cell states have been shown to have modest expression<span class="citation" data-cites="2gUfVvoy YdeeJclQ">(Martin and Sung, 2018; Wang et al., 2021)</span>.
Therefore, we hypothesize that using plate-based scRNA-seq approaches, such as Smart-Seq2, would allow more gene capture. Thus, Smart-seq2 will enable us to determine transcriptional heterogeneity and genotype-specific gene expression changes more precisely.</p>
<p>Although our analysis was lacking regarding transcriptional differences between genotypes, we did detect a robust transcriptional change with MAPK blockade (figure4.5). Since human lung cancer patients treated with MAPK-pathway inhibitors exhibit heterogeneity, one might hypothesize that each cell responds differently to MAPK-pathway inhibition. Our analysis determines that there is generally a homogenous response to MAPK inhibition. One straightforward explanation is that our short-term treatment (four days) did not allow cells to respond to pathway blockade. Perhaps there is an initial response to MAPK-pathway inhibition that is dampened over time dependent on transcriptional heterogeneity.</p>
<p>In conclusion, we used 10X genomics to determine the transcriptional heterogeneity of BRAF<sup>V600E</sup> lung tumors with five separate cooperating alterations and treatments. We did identify transcriptional heterogeneity that is stereotypic and reproducible across genotypes. However, we failed to identify robust differences between genotypes. Our work highlights the importance of accurately capturing complete transcriptomes across multiple time points to accurately determine cell states throughout BRAF<sup>V600E</sup>-driven lung tumor evolution.</p>
<p>##Methods
<strong>Animal Husbandry</strong> All mice were housed in an environmentally controlled room, and all animal care and experimental procedures were approved by (and in accordance with) the Institutional Animal Care and Use Committee Office of the Huntsman Cancer Institute at the University of Utah. Genetically engineered mouse breeding and genotyping was conducted as previously described <span class="citation" data-cites="1GkFEmggg">(van Veen et al., 2019)</span>. The <em>BRAF<sup>CAT</sup></em>, <em>Trp53<sup>fl/fl</sup></em>, <em>Cdkn2a<sup>fl/fl</sup></em>, and <em>Pik3ca<sup>H1047R/+</sup></em> mice were previously described <span class="citation" data-cites="kGt16xDw 1GkFEmggg dTvhNsG9 Ntv29NRu">(Dankort et al., 2007; Jonkers et al., 2001; Shai et al., 2015; van Veen et al., 2019)</span>. All viruses were administered in a Biosafety Level 2+ room, as is regulated by the Institutional Biosafety Committee Guidelines. Adeno-SPC-CRE virus (University of Iowa) was delivered through nasal instillation. The <em>Trp53<sup>wmR172H</sup></em> and <em>Trp53<sup>wmR245W</sup></em> mice and respective genotyping protocols were gifted by Gigi Lozano and her laboratory and previously published <span class="citation" data-cites="kitgYOkt o8jKDrsB">(Kim et al., 2021; Zhang et al., 2018)</span>.</p>
<p><strong>Single Cell RNA-Seq analysis</strong> Mice were initiated as described previusly using ad5-Spc-Cre. Four days prior to tissue harvesting, mice were either treated with corn oil control or corn oil with dabrafenib (150mg/kg) and trametenib(2mg/kg) once daily and again two hours prior to tissue harvest. single cell suspensions were made using enzymatic digestion, as previously detailed in cell line generation. Single-cell suspensions were then sorted with the BD ARIA II flow cytometer. 10X genomics library prep and sequencing was performed by HCI high throughput genomics core. Seurat objects where then created for downstream differential gene expression analysis and visualization with ggplots2.</p>
<p><strong>Immunoblotting</strong>Cells destined for western blot were scraped in ice cold PBS then centrifuged to pellet cells. After supernatant aspiration, cells were lysed with RIPA buffer supplemented with a protease/phosphatase inhibitor cocktail (Halt). Cell lysate protein concentration was determined with BCA assay. 40 microgram/lane was mixed with 4X SDS buffer and 10X sample buffer. Samples were run through a 4-12% Bis-tris gel at 200V for approximately 75 minutes then transferred to a PVDF membrane using the iBlot2 transfer apparatus. Membranes were then incubated in Odyssey blocking buffer for 30 minutes before being incubated overnight with primary antibody. Membranes were then washed 3X with TBS-T and incubated in secondary antibody for 2 hours before being washed 3X and imaged on the Licor Odyssey clX.</p>
<p><strong>RNA Sequencing</strong>Cells lines were routinely cultured for approximately 10 passages before RNA purification using Qiagen Mini kit from one million cells. Libraries were generated with Illumina TruSeq Stranded mRNA Library Prep with UDI kit and sequenced on a Nova-Seq targeting 25 million reads per sample. fastq files were processed on Galazy.org using FastQC, HISAT2, MultiQC, FeatureCounts, and DESeq2.</p>
<p><strong>Immunohistochemistry and Immunofluoresence</strong>Immunohistochemistry was performed as previously described, with the rabbit primary antibody against P53, SPC, phospho-ERK1/2, NKX2-1, HMGA2, p19/ARF <span class="citation" data-cites="Rky3nm5Q 1GkFEmggg">(Mollaoglu et al., 2017; van Veen et al., 2019)</span>.</p>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_1.png" class="white" style="width:100.0%" alt="Figure 21: Figure 4.1. 71,110 cells collected from six different genotypes." />
<figcaption aria-hidden="true"><span>Figure 21:</span> <strong>Figure 4.1. 71,110 cells collected from six different genotypes.</strong></figcaption>
</figure>
</div>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_2.png" class="white" style="width:100.0%" alt="Figure 22: Figure 4.2. Single cells broadly cluster into epithelial-, immune-, endothelial-, or mesenchymal- like cells." />
<figcaption aria-hidden="true"><span>Figure 22:</span> <strong>Figure 4.2. Single cells broadly cluster into epithelial-, immune-, endothelial-, or mesenchymal- like cells.</strong></figcaption>
</figure>
</div>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_3.png" class="white" style="width:100.0%" alt="Figure 23: Figure 4.3. Murine lung adenocarcinoma transcripional programs in BRAFV600E tumorigenesis" />
<figcaption aria-hidden="true"><span>Figure 23:</span> <strong>Figure 4.3. Murine lung adenocarcinoma transcripional programs in BRAF<sup>V600E</sup> tumorigenesis</strong></figcaption>
</figure>
</div>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_4.png" class="white" style="width:100.0%" alt="Figure 24: Figure 4.4. Transcriptional heterogeneity of tumor cells.(A) UMAP plot showing different tumor cluster. (B) Heatmap of top 3 differentially expressed genes in each cluster" />
<figcaption aria-hidden="true"><span>Figure 24:</span> <strong>Figure 4.4. Transcriptional heterogeneity of tumor cells.</strong>(A) UMAP plot showing different tumor cluster. (B) Heatmap of top 3 differentially expressed genes in each cluster</figcaption>
</figure>
</div>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_5.png" class="white" style="width:100.0%" alt="Figure 25: Figure 4.5. MAPK pathway transcriptional sigure is identifed in BRAVV600E tumor models. (A) Heatmap of top differentially expressed genes. (B) Program score of MAPK pathway activation" />
<figcaption aria-hidden="true"><span>Figure 25:</span> <strong>Figure 4.5. MAPK pathway transcriptional sigure is identifed in BRAV<sup>V600E</sup> tumor models.</strong> (A) Heatmap of top differentially expressed genes. (B) Program score of MAPK pathway activation</figcaption>
</figure>
</div>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_6.png" class="white" style="width:100.0%" alt="Figure 26: Figure 4.6. P53 transcriptional signature is identified in BRAVV600E, p53-wildtype tumor cells. (A) Heatmap of top differentially expressed genes in p53 pathway. (B) P53 program score is revealed to be upregulated in only p53-wildtype mice." />
<figcaption aria-hidden="true"><span>Figure 26:</span> <strong>Figure 4.6. P53 transcriptional signature is identified in BRAV<sup>V600E</sup>, p53-wildtype tumor cells.</strong> (A) Heatmap of top differentially expressed genes in p53 pathway. (B) P53 program score is revealed to be upregulated in only p53-wildtype mice.</figcaption>
</figure>
</div>
<div id="fig:04" class="fignos">
<figure>
<img src="images/scrna_7.png" class="white" style="width:100.0%" alt="Figure 27: Figure 4.7. Heatmap of top Genotpye-specific gene expression changes." />
<figcaption aria-hidden="true"><span>Figure 27:</span> <strong>Figure 4.7. Heatmap of top Genotpye-specific gene expression changes.</strong></figcaption>
</figure>
</div>
<h2 class="page_break_before" id="chapter-six-future-perspectives">Chapter Six: Future Perspectives</h2>
<p>This work’s general purpose was to understand better how BRAF<sup>V600E</sup> tumors form and progress to lung adenocarcinoma. Our strategy was (1) to determine additional tumor suppressor genes in BRAF<sup>V600E</sup> lung tumorigenesis, (2) to evaluate the effects of different mutant-p53 alleles, and (3) to profile the tumor cell heterogeneity using scRNA-seq. To achieve these goals, we heavily relied on genetically engineered mouse models of B_BRAF<sup>V600E</sup> lung tumors to characterize and compare the contribution of candidate-based tumor suppressors. This chapter will examine the exciting possibilities that arose due to our work.</p>
<p>Every year in the United States, approximately 4000 patients succumb to <em>BRAF</em>-mutant lung cancer. Unlike melanoma or thyroid cancer, <em>BRAF<sup>V600E</sup></em> is not the most common <em>BRAF</em> alteration. Less potent MAPK activating mutations such as <em>BRAF<sup>G469A</sup></em> are more common<span class="citation" data-cites="NqyqCnIL v2CDt1N5">(Bracht et al., 2019; Yaeger and Corcoran, 2019)</span>. It is likely that less potent MAPK activators serve as support signaling molecules for oncogenes like <em>KRAS(G12D)</em>, which fine-tune pathway activation. However, little is known about how vital non-V600E <em>BRAF</em> mutations are in the evolution of lung adenocarcinoma. Therefore, preclinical models such as genetically engineered mice will be vital to determine the role of non-<em>BRAF<sup>V600E</sup></em> mutations in lung adenocarcinoma. Previous work from our lab has established that <em>BRAF<sup>V600E</sup></em> plays a substantial role in the early formation of lung benign lung tumors but fails to form lung adenocarcinoma. Rather, benign to malignant lung cancer progression depends on additional genetic alterations<span class="citation" data-cites="1HVB1AoBP">(Trejo et al., 2013)</span>.</p>
<p>The mechanisms by which the mutationally activated MAPK-pathway promotes lung tumorigenesis remain incompletely understood. However, recent work suggests that the levels of MAPK-pathway activation correlate with tumor initiation and progression<span class="citation" data-cites="CMVwSV1C OK8s0ke5">(Cicchini et al., 2017; Feldser et al., 2010)</span>. Therefore, there must be secondary events after mutational activation of KRAS or BRAF that alter the tone of the MAPK pathway. Recent work has highlighted potential mechanisms tumor cells adapt to increase pathway activation. Amplification of the KRAS locus occurs during early murine tumorigenesis initiated by KRAS-G12D<span class="citation" data-cites="iF6d3gGU">(To et al., 2011)</span>. During late tumorigenesis, MAPK-pathway hyperactivation can occur through loss of DUSP6 regulation by NKX2-1<span class="citation" data-cites="HhXx5lwJ">(Ingram et al., 2021)</span>. However, NKX2-1 loss is also known to affect tumor cell identity and is associated with dedifferentiation<span class="citation" data-cites="u4tRgATP 7H88gN81">(Mollaoglu et al., 2018; Zewdu et al., 2021)</span>; <span class="citation" data-cites="aazIWGTV">Snyder et al. (2013)</span>]. For increased MAPK pathway activation to drive tumor progression, p53 function needs to be inhibited. Therefore, there are two main barriers to tumor progression in lung adenocarcinoma: (1) MAPK pathway activation and (2) subversion of p53-mediated tumor suppression.</p>
<p>We initially hypothesized that the barrier to tumor progression was the initiation of oncogene-induced senescence. Previous studies have linked oncogenic MAPK signaling to p53-mediated tumor supression<span class="citation" data-cites="qiDMl6l6 N2yNjRyX">(Woods et al., 1997; Zhu et al., 1998)</span>. One hypothesis is that MAPK pathway activation crescendos during tumorigenesis until p19/ARF expression is induced, which triggers p53 activation and tumor suppressor. However, it is unclear if p53 needs to be silenced, as work from our lab and others have provided evidence that WNT signaling is also a barrier<span class="citation" data-cites="Djv3zv5n 4TvMmTwj">(Juan et al., 2014; Tammela et al., 2017)</span>. Whether increased WNT-signaling can overcome p53-induction remains to be understood. However, it is clear that p53 suppression and MAPK activation are both frequent events in human cancers and cooperate to form malignant lung adenocarcinoma in mice. Therefore, more research is needed to determine the precise mechanisms governing p53 tumor suppression and tumor progression.</p>
<p>The diversity of p53 mutations in human cancers is unlike other tumor suppressor genes. Whereas mutations in tumor suppressors APC, RB, and BRCA1, are inactivated thru aberrant stop-codons or gene deletion events, mutations in p53 are often missense mutations in the DNA-binding domain<span class="citation" data-cites="I9heYgx4">(Hainaut and Pfeifer, 2016)</span>. Although mutational inactivation of p53 is important for <em>BRAF<sup>V600E</sup></em> driven lung tumor progression, common p53 alterations have not been tested for cooperating with BRAF<sup>V600E</sup>. Instead, null alleles of p53 have been studied. This is undoubtedly due to the poor design of hotspot mutants of p53, such as p53-R175H, p53-R273H, and p53-R248W. These alleles of mutant p53 are designed to be null for p53 prior to recombination, and mice harboring these alleles are prone to developing cancers that have nothing to do with MAPK activation in the lung epithelium. Therefore, we utilized the recently generation wild-type-to-mutant p53 alleles <em>TRp53<sup>R172H</sup></em> and <em>Trp53<sup>R245W</sup></em> (<em>Tp53<sup>R175H</sup></em> and <em>Tp53<sup>R248W</sup></em> in humans)<span class="citation" data-cites="kitgYOkt">(Zhang et al., 2018)</span>. Part of this work has been dedicated to characterizing the effect of <em>Trp53<sup>fl/fl</sup></em>, <em>Trp53<sup>R172H</sup></em>, and <em>Trp53<sup>R245W</sup></em> on <em>Braf<sup>V600E</sup></em> lung tumorigenesis. Strong evidence and careful characterization of mutant-p53 alleles have determined that not all p53-mutations are created equal. We have shown that p53<sup>R172H</sup> and p53<sup>245W</sup> differentially alter BRAF<sup>V600E</sup> tumor progression. The mechanism by which p53-missense mutants abrogate tumor suppression is two-fold. Mutant-p53 inactivates wild-type p53 through acting as a dominant-negative<span class="citation" data-cites="CIReXnPL">(Chan et al., 2004)</span>. More controversial is the ability of mutant-p53 to exert gain-of-function effects. Our studies failed to determine a molecular explanation as to why P53<sup>R172H</sup> and P53<sup>R245W</sup> are more potent drivers of cancer growth and drug resistance. However, we did determine that cell lines derived from mouse lung tumors have altered transcriptional profiles. Recently, evidence has emerged suggesting aneuploidy is a common event in mutant-p53 cells<span class="citation" data-cites="pXJPz758">(Redman-Rivera et al., 2021)</span>. Specifically, copy number alterations were more common in mutant-p53 cells than in p53-null cells in an isogenic study<span class="citation" data-cites="pXJPz758">(Redman-Rivera et al., 2021)</span>. Therefore, the phenotypic diversity underlying many p53 gain-of-function studies could reflect inherent genomic instability imparted by p53-mutations. This idea could help explain our data, as loss of mutant-p53 had no apparent phenotype in mutant-p53 cells. Therefore, it is of great interest to determine the copy number variation in our various p53-mutant models.</p>
<p>This dissertation support previous studies that found gain-of-function effects imparted by mutant-p53<span class="citation" data-cites="nZOmv1za">(Pfister and Prives, 2017)</span>. However, the field of mutant-p53 gain-of-function is still murky with controversy. In the future, it is essential to resolve our understanding of mutant-p53 proteins and how they influence the properties of cancer cells. We believe the diversity of mutant-p53 mechanisms can be explained by complex interactions between tumor types and each mutant-p53. Therefore, careful analysis of each p53-mutant in each cancer type might be required. Therefore, we propose to combine Tuba-Seq with CRISPR/CAS9-mediated homology-directed repair to determine the fitness landscape of every p53 mutation possible, similar to what has been performed with <em>Kras</em> in lung cancer and more recently in colorectal cancer <span class="citation" data-cites="1FBeCW3xl 6jSWGuKW">(Cai et al., 2021; Zafra et al., 2020)</span>. This will allow for the unbiased quantification of the growth advantage imparted by individual p53-mutants in a single animal.</p>
<p>Our studies also shed insight into p53 stability in BRAF<sup>V600E</sup> lung cancer. We have observed that mutant-p53 accumulation correlates with p19/ARF, consistent with previously published research<span class="citation" data-cites="okfbKJaD CMVwSV1C">(Feldser et al., 2010; Junttila et al., 2010)</span>. Therefore, p19/ARF may play a dichotomous role in tumor development. In the context of wild-type p53, p19/ARF might play a tumor-suppressive role in response to overactive MAPK signaling. In mutant-p53 expressing tumors, p19/ARF can stabilize mutant-p53, potentiating cancer progression. In order to fully understand p53 tumor suppression and mutant-p53 dominant-negative effects, we propose to study p19/ARF-specific deletion in the context of wild-type and mutant p53 in BRAF<sup>V600E</sup> tumorigenesis. Since the <em>CDKN2A</em> locus encodes p19/ARF, it is difficult to determine the effect of p19-specific roles in genetically-engineered mice. However, one could still test the role of p19/ARF in mutant-p53 stabilization <em>in vivo</em> and <em>in vitro</em> using p19/ARF specific RNA-interference such as siRNAs or short-hairpin RNAs.</p>
<p><strong>Pathway Targeted Therapy</strong></p>
<p>Resistance is inevitable to pathway-targeted therapy. Drug resistance is a consequence of complex tumor heterogeneity that drives tumor evolution. Interestingly, it has been suggested that the baseline mutation rate of tumor cells is high enough that in a sufficiently large tumor, all possible single nucleotide variants have likely occured<span class="citation" data-cites="7GTCEP43 BUDX0W0n">(Ruan et al., 2020.; Tomlinson et al., 1996)</span> Furthermore, the more prolonged the therapy, the more likely that resistant cells can propagate and fuel tumor growth after an initial tumor regression. In terms of MAPK-pathway inhibition in melanoma and lung cancer, many resistance mechanisms have been identified<span class="citation" data-cites="btzNHt9A 1F2tY5YCh">(Maynard et al., 2020; Rudin et al., 2013)</span>. Not surprisingly, many resistance mechanisms involved reactivation of the MAPK by mutational activation of upstream or downstream regulators<span class="citation" data-cites="WiSGMfl3 1F2tY5YCh 15BsPbLof">(Holderfield et al., 2014; Lin et al., 2014; Rudin et al., 2013)</span>. Our work has provided evidence that p53-mutations can alter responses to pathway-targeted therapy.
Our data indicate that <em>Trp53</em>-mutant cell lines have greater basal MAPK pathway activation. Moreover, the MAPK pathway is still active in response to MAPK targeted therapy. Therefore, we hypothesize that mutant-p53 can alter MAPK pathway activity to drive therapeutic resistance. However, we have yet to determine a mechanism to explain how mutant-p53 alters MAPK signaling. Thus, understanding the molecular mechanisms governing mutant-p53 cell biology and biochemistry is warranted.</p>
<p>Currently, p53 mutational status is not a contraindicator of MAPK targeted therapy in <em>BRAF<sup>V600E</sup></em> mutant lung cancer<span class="citation" data-cites="RdTFM0c3">(Matsumura et al., 2018)</span>. Patients are often stratified into wild-type or mutant p53 during diagnosis. However, Our data suggest that there are substantial differences between p53-mutants regarding tumor progression and therapeutic resistance. There are complex gene-gene interactions at play that affect therapeutic resistance. Therefore, careful analysis of cooperating genomic events’ contribution to pathway-targeted therapy needs to be tested in preclinical models.</p>
<p>Since reactivation of the MAPK is a common mechanism of dabrafenib plus trametinib resistance, further vertical inhibition of the pathway is considered an attractive therapeutic strategy. Previous research has shown that triple combination (BRAFi+MEKi+ERKi) imposes a more substantial threshold for pathway reactivation and subsequent resistance<span class="citation" data-cites="2NpsUe92">(Xue et al., 2017)</span>. Although more effective at tumor regression and evasion of resistance, vertical inhibition undoubtedly causes severe side effects that can not be tolerated. Therefore, more combination strategies are likely required. Fortunately, exciting therapeutic strategies are emerging from our lab that appears to be efficacious in MAPK-driven tumors, such as the combination of MAPK-pathway inhibitors and autophagy inhibition<span class="citation" data-cites="11XoGPg9g c63Ld4El jKCnCnG4">(Foth and McMahon, 2021; Kinsey et al., 2019; Truong et al., 2020)</span>.</p>
<p>In parallel to my efforts to characterize three p53-mutations in BRAF<sup>V600E</sup> lung cancer, we determined the contribution of a broad range of tumor suppressor proteins to tumor progression. Our work has highlighted the role SETD2 plays in suppressing lung tumorigenesis. However, we currently do not understand the mechanism behind SETD2 tumor suppression. One hypothesis is that SETD2 prevents spurious gene transcription. Thus when SETD2 function is lost, tumor cells adopt multiple cell states due to altered gene expression. The more cell states a tumor cell can access increases the odds of adopting pro-growth characteristics best for the tumor environment. In order to test this hypothesis, we can profile the transcriptomes of single cells that have intact SETD2 and SETD2 deficient tumor cells. Importantly, we must consider alternative transcriptional start sites and alternative splicing. Since p53 dysregulation is at the heart of tumor progression, we would like to know if SETD2 affects p53-stability and target-gene expression. Alternatively, SETD2 loss might circumvent p53 activation through some undescribed mechanism.</p>
<p>Overall, lung adenocarcinoma is a disease that is characterized by a high mutational burden that results in mutational activation of the MAPK pathway and tumorigenesis. Large-scale genomic studies have provided great insight into the molecular underpinning of lung adenocarcinoma. However, due to the complex interactions between gene products, the future of personalized medicine still relies on rigorous testing in preclinical models like genetically-engineered mice. These efforts of this work will aid in the ultimate understanding of cooperating genetic events in lung adenocarcinoma. Ultimately, this dissertation has helped answer several questions regarding BRAF<sup>V600E</sup> tumorigenesis and cooperating events for tumor progression. Additionally, this work has also generated new questions that need to be answered to fully understand how BRAF<sup>V600E</sup> and p53 mutations cooperate to form deadly lung cancers.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-8Gf0ScB2" class="csl-entry" role="doc-biblioentry">
(2014). Comprehensive molecular profiling of lung adenocarcinoma. Nature <em>511</em>, 543–550. <a href="https://doi.org/10.1038/nature13385">https://doi.org/10.1038/nature13385</a>.
</div>
<div id="ref-1CW3EbE5O" class="csl-entry" role="doc-biblioentry">
(2017). AACR Project GENIE: Powering Precision Medicine through an International Consortium. Cancer Discov <em>7</em>, 818–831. <a href="https://doi.org/10.1158/2159-8290.cd-17-0151">https://doi.org/10.1158/2159-8290.cd-17-0151</a>.
</div>
<div id="ref-179MWicwA" class="csl-entry" role="doc-biblioentry">
(2020). A single-cell transcriptomic atlas characterizes ageing tissues in the mouse. Nature <em>583</em>, 590–595. <a href="https://doi.org/10.1038/s41586-020-2496-1">https://doi.org/10.1038/s41586-020-2496-1</a>.
</div>
<div id="ref-JyurLVYi" class="csl-entry" role="doc-biblioentry">
Abegglen, L.M., Caulin, A.F., Chan, A., Lee, K., Robinson, R., Campbell, M.S., Kiso, W.K., Schmitt, D.L., Waddell, P.J., Bhaskara, S., et al. (2015). Potential Mechanisms for Cancer Resistance in Elephants and Comparative Cellular Response to DNA Damage in Humans. JAMA <em>314</em>, 1850–1860. <a href="https://doi.org/10.1001/jama.2015.13134">https://doi.org/10.1001/jama.2015.13134</a>.
</div>
<div id="ref-1HKPvGqty" class="csl-entry" role="doc-biblioentry">
Adorno, M., Cordenonsi, M., Montagner, M., Dupont, S., Wong, C., Hann, B., Solari, A., Bobisse, S., Rondina, M.B., Guzzardo, V., et al. (2009). A Mutant-p53/Smad complex opposes p63 to empower TGFbeta-induced metastasis. Cell <em>137</em>, 87–98. <a href="https://doi.org/10.1016/j.cell.2009.01.039">https://doi.org/10.1016/j.cell.2009.01.039</a>.
</div>
<div id="ref-vsbCfYx4" class="csl-entry" role="doc-biblioentry">
Alanis, D.M., Chang, D.R., Akiyama, H., Krasnow, M.A., and Chen, J. (2014). Two nested developmental waves demarcate a compartment boundary in the mouse lung. Nat Commun <em>5</em>, 3923. <a href="https://doi.org/10.1038/ncomms4923">https://doi.org/10.1038/ncomms4923</a>.
</div>
<div id="ref-e3rkxhDE" class="csl-entry" role="doc-biblioentry">
Alexandrov, L.B., Ju, Y.S., Haase, K., Van Loo, P., Martincorena, I., Nik-Zainal, S., Totoki, Y., Fujimoto, A., Nakagawa, H., Shibata, T., et al. (2016). Mutational signatures associated with tobacco smoking in human cancer. Science <em>354</em>, 618–622. <a href="https://doi.org/10.1126/science.aag0299">https://doi.org/10.1126/science.aag0299</a>.
</div>
<div id="ref-6j21xB75" class="csl-entry" role="doc-biblioentry">
Arteaga, C.L., and Engelman, J.A. (2014). ERBB receptors: from oncogene discovery to basic science to mechanism-based cancer therapeutics. Cancer Cell <em>25</em>, 282–303. <a href="https://doi.org/10.1016/j.ccr.2014.02.025">https://doi.org/10.1016/j.ccr.2014.02.025</a>.
</div>
<div id="ref-HnQw16l7" class="csl-entry" role="doc-biblioentry">
Bartek, J., Iggo, R., Gannon, J., and Lane, D.P. (1990). <a href="https://www.ncbi.nlm.nih.gov/pubmed/1694291">Genetic and immunochemical analysis of mutant p53 in human breast cancer cell lines.</a> Oncogene <em>5</em>, 893–899.
</div>
<div id="ref-EpOVYdFF" class="csl-entry" role="doc-biblioentry">
Bartha, Á., and Győrffy, B. (2021). TNMplot.com: A Web Tool for the Comparison of Gene Expression in Normal, Tumor and Metastatic Tissues. Int J Mol Sci <em>22</em>. <a href="https://doi.org/10.3390/ijms22052622">https://doi.org/10.3390/ijms22052622</a>.
</div>
<div id="ref-1AV7mz5Fc" class="csl-entry" role="doc-biblioentry">
Bhattacharya, S., and Workman, J.L. (2020). Regulation of SETD2 stability is important for the fidelity of H3K36me3 deposition. Epigenetics Chromatin <em>13</em>, 40. <a href="https://doi.org/10.1186/s13072-020-00362-8">https://doi.org/10.1186/s13072-020-00362-8</a>.
</div>
<div id="ref-ju5z9qDz" class="csl-entry" role="doc-biblioentry">
Bouaoun, L., Sonkin, D., Ardin, M., Hollstein, M., Byrnes, G., Zavadil, J., and Olivier, M. (2016). TP53 Variations in Human Cancers: New Lessons from the IARC TP53 Database and Genomics Data. Hum Mutat <em>37</em>, 865–876. <a href="https://doi.org/10.1002/humu.23035">https://doi.org/10.1002/humu.23035</a>.
</div>
<div id="ref-GiCMgc6K" class="csl-entry" role="doc-biblioentry">
Bougeard, G., Sesboüé, R., Baert-Desurmont, S., Vasseur, S., Martin, C., Tinat, J., Brugières, L., Chompret, A., de Paillerets, B.B., Stoppa-Lyonnet, D., et al. (2008). Molecular basis of the Li-Fraumeni syndrome: an update from the French LFS families. J Med Genet <em>45</em>, 535–538. <a href="https://doi.org/10.1136/jmg.2008.057570">https://doi.org/10.1136/jmg.2008.057570</a>.
</div>
<div id="ref-KlPsMirp" class="csl-entry" role="doc-biblioentry">
Boutelle, A.M., and Attardi, L.D. (2021). p53 and Tumor Suppression: It Takes a Network. Trends Cell Biol <em>31</em>, 298–310. <a href="https://doi.org/10.1016/j.tcb.2020.12.011">https://doi.org/10.1016/j.tcb.2020.12.011</a>.
</div>
<div id="ref-v2CDt1N5" class="csl-entry" role="doc-biblioentry">
Bracht, J.W.P., Karachaliou, N., Bivona, T., Lanman, R.B., Faull, I., Nagy, R.J., Drozdowskyj, A., Berenguer, J., Fernandez-Bruno, M., Molina-Vila, M.A., et al. (2019). Cancers (Basel) <em>11</em>. <a href="https://doi.org/10.3390/cancers11091381">https://doi.org/10.3390/cancers11091381</a>.
</div>
<div id="ref-6jSWGuKW" class="csl-entry" role="doc-biblioentry">
Cai, H., Chew, S.K., Li, C., Tsai, M.K., Andrejka, L., Murray, C.W., Hughes, N.W., Shuldiner, E.G., Ashkin, E.L., Tang, R., et al. (2021). A Functional Taxonomy of Tumor Suppression in Oncogenic KRAS-Driven Lung Cancer. Cancer Discov <em>11</em>, 1754–1773. <a href="https://doi.org/10.1158/2159-8290.cd-20-1325">https://doi.org/10.1158/2159-8290.cd-20-1325</a>.
</div>
<div id="ref-xy4r2sTu" class="csl-entry" role="doc-biblioentry">
Campbell, J.D., Alexandrov, A., Kim, J., Wala, J., Berger, A.H., Pedamallu, C.S., Shukla, S.A., Guo, G., Brooks, A.N., Murray, B.A., et al. (2016). Distinct patterns of somatic genome alterations in lung adenocarcinomas and squamous cell carcinomas. Nat Genet <em>48</em>, 607–616. <a href="https://doi.org/10.1038/ng.3564">https://doi.org/10.1038/ng.3564</a>.
</div>
<div id="ref-Ph0Z7rTW" class="csl-entry" role="doc-biblioentry">
Cerami, E., Gao, J., Dogrusoz, U., Gross, B.E., Sumer, S.O., Aksoy, B.A., Jacobsen, A., Byrne, C.J., Heuer, M.L., Larsson, E., et al. (2012). The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov <em>2</em>, 401–404. <a href="https://doi.org/10.1158/2159-8290.cd-12-0095">https://doi.org/10.1158/2159-8290.cd-12-0095</a>.
</div>
<div id="ref-CIReXnPL" class="csl-entry" role="doc-biblioentry">
Chan, W.M., Siu, W.Y., Lau, A., and Poon, R.Y.C. (2004). How many mutant p53 molecules are needed to inactivate a tetramer? Mol Cell Biol <em>24</em>, 3536–3551. <a href="https://doi.org/10.1128/mcb.24.8.3536-3551.2004">https://doi.org/10.1128/mcb.24.8.3536-3551.2004</a>.
</div>
<div id="ref-1EwtuJJw6" class="csl-entry" role="doc-biblioentry">
Chiou, S.-H., Winters, I.P., Wang, J., Naranjo, S., Dudgeon, C., Tamburini, F.B., Brady, J.J., Yang, D., Grüner, B.M., Chuang, C.-H., et al. (2015). Pancreatic cancer modeling using retrograde viral vector delivery and in vivo CRISPR/Cas9-mediated somatic genome editing. Genes Dev <em>29</em>, 1576–1585. <a href="https://doi.org/10.1101/gad.264861.115">https://doi.org/10.1101/gad.264861.115</a>.
</div>
<div id="ref-G5wVcRrJ" class="csl-entry" role="doc-biblioentry">
Christophorou, M.A., Ringshausen, I., Finch, A.J., Swigart, L.B., and Evan, G.I. (2006). The pathological response to DNA damage does not contribute to p53-mediated tumour suppression. Nature <em>443</em>, 214–217. <a href="https://doi.org/10.1038/nature05077">https://doi.org/10.1038/nature05077</a>.
</div>
<div id="ref-OK8s0ke5" class="csl-entry" role="doc-biblioentry">
Cicchini, M., Buza, E.L., Sagal, K.M., Gudiel, A.A., Durham, A.C., and Feldser, D.M. (2017). Context-Dependent Effects of Amplified MAPK Signaling during Lung Adenocarcinoma Initiation and Progression. Cell Rep <em>18</em>, 1958–1969. <a href="https://doi.org/10.1016/j.celrep.2017.01.069">https://doi.org/10.1016/j.celrep.2017.01.069</a>.
</div>
<div id="ref-wMtu748b" class="csl-entry" role="doc-biblioentry">
Corcoran, R.B., Ebi, H., Turke, A.B., Coffee, E.M., Nishino, M., Cogdill, A.P., Brown, R.D., Della Pelle, P., Dias-Santagata, D., Hung, K.E., et al. (2012). EGFR-mediated re-activation of MAPK signaling contributes to insensitivity of BRAF mutant colorectal cancers to RAF inhibition with vemurafenib. Cancer Discov <em>2</em>, 227–235. <a href="https://doi.org/10.1158/2159-8290.cd-11-0341">https://doi.org/10.1158/2159-8290.cd-11-0341</a>.
</div>
<div id="ref-dTvhNsG9" class="csl-entry" role="doc-biblioentry">
Dankort, D., Filenova, E., Collado, M., Serrano, M., Jones, K., and McMahon, M. (2007). A new mouse model to explore the initiation, progression, and therapy of BRAFV600E-induced lung tumors. Genes Dev <em>21</em>, 379–384. <a href="https://doi.org/10.1101/gad.1516407">https://doi.org/10.1101/gad.1516407</a>.
</div>
<div id="ref-1F86sLj5s" class="csl-entry" role="doc-biblioentry">
de Groot, P., and Munden, R.F. (2012). Lung cancer epidemiology, risk factors, and prevention. Radiol Clin North Am <em>50</em>, 863–876. <a href="https://doi.org/10.1016/j.rcl.2012.06.006">https://doi.org/10.1016/j.rcl.2012.06.006</a>.
</div>
<div id="ref-n08HchE9" class="csl-entry" role="doc-biblioentry">
Desai, T.J., Brownfield, D.G., and Krasnow, M.A. (2014). Alveolar progenitor and stem cells in lung development, renewal and cancer. Nature <em>507</em>, 190–194. <a href="https://doi.org/10.1038/nature12930">https://doi.org/10.1038/nature12930</a>.
</div>
<div id="ref-WyV78aHe" class="csl-entry" role="doc-biblioentry">
Deuker, M.M., Marsh Durban, V., Phillips, W.A., and McMahon, M. (2014). PI3'-kinase inhibition forestalls the onset of MEK1/2 inhibitor resistance in BRAF-mutated melanoma. Cancer Discov <em>5</em>, 143–153. <a href="https://doi.org/10.1158/2159-8290.cd-14-0856">https://doi.org/10.1158/2159-8290.cd-14-0856</a>.
</div>
<div id="ref-to1qRTTV" class="csl-entry" role="doc-biblioentry">
Di Cello, F., Hillion, J., Hristov, A., Wood, L.J., Mukherjee, M., Schuldenfrei, A., Kowalski, J., Bhattacharya, R., Ashfaq, R., and Resar, L.M.S. (2008). HMGA2 participates in transformation in human lung cancer. Mol Cancer Res <em>6</em>, 743–750. <a href="https://doi.org/10.1158/1541-7786.mcr-07-0095">https://doi.org/10.1158/1541-7786.mcr-07-0095</a>.
</div>
<div id="ref-kxXMFjnc" class="csl-entry" role="doc-biblioentry">
Donehower, L.A., Harvey, M., Slagle, B.L., McArthur, M.J., Montgomery, C.A., Butel, J.S., and Bradley, A. (1992). Mice deficient for p53 are developmentally normal but susceptible to spontaneous tumours. Nature <em>356</em>, 215–221. <a href="https://doi.org/10.1038/356215a0">https://doi.org/10.1038/356215a0</a>.
</div>
<div id="ref-iZbc8sIr" class="csl-entry" role="doc-biblioentry">
Duggan, M.A., Anderson, W.F., Altekruse, S., Penberthy, L., and Sherman, M.E. (2016). The Surveillance, Epidemiology, and End Results (SEER) Program and Pathology: Toward Strengthening the Critical Relationship. Am J Surg Pathol <em>40</em>, e94–e102. <a href="https://doi.org/10.1097/pas.0000000000000749">https://doi.org/10.1097/pas.0000000000000749</a>.
</div>
<div id="ref-bhNe3oCb" class="csl-entry" role="doc-biblioentry">
DuPage, M., Dooley, A.L., and Jacks, T. (2009). Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase. Nat Protoc <em>4</em>, 1064–1072. <a href="https://doi.org/10.1038/nprot.2009.95">https://doi.org/10.1038/nprot.2009.95</a>.
</div>
<div id="ref-M2kGnbOE" class="csl-entry" role="doc-biblioentry">
Edmunds, J.W., Mahadevan, L.C., and Clayton, A.L. (2007). Dynamic histone H3 methylation during gene induction: HYPB/Setd2 mediates all H3K36 trimethylation. EMBO J <em>27</em>, 406–420. <a href="https://doi.org/10.1038/sj.emboj.7601967">https://doi.org/10.1038/sj.emboj.7601967</a>.
</div>
<div id="ref-IILnKdOc" class="csl-entry" role="doc-biblioentry">
el-Deiry, W.S., Tokino, T., Velculescu, V.E., Levy, D.B., Parsons, R., Trent, J.M., Lin, D., Mercer, W.E., Kinzler, K.W., and Vogelstein, B. (1993). WAF1, a potential mediator of p53 tumor suppression. Cell <em>75</em>, 817–825. <a href="https://doi.org/10.1016/0092-8674(93)90500-p">https://doi.org/10.1016/0092-8674(93)90500-p</a>.
</div>
<div id="ref-CMVwSV1C" class="csl-entry" role="doc-biblioentry">
Feldser, D.M., Kostova, K.K., Winslow, M.M., Taylor, S.E., Cashman, C., Whittaker, C.A., Sanchez-Rivera, F.J., Resnick, R., Bronson, R., Hemann, M.T., et al. (2010). Stage-specific sensitivity to p53 restoration during lung cancer progression. Nature <em>468</em>, 572–575. <a href="https://doi.org/10.1038/nature09535">https://doi.org/10.1038/nature09535</a>.
</div>
<div id="ref-oVtv8JOR" class="csl-entry" role="doc-biblioentry">
Ferone, G., Lee, M.C., Sage, J., and Berns, A. (2020). Cells of origin of lung cancers: lessons from mouse studies. Genes Dev <em>34</em>, 1017–1032. <a href="https://doi.org/10.1101/gad.338228.120">https://doi.org/10.1101/gad.338228.120</a>.
</div>
<div id="ref-11XoGPg9g" class="csl-entry" role="doc-biblioentry">
Foth, M., and McMahon, M. (2021). Autophagy Inhibition in BRAF-Driven Cancers. Cancers (Basel) <em>13</em>. <a href="https://doi.org/10.3390/cancers13143498">https://doi.org/10.3390/cancers13143498</a>.
</div>
<div id="ref-Vbw51s50" class="csl-entry" role="doc-biblioentry">
Gaiddon, C., Lokshin, M., Ahn, J., Zhang, T., and Prives, C. (2001). A subset of tumor-derived mutant forms of p53 down-regulate p63 and p73 through a direct interaction with the p53 core domain. Mol Cell Biol <em>21</em>, 1874–1887. <a href="https://doi.org/10.1128/mcb.21.5.1874-1887.2001">https://doi.org/10.1128/mcb.21.5.1874-1887.2001</a>.
</div>
<div id="ref-gI9Yeagx" class="csl-entry" role="doc-biblioentry">
Gao, J., Aksoy, B.A., Dogrusoz, U., Dresdner, G., Gross, B., Sumer, S.O., Sun, Y., Jacobsen, A., Sinha, R., Larsson, E., et al. (2013). Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci Signal <em>6</em>, pl1. <a href="https://doi.org/10.1126/scisignal.2004088">https://doi.org/10.1126/scisignal.2004088</a>.
</div>
<div id="ref-1C8aTrJLL" class="csl-entry" role="doc-biblioentry">
Gao, X., Dai, M., Li, Q., Wang, Z., Lu, Y., and Song, Z. (2017). HMGA2 regulates lung cancer proliferation and metastasis. Thorac Cancer <em>8</em>, 501–510. <a href="https://doi.org/10.1111/1759-7714.12476">https://doi.org/10.1111/1759-7714.12476</a>.
</div>
<div id="ref-ztWGsqcf" class="csl-entry" role="doc-biblioentry">
Garcia, P.B., and Attardi, L.D. (2014). Illuminating p53 function in cancer with genetically engineered mouse models. Semin Cell Dev Biol <em>27</em>, 74–85. <a href="https://doi.org/10.1016/j.semcdb.2013.12.014">https://doi.org/10.1016/j.semcdb.2013.12.014</a>.
</div>
<div id="ref-1EmZp72ZM" class="csl-entry" role="doc-biblioentry">
Gerlinger, M., Rowan, A.J., Horswell, S., Math, M., Larkin, J., Endesfelder, D., Gronroos, E., Martinez, P., Matthews, N., Stewart, A., et al. (2012). Intratumor heterogeneity and branched evolution revealed by multiregion sequencing. N Engl J Med <em>366</em>, 883–892. <a href="https://doi.org/10.1056/nejmoa1113205">https://doi.org/10.1056/nejmoa1113205</a>.
</div>
<div id="ref-hRW7rjfZ" class="csl-entry" role="doc-biblioentry">
Ghimessy, A., Radeczky, P., Laszlo, V., Hegedus, B., Renyi-Vamos, F., Fillinger, J., Klepetko, W., Lang, C., Dome, B., and Megyesfalvi, Z. (2020). Current therapy of KRAS-mutant lung cancer. Cancer Metastasis Rev <em>39</em>, 1159–1177. <a href="https://doi.org/10.1007/s10555-020-09903-9">https://doi.org/10.1007/s10555-020-09903-9</a>.
</div>
<div id="ref-Pje1Soz4" class="csl-entry" role="doc-biblioentry">
Green, S., Trejo, C.L., and McMahon, M. (2015). PIK3CA(H1047R) Accelerates and Enhances KRAS(G12D)-Driven Lung Tumorigenesis. Cancer Res <em>75</em>, 5378–5391. <a href="https://doi.org/10.1158/0008-5472.can-15-1249">https://doi.org/10.1158/0008-5472.can-15-1249</a>.
</div>
<div id="ref-I9heYgx4" class="csl-entry" role="doc-biblioentry">
Hainaut, P., and Pfeifer, G.P. (2016). Somatic TP53 Mutations in the Era of Genome Sequencing. Cold Spring Harb Perspect Med <em>6</em>. <a href="https://doi.org/10.1101/cshperspect.a026179">https://doi.org/10.1101/cshperspect.a026179</a>.
</div>
<div id="ref-kgaUYMuT" class="csl-entry" role="doc-biblioentry">
Hanahan, D. (2022). Hallmarks of Cancer: New Dimensions. Cancer Discov <em>12</em>, 31–46. <a href="https://doi.org/10.1158/2159-8290.cd-21-1059">https://doi.org/10.1158/2159-8290.cd-21-1059</a>.
</div>
<div id="ref-4H4Kjv9w" class="csl-entry" role="doc-biblioentry">
Hanahan, D., and Weinberg, R.A. (2011). Hallmarks of cancer: the next generation. Cell <em>144</em>, 646–674. <a href="https://doi.org/10.1016/j.cell.2011.02.013">https://doi.org/10.1016/j.cell.2011.02.013</a>.
</div>
<div id="ref-19duX0kT0" class="csl-entry" role="doc-biblioentry">
Herbst, R.S., Heymach, J.V., and Lippman, S.M. (2008). Lung cancer. N Engl J Med <em>359</em>, 1367–1380. <a href="https://doi.org/10.1056/nejmra0802714">https://doi.org/10.1056/nejmra0802714</a>.
</div>
<div id="ref-PAjnX7uA" class="csl-entry" role="doc-biblioentry">
Hoadley, K.A., Yau, C., Wolf, D.M., Cherniack, A.D., Tamborero, D., Ng, S., Leiserson, M.D.M., Niu, B., McLellan, M.D., Uzunangelov, V., et al. (2014). Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. Cell <em>158</em>, 929–944. <a href="https://doi.org/10.1016/j.cell.2014.06.049">https://doi.org/10.1016/j.cell.2014.06.049</a>.
</div>
<div id="ref-15BsPbLof" class="csl-entry" role="doc-biblioentry">
Holderfield, M., Deuker, M.M., McCormick, F., and McMahon, M. (2014). Targeting RAF kinases for cancer therapy: BRAF-mutated melanoma and beyond. Nat Rev Cancer <em>14</em>, 455–467. <a href="https://doi.org/10.1038/nrc3760">https://doi.org/10.1038/nrc3760</a>.
</div>
<div id="ref-17uu2v5lM" class="csl-entry" role="doc-biblioentry">
Huang, S., Benavente, S., Armstrong, E.A., Li, C., Wheeler, D.L., and Harari, P.M. (2011). p53 modulates acquired resistance to EGFR inhibitors and radiation. Cancer Res <em>71</em>, 7071–7079. <a href="https://doi.org/10.1158/0008-5472.can-11-0128">https://doi.org/10.1158/0008-5472.can-11-0128</a>.
</div>
<div id="ref-bLXbRmvx" class="csl-entry" role="doc-biblioentry">
Iggo, R., Gatter, K., Bartek, J., Lane, D., and Harris, A.L. (1990). Increased expression of mutant forms of p53 oncogene in primary lung cancer. Lancet <em>335</em>, 675–679. <a href="https://doi.org/10.1016/0140-6736(90)90801-b">https://doi.org/10.1016/0140-6736(90)90801-b</a>.
</div>
<div id="ref-HhXx5lwJ" class="csl-entry" role="doc-biblioentry">
Ingram, K., Samson, S.C., Zewdu, R., Zitnay, R.G., Snyder, E.L., and Mendoza, M.C. (2021). NKX2-1 controls lung cancer progression by inducing DUSP6 to dampen ERK activity. Oncogene <em>41</em>, 293–300. <a href="https://doi.org/10.1038/s41388-021-02076-x">https://doi.org/10.1038/s41388-021-02076-x</a>.
</div>
<div id="ref-cD4M6ONJ" class="csl-entry" role="doc-biblioentry">
Jackson, E.L., Willis, N., Mercer, K., Bronson, R.T., Crowley, D., Montoya, R., Jacks, T., and Tuveson, D.A. (2001). Analysis of lung tumor initiation and progression using conditional expression of oncogenic K-ras. Genes Dev <em>15</em>, 3243–3248. <a href="https://doi.org/10.1101/gad.943001">https://doi.org/10.1101/gad.943001</a>.
</div>
<div id="ref-KKkLA3TK" class="csl-entry" role="doc-biblioentry">
Jackson, E.L., Olive, K.P., Tuveson, D.A., Bronson, R., Crowley, D., Brown, M., and Jacks, T. (2005). The differential effects of mutant p53 alleles on advanced murine lung cancer. Cancer Res <em>65</em>, 10280–10288. <a href="https://doi.org/10.1158/0008-5472.can-05-2193">https://doi.org/10.1158/0008-5472.can-05-2193</a>.
</div>
<div id="ref-kGt16xDw" class="csl-entry" role="doc-biblioentry">
Jonkers, J., Meuwissen, R., van der Gulden, H., Peterse, H., van der Valk, M., and Berns, A. (2001). Synergistic tumor suppressor activity of BRCA2 and p53 in a conditional mouse model for breast cancer. Nat Genet <em>29</em>, 418–425. <a href="https://doi.org/10.1038/ng747">https://doi.org/10.1038/ng747</a>.
</div>
<div id="ref-Djv3zv5n" class="csl-entry" role="doc-biblioentry">
Juan, J., Muraguchi, T., Iezza, G., Sears, R.C., and McMahon, M. (2014). Diminished WNT -&gt; β-catenin -&gt; c-MYC signaling is a barrier for malignant progression of BRAFV600E-induced lung tumors. Genes Dev <em>28</em>, 561–575. <a href="https://doi.org/10.1101/gad.233627.113">https://doi.org/10.1101/gad.233627.113</a>.
</div>
<div id="ref-okfbKJaD" class="csl-entry" role="doc-biblioentry">
Junttila, M.R., Karnezis, A.N., Garcia, D., Madriles, F., Kortlever, R.M., Rostker, F., Brown Swigart, L., Pham, D.M., Seo, Y., Evan, G.I., et al. (2010). Selective activation of p53-mediated tumour suppression in high-grade tumours. Nature <em>468</em>, 567–571. <a href="https://doi.org/10.1038/nature09526">https://doi.org/10.1038/nature09526</a>.
</div>
<div id="ref-12CuLfxxd" class="csl-entry" role="doc-biblioentry">
Kandoth, C., McLellan, M.D., Vandin, F., Ye, K., Niu, B., Lu, C., Xie, M., Zhang, Q., McMichael, J.F., Wyczalkowski, M.A., et al. (2013). Mutational landscape and significance across 12 major cancer types. Nature <em>502</em>, 333–339. <a href="https://doi.org/10.1038/nature12634">https://doi.org/10.1038/nature12634</a>.
</div>
<div id="ref-9P0mUQ2t" class="csl-entry" role="doc-biblioentry">
Kemp, C.J., Wheldon, T., and Balmain, A. (1994). p53-deficient mice are extremely susceptible to radiation-induced tumorigenesis. Nat Genet <em>8</em>, 66–69. <a href="https://doi.org/10.1038/ng0994-66">https://doi.org/10.1038/ng0994-66</a>.
</div>
<div id="ref-o8jKDrsB" class="csl-entry" role="doc-biblioentry">
Kim, M.P., Li, X., Deng, J., Zhang, Y., Dai, B., Allton, K.L., Hughes, T.G., Siangco, C., Augustine, J.J., Kang, Y., et al. (2021). Oncogenic. Cancer Discov <em>11</em>, 2094–2111. <a href="https://doi.org/10.1158/2159-8290.cd-20-1228">https://doi.org/10.1158/2159-8290.cd-20-1228</a>.
</div>
<div id="ref-c63Ld4El" class="csl-entry" role="doc-biblioentry">
Kinsey, C.G., Camolotto, S.A., Boespflug, A.M., Guillen, K.P., Foth, M., Truong, A., Schuman, S.S., Shea, J.E., Seipp, M.T., Yap, J.T., et al. (2019). Protective autophagy elicited by RAF→MEK→ERK inhibition suggests a treatment strategy for RAS-driven cancers. Nat Med <em>25</em>, 620–627. <a href="https://doi.org/10.1038/s41591-019-0367-9">https://doi.org/10.1038/s41591-019-0367-9</a>.
</div>
<div id="ref-B0ClcVtn" class="csl-entry" role="doc-biblioentry">
Klemke, L., Fehlau, C.F., Winkler, N., Toboll, F., Singh, S.K., Moll, U.M., and Schulz-Heddergott, R. (2021). The Gain-of-Function p53 R248W Mutant Promotes Migration by STAT3 Deregulation in Human Pancreatic Cancer Cells. Front Oncol <em>11</em>, 642603. <a href="https://doi.org/10.3389/fonc.2021.642603">https://doi.org/10.3389/fonc.2021.642603</a>.
</div>
<div id="ref-RCmUro9f" class="csl-entry" role="doc-biblioentry">
LaFave, L.M., Kartha, V.K., Ma, S., Meli, K., Del Priore, I., Lareau, C., Naranjo, S., Westcott, P.M.K., Duarte, F.M., Sankar, V., et al. (2020). Epigenomic State Transitions Characterize Tumor Progression in Mouse Lung Adenocarcinoma. Cancer Cell <em>38</em>, 212–228.e13. <a href="https://doi.org/10.1016/j.ccell.2020.06.006">https://doi.org/10.1016/j.ccell.2020.06.006</a>.
</div>
<div id="ref-68clBYF4" class="csl-entry" role="doc-biblioentry">
Lánczky, A., and Győrffy, B. (2021). Web-Based Survival Analysis Tool Tailored for Medical Research (KMplot): Development and Implementation. J Med Internet Res <em>23</em>, e27633. <a href="https://doi.org/10.2196/27633">https://doi.org/10.2196/27633</a>.
</div>
<div id="ref-V4ONQpBd" class="csl-entry" role="doc-biblioentry">
Lang, G.A., Iwakuma, T., Suh, Y.-A., Liu, G., Rao, V.A., Parant, J.M., Valentin-Vega, Y.A., Terzian, T., Caldwell, L.C., Strong, L.C., et al. (2004). Gain of function of a p53 hot spot mutation in a mouse model of Li-Fraumeni syndrome. Cell <em>119</em>, 861–872. <a href="https://doi.org/10.1016/j.cell.2004.11.006">https://doi.org/10.1016/j.cell.2004.11.006</a>.
</div>
<div id="ref-qbfESpXa" class="csl-entry" role="doc-biblioentry">
Laptenko, O., and Prives, C. (2006). Transcriptional regulation by p53: one protein, many possibilities. Cell Death Differ <em>13</em>, 951–961. <a href="https://doi.org/10.1038/sj.cdd.4401916">https://doi.org/10.1038/sj.cdd.4401916</a>.
</div>
<div id="ref-13QSbJkMk" class="csl-entry" role="doc-biblioentry">
Levine, A.J. (2020). p53: 800 million years of evolution and 40 years of discovery. Nat Rev Cancer <em>20</em>, 471–480. <a href="https://doi.org/10.1038/s41568-020-0262-1">https://doi.org/10.1038/s41568-020-0262-1</a>.
</div>
<div id="ref-SWNWMYmm" class="csl-entry" role="doc-biblioentry">
Levine, A.J., and Oren, M. (2009). The first 30 years of p53: growing ever more complex. Nat Rev Cancer <em>9</em>, 749–758. <a href="https://doi.org/10.1038/nrc2723">https://doi.org/10.1038/nrc2723</a>.
</div>
<div id="ref-TYj65wKz" class="csl-entry" role="doc-biblioentry">
Li, F.P., Fraumeni, J.F., Mulvihill, J.J., Blattner, W.A., Dreyfus, M.G., Tucker, M.A., and Miller, R.W. (1988). <a href="https://www.ncbi.nlm.nih.gov/pubmed/3409256">A cancer family syndrome in twenty-four kindreds.</a> Cancer Res <em>48</em>, 5358–5362.
</div>
<div id="ref-WiSGMfl3" class="csl-entry" role="doc-biblioentry">
Lin, L., Asthana, S., Chan, E., Bandyopadhyay, S., Martins, M.M., Olivas, V., Yan, J.J., Pham, L., Wang, M.M., Bollag, G., et al. (2014). Mapping the molecular determinants of BRAF oncogene dependence in human lung cancer. Proc Natl Acad Sci U S A <em>111</em>, E748–57. <a href="https://doi.org/10.1073/pnas.1320956111">https://doi.org/10.1073/pnas.1320956111</a>.
</div>
<div id="ref-1FKCm8fuC" class="csl-entry" role="doc-biblioentry">
Linzer, D.I., and Levine, A.J. (1979). Characterization of a 54K dalton cellular SV40 tumor antigen present in SV40-transformed cells and uninfected embryonal carcinoma cells. Cell <em>17</em>, 43–52. <a href="https://doi.org/10.1016/0092-8674(79)90293-9">https://doi.org/10.1016/0092-8674(79)90293-9</a>.
</div>
<div id="ref-ku3fwB6c" class="csl-entry" role="doc-biblioentry">
Malhotra, J., Malvezzi, M., Negri, E., La Vecchia, C., and Boffetta, P. (2016). Risk factors for lung cancer worldwide. Eur Respir J <em>48</em>, 889–902. <a href="https://doi.org/10.1183/13993003.00359-2016">https://doi.org/10.1183/13993003.00359-2016</a>.
</div>
<div id="ref-pjuOgKHJ" class="csl-entry" role="doc-biblioentry">
Marjanovic, N.D., Hofree, M., Chan, J.E., Canner, D., Wu, K., Trakala, M., Hartmann, G.G., Smith, O.C., Kim, J.Y., Evans, K.V., et al. (2020). Emergence of a High-Plasticity Cell State during Lung Cancer Evolution. Cancer Cell <em>38</em>, 229–246.e13. <a href="https://doi.org/10.1016/j.ccell.2020.06.012">https://doi.org/10.1016/j.ccell.2020.06.012</a>.
</div>
<div id="ref-YdeeJclQ" class="csl-entry" role="doc-biblioentry">
Martin, E.W., and Sung, M.-H. (2018). Challenges of Decoding Transcription Factor Dynamics in Terms of Gene Regulation. Cells <em>7</em>. <a href="https://doi.org/10.3390/cells7090132">https://doi.org/10.3390/cells7090132</a>.
</div>
<div id="ref-RdTFM0c3" class="csl-entry" role="doc-biblioentry">
Matsumura, Y., Owada-Ozaki, Y., and Suzuki, H. (2018). Significance of testing for. J Thorac Dis <em>10</em>, S4147–S4150. <a href="https://doi.org/10.21037/jtd.2018.10.66">https://doi.org/10.21037/jtd.2018.10.66</a>.
</div>
<div id="ref-SY8r3LQy" class="csl-entry" role="doc-biblioentry">
Matsunaga, M., and Shida, K. (1971). <a href="https://www.ncbi.nlm.nih.gov/pubmed/5130083">[Case of progressive external ophthalmoplegia with hemiatrophia totalis].</a> Naika <em>28</em>, 784–788.
</div>
<div id="ref-btzNHt9A" class="csl-entry" role="doc-biblioentry">
Maynard, A., McCoach, C.E., Rotow, J.K., Harris, L., Haderk, F., Kerr, D.L., Yu, E.A., Schenk, E.L., Tan, W., Zee, A., et al. (2020). Therapy-Induced Evolution of Human Lung Cancer Revealed by Single-Cell RNA Sequencing. Cell <em>182</em>, 1232–1251.e22. <a href="https://doi.org/10.1016/j.cell.2020.07.017">https://doi.org/10.1016/j.cell.2020.07.017</a>.
</div>
<div id="ref-1DxvFFUzE" class="csl-entry" role="doc-biblioentry">
McNeal, A.S., Belote, R.L., Zeng, H., Urquijo, M., Barker, K., Torres, R., Curtin, M., Shain, A.H., Andtbacka, R.H., Holmen, S., et al. (2021). BRAF. Elife <em>10</em>. <a href="https://doi.org/10.7554/elife.70385">https://doi.org/10.7554/elife.70385</a>.
</div>
<div id="ref-OuXi55DW" class="csl-entry" role="doc-biblioentry">
Metzger, R.J., Klein, O.D., Martin, G.R., and Krasnow, M.A. (2008). The branching programme of mouse lung development. Nature <em>453</em>, 745–750. <a href="https://doi.org/10.1038/nature07005">https://doi.org/10.1038/nature07005</a>.
</div>
<div id="ref-Rky3nm5Q" class="csl-entry" role="doc-biblioentry">
Mollaoglu, G., Guthrie, M.R., Böhm, S., Brägelmann, J., Can, I., Ballieu, P.M., Marx, A., George, J., Heinen, C., Chalishazar, M.D., et al. (2017). MYC Drives Progression of Small Cell Lung Cancer to a Variant Neuroendocrine Subtype with Vulnerability to Aurora Kinase Inhibition. Cancer Cell <em>31</em>, 270–285. <a href="https://doi.org/10.1016/j.ccell.2016.12.005">https://doi.org/10.1016/j.ccell.2016.12.005</a>.
</div>
<div id="ref-7H88gN81" class="csl-entry" role="doc-biblioentry">
Mollaoglu, G., Jones, A., Wait, S.J., Mukhopadhyay, A., Jeong, S., Arya, R., Camolotto, S.A., Mosbruger, T.L., Stubben, C.J., Conley, C.J., et al. (2018). The Lineage-Defining Transcription Factors SOX2 and NKX2-1 Determine Lung Cancer Cell Fate and Shape the Tumor Immune Microenvironment. Immunity <em>49</em>, 764–779.e9. <a href="https://doi.org/10.1016/j.immuni.2018.09.020">https://doi.org/10.1016/j.immuni.2018.09.020</a>.
</div>
<div id="ref-cWKMNUBI" class="csl-entry" role="doc-biblioentry">
Mukhopadhyay, A., and Oliver, T.G. (2014). Mighty mouse breakthroughs: a Sox2-driven model for squamous cell lung cancer. Mol Cell Oncol <em>2</em>, e969651. <a href="https://doi.org/10.4161/23723548.2014.969651">https://doi.org/10.4161/23723548.2014.969651</a>.
</div>
<div id="ref-7Yb2p7xx" class="csl-entry" role="doc-biblioentry">
Muzumdar, M.D., Dorans, K.J., Chung, K.M., Robbins, R., Tammela, T., Gocheva, V., Li, C.M.-C., and Jacks, T. (2016). Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers. Nat Commun <em>7</em>, 12685. <a href="https://doi.org/10.1038/ncomms12685">https://doi.org/10.1038/ncomms12685</a>.
</div>
<div id="ref-wDnAOh0T" class="csl-entry" role="doc-biblioentry">
Nichols, L., Saunders, R., and Knollmann, F.D. (2012). Causes of death of patients with lung cancer. Arch Pathol Lab Med <em>136</em>, 1552–1557. <a href="https://doi.org/10.5858/arpa.2011-0521-oa">https://doi.org/10.5858/arpa.2011-0521-oa</a>.
</div>
<div id="ref-Co9gQ0V6" class="csl-entry" role="doc-biblioentry">
Nikitin, A.Y., Alcaraz, A., Anver, M.R., Bronson, R.T., Cardiff, R.D., Dixon, D., Fraire, A.E., Gabrielson, E.W., Gunning, W.T., Haines, D.C., et al. (2004). Classification of proliferative pulmonary lesions of the mouse: recommendations of the mouse models of human cancers consortium. Cancer Res <em>64</em>, 2307–2316. <a href="https://doi.org/10.1158/0008-5472.can-03-3376">https://doi.org/10.1158/0008-5472.can-03-3376</a>.
</div>
<div id="ref-MLY9vR0E" class="csl-entry" role="doc-biblioentry">
Olive, K.P., Tuveson, D.A., Ruhe, Z.C., Yin, B., Willis, N.A., Bronson, R.T., Crowley, D., and Jacks, T. (2004). Mutant p53 gain of function in two mouse models of Li-Fraumeni syndrome. Cell <em>119</em>, 847–860. <a href="https://doi.org/10.1016/j.cell.2004.11.004">https://doi.org/10.1016/j.cell.2004.11.004</a>.
</div>
<div id="ref-1Aj9gKIb1" class="csl-entry" role="doc-biblioentry">
Olivier, M., Hollstein, M., and Hainaut, P. (2010). TP53 mutations in human cancers: origins, consequences, and clinical use. Cold Spring Harb Perspect Biol <em>2</em>, a001008. <a href="https://doi.org/10.1101/cshperspect.a001008">https://doi.org/10.1101/cshperspect.a001008</a>.
</div>
<div id="ref-WVwMP2mo" class="csl-entry" role="doc-biblioentry">
Orth, M., Metzger, P., Gerum, S., Mayerle, J., Schneider, G., Belka, C., Schnurr, M., and Lauber, K. (2019). Pancreatic ductal adenocarcinoma: biological hallmarks, current status, and future perspectives of combined modality treatment approaches. Radiat Oncol <em>14</em>, 141. <a href="https://doi.org/10.1186/s13014-019-1345-6">https://doi.org/10.1186/s13014-019-1345-6</a>.
</div>
<div id="ref-E3yjcNvW" class="csl-entry" role="doc-biblioentry">
Perry, M.E., Piette, J., Zawadzki, J.A., Harvey, D., and Levine, A.J. (1993). The mdm-2 gene is induced in response to UV light in a p53-dependent manner. Proc Natl Acad Sci U S A <em>90</em>, 11623–11627. <a href="https://doi.org/10.1073/pnas.90.24.11623">https://doi.org/10.1073/pnas.90.24.11623</a>.
</div>
<div id="ref-nZOmv1za" class="csl-entry" role="doc-biblioentry">
Pfister, N.T., and Prives, C. (2017). Transcriptional Regulation by Wild-Type and Cancer-Related Mutant Forms of p53. Cold Spring Harb Perspect Med <em>7</em>. <a href="https://doi.org/10.1101/cshperspect.a026054">https://doi.org/10.1101/cshperspect.a026054</a>.
</div>
<div id="ref-BpRBkUVl" class="csl-entry" role="doc-biblioentry">
Planchard, D., Besse, B., Groen, H.J.M., Souquet, P.-J., Quoix, E., Baik, C.S., Barlesi, F., Kim, T.M., Mazieres, J., Novello, S., et al. (2016). Dabrafenib plus trametinib in patients with previously treated BRAF(V600E)-mutant metastatic non-small cell lung cancer: an open-label, multicentre phase 2 trial. Lancet Oncol <em>17</em>, 984–993. <a href="https://doi.org/10.1016/s1470-2045(16)30146-2">https://doi.org/10.1016/s1470-2045(16)30146-2</a>.
</div>
<div id="ref-4S1jzORt" class="csl-entry" role="doc-biblioentry">
Planchard, D., Besse, B., Groen, H.J.M., Hashemi, S.M.S., Mazieres, J., Kim, T.M., Quoix, E., Souquet, P.-J., Barlesi, F., Baik, C., et al. (2021). Phase 2 Study of Dabrafenib Plus Trametinib in Patients With BRAF V600E-Mutant Metastatic NSCLC: Updated 5-Year Survival Rates and Genomic Analysis. J Thorac Oncol <em>17</em>, 103–115. <a href="https://doi.org/10.1016/j.jtho.2021.08.011">https://doi.org/10.1016/j.jtho.2021.08.011</a>.
</div>
<div id="ref-kQzAqqtP" class="csl-entry" role="doc-biblioentry">
Politi, K., Zakowski, M.F., Fan, P.-D., Schonfeld, E.A., Pao, W., and Varmus, H.E. (2006). Lung adenocarcinomas induced in mice by mutant EGF receptors found in human lung cancers respond to a tyrosine kinase inhibitor or to down-regulation of the receptors. Genes Dev <em>20</em>, 1496–1510. <a href="https://doi.org/10.1101/gad.1417406">https://doi.org/10.1101/gad.1417406</a>.
</div>
<div id="ref-1fsz5jxh" class="csl-entry" role="doc-biblioentry">
Powell, S.M., Zilz, N., Beazer-Barclay, Y., Bryan, T.M., Hamilton, S.R., Thibodeau, S.N., Vogelstein, B., and Kinzler, K.W. (1992). APC mutations occur early during colorectal tumorigenesis. Nature <em>359</em>, 235–237. <a href="https://doi.org/10.1038/359235a0">https://doi.org/10.1038/359235a0</a>.
</div>
<div id="ref-pXJPz758" class="csl-entry" role="doc-biblioentry">
Redman-Rivera, L.N., Shaver, T.M., Jin, H., Marshall, C.B., Schafer, J.M., Sheng, Q., Hongo, R.A., Beckermann, K.E., Wheeler, F.C., Lehmann, B.D., et al. (2021). Acquisition of aneuploidy drives mutant p53-associated gain-of-function phenotypes. Nat Commun <em>12</em>, 5184. <a href="https://doi.org/10.1038/s41467-021-25359-z">https://doi.org/10.1038/s41467-021-25359-z</a>.
</div>
<div id="ref-1CvrKB1k3" class="csl-entry" role="doc-biblioentry">
Reita, D., Pabst, L., Pencreach, E., Guérin, E., Dano, L., Rimelen, V., Voegeli, A.-C., Vallat, L., Mascaux, C., and Beau-Faller, M. (2022). Direct Targeting. Cancers (Basel) <em>14</em>. <a href="https://doi.org/10.3390/cancers14051321">https://doi.org/10.3390/cancers14051321</a>.
</div>
<div id="ref-CgOGrXoS" class="csl-entry" role="doc-biblioentry">
Robert, C., Karaszewska, B., Schachter, J., Rutkowski, P., Mackiewicz, A., Stroiakovski, D., Lichinitser, M., Dummer, R., Grange, F., Mortier, L., et al. (2014). Improved overall survival in melanoma with combined dabrafenib and trametinib. N Engl J Med <em>372</em>, 30–39. <a href="https://doi.org/10.1056/nejmoa1412690">https://doi.org/10.1056/nejmoa1412690</a>.
</div>
<div id="ref-sufLyk4G" class="csl-entry" role="doc-biblioentry">
Rogers, Z.N., McFarland, C.D., Winters, I.P., Naranjo, S., Chuang, C.-H., Petrov, D., and Winslow, M.M. (2017). A quantitative and multiplexed approach to uncover the fitness landscape of tumor suppression in vivo. Nat Methods <em>14</em>, 737–742. <a href="https://doi.org/10.1038/nmeth.4297">https://doi.org/10.1038/nmeth.4297</a>.
</div>
<div id="ref-17E8qJFQ0" class="csl-entry" role="doc-biblioentry">
Rogers, Z.N., McFarland, C.D., Winters, I.P., Seoane, J.A., Brady, J.J., Yoon, S., Curtis, C., Petrov, D.A., and Winslow, M.M. (2018). Mapping the in vivo fitness landscape of lung adenocarcinoma tumor suppression in mice. Nat Genet <em>50</em>, 483–486. <a href="https://doi.org/10.1038/s41588-018-0083-2">https://doi.org/10.1038/s41588-018-0083-2</a>.
</div>
<div id="ref-BUDX0W0n" class="csl-entry" role="doc-biblioentry">
Ruan, Y., Wang, H., Chen, B., Wen, H., and Wu, C.-I. (2020). Mutations Beget More Mutations-Rapid Evolution of Mutation Rate in Response to the Risk of Runaway Accumulation. Mol Biol Evol <em>37</em>, 1007–1019. <a href="https://doi.org/10.1093/molbev/msz283">https://doi.org/10.1093/molbev/msz283</a>.
</div>
<div id="ref-1F2tY5YCh" class="csl-entry" role="doc-biblioentry">
Rudin, C.M., Hong, K., and Streit, M. (2013). Molecular characterization of acquired resistance to the BRAF inhibitor dabrafenib in a patient with BRAF-mutant non-small-cell lung cancer. J Thorac Oncol <em>8</em>, e41–2. <a href="https://doi.org/10.1097/jto.0b013e31828bb1b3">https://doi.org/10.1097/jto.0b013e31828bb1b3</a>.
</div>
<div id="ref-PX0Ki15S" class="csl-entry" role="doc-biblioentry">
Rusch, V., Klimstra, D., Venkatraman, E., Oliver, J., Martini, N., Gralla, R., Kris, M., and Dmitrovsky, E. (1995). <a href="https://www.ncbi.nlm.nih.gov/pubmed/7585548">Aberrant p53 expression predicts clinical resistance to cisplatin-based chemotherapy in locally advanced non-small cell lung cancer.</a> Cancer Res <em>55</em>, 5038–5042.
</div>
<div id="ref-UfKKrJ6l" class="csl-entry" role="doc-biblioentry">
Sainz de Aja, J., and Kim, C.F. (2020). May the (Mechanical) Force Be with AT2. Cell <em>180</em>, 20–22. <a href="https://doi.org/10.1016/j.cell.2019.12.020">https://doi.org/10.1016/j.cell.2019.12.020</a>.
</div>
<div id="ref-Ntv29NRu" class="csl-entry" role="doc-biblioentry">
Shai, A., Dankort, D., Juan, J., Green, S., and McMahon, M. (2015). TP53 Silencing Bypasses Growth Arrest of BRAFV600E-Induced Lung Tumor Cells in a Two-Switch Model of Lung Tumorigenesis. Cancer Res <em>75</em>, 3167–3180. <a href="https://doi.org/10.1158/0008-5472.can-14-3701">https://doi.org/10.1158/0008-5472.can-14-3701</a>.
</div>
<div id="ref-gY2NonCA" class="csl-entry" role="doc-biblioentry">
Siegel, R.L., Miller, K.D., Fuchs, H.E., and Jemal, A. (2021). Cancer Statistics, 2021. CA A Cancer J. Clin. <em>71</em>, 7–33. <a href="https://doi.org/10.3322/caac.21654">https://doi.org/10.3322/caac.21654</a>.
</div>
<div id="ref-aazIWGTV" class="csl-entry" role="doc-biblioentry">
Snyder, E.L., Watanabe, H., Magendantz, M., Hoersch, S., Chen, T.A., Wang, D.G., Crowley, D., Whittaker, C.A., Meyerson, M., Kimura, S., et al. (2013). Nkx2-1 represses a latent gastric differentiation program in lung adenocarcinoma. Mol Cell <em>50</em>, 185–199. <a href="https://doi.org/10.1016/j.molcel.2013.02.018">https://doi.org/10.1016/j.molcel.2013.02.018</a>.
</div>
<div id="ref-4BJVCLfM" class="csl-entry" role="doc-biblioentry">
Strong, L.C. (1978). Three mice, two blind. J Surg Oncol <em>10</em>, 353–360. <a href="https://doi.org/10.1002/jso.2930100410">https://doi.org/10.1002/jso.2930100410</a>.
</div>
<div id="ref-dAZ32h3a" class="csl-entry" role="doc-biblioentry">
Sutherland, K.D., Song, J.-Y., Kwon, M.C., Proost, N., Zevenhoven, J., and Berns, A. (2014). Multiple cells-of-origin of mutant K-Ras-induced mouse lung adenocarcinoma. Proc Natl Acad Sci U S A <em>111</em>, 4952–4957. <a href="https://doi.org/10.1073/pnas.1319963111">https://doi.org/10.1073/pnas.1319963111</a>.
</div>
<div id="ref-fXe5uEAl" class="csl-entry" role="doc-biblioentry">
Tabbò, F., Pisano, C., Mazieres, J., Mezquita, L., Nadal, E., Planchard, D., Pradines, A., Santamaria, D., Swalduz, A., Ambrogio, C., et al. (2021). How far we have come targeting BRAF-mutant non-small cell lung cancer (NSCLC). Cancer Treat Rev <em>103</em>, 102335. <a href="https://doi.org/10.1016/j.ctrv.2021.102335">https://doi.org/10.1016/j.ctrv.2021.102335</a>.
</div>
<div id="ref-4TvMmTwj" class="csl-entry" role="doc-biblioentry">
Tammela, T., Sanchez-Rivera, F.J., Cetinbas, N.M., Wu, K., Joshi, N.S., Helenius, K., Park, Y., Azimi, R., Kerper, N.R., Wesselhoeft, R.A., et al. (2017). A Wnt-producing niche drives proliferative potential and progression in lung adenocarcinoma. Nature <em>545</em>, 355–359. <a href="https://doi.org/10.1038/nature22334">https://doi.org/10.1038/nature22334</a>.
</div>
<div id="ref-iF6d3gGU" class="csl-entry" role="doc-biblioentry">
To, M.D., Quigley, D.A., Mao, J.-H., Del Rosario, R., Hsu, J., Hodgson, G., Jacks, T., and Balmain, A. (2011). Progressive genomic instability in the FVB/Kras(LA2) mouse model of lung cancer. Mol Cancer Res <em>9</em>, 1339–1345. <a href="https://doi.org/10.1158/1541-7786.mcr-11-0219">https://doi.org/10.1158/1541-7786.mcr-11-0219</a>.
</div>
<div id="ref-7GTCEP43" class="csl-entry" role="doc-biblioentry">
Tomlinson, I.P., Novelli, M.R., and Bodmer, W.F. (1996). The mutation rate and cancer. Proc Natl Acad Sci U S A <em>93</em>, 14800–14803. <a href="https://doi.org/10.1073/pnas.93.25.14800">https://doi.org/10.1073/pnas.93.25.14800</a>.
</div>
<div id="ref-1HVB1AoBP" class="csl-entry" role="doc-biblioentry">
Trejo, C.L., Green, S., Marsh, V., Collisson, E.A., Iezza, G., Phillips, W.A., and McMahon, M. (2013). Mutationally activated PIK3CA(H1047R) cooperates with BRAF(V600E) to promote lung cancer progression. Cancer Res <em>73</em>, 6448–6461. <a href="https://doi.org/10.1158/0008-5472.can-13-0681">https://doi.org/10.1158/0008-5472.can-13-0681</a>.
</div>
<div id="ref-NLPwvKCP" class="csl-entry" role="doc-biblioentry">
Treutlein, B., Brownfield, D.G., Wu, A.R., Neff, N.F., Mantalas, G.L., Espinoza, F.H., Desai, T.J., Krasnow, M.A., and Quake, S.R. (2014). Reconstructing lineage hierarchies of the distal lung epithelium using single-cell RNA-seq. Nature <em>509</em>, 371–375. <a href="https://doi.org/10.1038/nature13173">https://doi.org/10.1038/nature13173</a>.
</div>
<div id="ref-jKCnCnG4" class="csl-entry" role="doc-biblioentry">
Truong, A., Yoo, J.H., Scherzer, M.T., Sanchez, J.M.S., Dale, K.J., Kinsey, C.G., Richards, J.R., Shin, D., Ghazi, P.C., Onken, M.D., et al. (2020). Chloroquine Sensitizes. Clin Cancer Res <em>26</em>, 6374–6386. <a href="https://doi.org/10.1158/1078-0432.ccr-20-1675">https://doi.org/10.1158/1078-0432.ccr-20-1675</a>.
</div>
<div id="ref-1GkFEmggg" class="csl-entry" role="doc-biblioentry">
van Veen, J.E., Scherzer, M., Boshuizen, J., Chu, M., Liu, A., Landman, A., Green, S., Trejo, C., and McMahon, M. (2019). Mutationally-activated PI3'-kinase-α promotes de-differentiation of lung tumors initiated by the BRAF. Elife <em>8</em>. <a href="https://doi.org/10.7554/elife.43668">https://doi.org/10.7554/elife.43668</a>.
</div>
<div id="ref-14fESvHZW" class="csl-entry" role="doc-biblioentry">
Vogelstein, B., Lane, D., and Levine, A.J. (2000). Surfing the p53 network. Nature <em>408</em>, 307–310. <a href="https://doi.org/10.1038/35042675">https://doi.org/10.1038/35042675</a>.
</div>
<div id="ref-S6qbb6kC" class="csl-entry" role="doc-biblioentry">
Walter, D.M., Venancio, O.S., Buza, E.L., Tobias, J.W., Deshpande, C., Gudiel, A.A., Kim-Kiselak, C., Cicchini, M., Yates, T.J., and Feldser, D.M. (2017). Systematic. Cancer Res <em>77</em>, 1719–1729. <a href="https://doi.org/10.1158/0008-5472.can-16-2159">https://doi.org/10.1158/0008-5472.can-16-2159</a>.
</div>
<div id="ref-2gUfVvoy" class="csl-entry" role="doc-biblioentry">
Wang, X., He, Y., Zhang, Q., Ren, X., and Zhang, Z. (2021). Direct Comparative Analyses of 10X Genomics Chromium and Smart-seq2. Genomics Proteomics Bioinformatics <em>19</em>, 253–266. <a href="https://doi.org/10.1016/j.gpb.2020.02.005">https://doi.org/10.1016/j.gpb.2020.02.005</a>.
</div>
<div id="ref-11g54rA8N" class="csl-entry" role="doc-biblioentry">
Warren, G.W., and Cummings, K.M. (2013). Tobacco and lung cancer: risks, trends, and outcomes in patients with cancer. Am Soc Clin Oncol Educ Book 359–364. <a href="https://doi.org/10.14694/edbook_am.2013.33.359">https://doi.org/10.14694/edbook_am.2013.33.359</a>.
</div>
<div id="ref-jHVcJWec" class="csl-entry" role="doc-biblioentry">
Weissmueller, S., Manchado, E., Saborowski, M., Morris, J.P., Wagenblast, E., Davis, C.A., Moon, S.-H., Pfister, N.T., Tschaharganeh, D.F., Kitzing, T., et al. (2014). Mutant p53 drives pancreatic cancer metastasis through cell-autonomous PDGF receptor β signaling. Cell <em>157</em>, 382–394. <a href="https://doi.org/10.1016/j.cell.2014.01.066">https://doi.org/10.1016/j.cell.2014.01.066</a>.
</div>
<div id="ref-zOzqGxQM" class="csl-entry" role="doc-biblioentry">
Westcott, P.M.K., Halliwill, K.D., To, M.D., Rashid, M., Rust, A.G., Keane, T.M., Delrosario, R., Jen, K.-Y., Gurley, K.E., Kemp, C.J., et al. (2014). The mutational landscapes of genetic and chemical models of Kras-driven lung cancer. Nature <em>517</em>, 489–492. <a href="https://doi.org/10.1038/nature13898">https://doi.org/10.1038/nature13898</a>.
</div>
<div id="ref-CVOIe1Dg" class="csl-entry" role="doc-biblioentry">
Wijnhoven, S.W.P., Zwart, E., Speksnijder, E.N., Beems, R.B., Olive, K.P., Tuveson, D.A., Jonkers, J., Schaap, M.M., van den Berg, J., Jacks, T., et al. (2005). Mice expressing a mammary gland-specific R270H mutation in the p53 tumor suppressor gene mimic human breast cancer development. Cancer Res <em>65</em>, 8166–8173. <a href="https://doi.org/10.1158/0008-5472.can-05-1650">https://doi.org/10.1158/0008-5472.can-05-1650</a>.
</div>
<div id="ref-hYWjwiQA" class="csl-entry" role="doc-biblioentry">
Winslow, M.M., Dayton, T.L., Verhaak, R.G.W., Kim-Kiselak, C., Snyder, E.L., Feldser, D.M., Hubbard, D.D., DuPage, M.J., Whittaker, C.A., Hoersch, S., et al. (2011). Suppression of lung adenocarcinoma progression by Nkx2-1. Nature <em>473</em>, 101–104. <a href="https://doi.org/10.1038/nature09881">https://doi.org/10.1038/nature09881</a>.
</div>
<div id="ref-2gOIsLdX" class="csl-entry" role="doc-biblioentry">
Winters, I.P., Chiou, S.-H., Paulk, N.K., McFarland, C.D., Lalgudi, P.V., Ma, R.K., Lisowski, L., Connolly, A.J., Petrov, D.A., Kay, M.A., et al. (2017). Multiplexed in vivo homology-directed repair and tumor barcoding enables parallel quantification of Kras variant oncogenicity. Nat Commun <em>8</em>, 2053. <a href="https://doi.org/10.1038/s41467-017-01519-y">https://doi.org/10.1038/s41467-017-01519-y</a>.
</div>
<div id="ref-tR3WrnP9" class="csl-entry" role="doc-biblioentry">
Woodard, G.A., Jones, K.D., and Jablons, D.M. Lung Cancer Staging and Prognosis. Cancer Treat Res <em>170</em>, 47–75. <a href="https://doi.org/10.1007/978-3-319-40389-2_3">https://doi.org/10.1007/978-3-319-40389-2_3</a>.
</div>
<div id="ref-qiDMl6l6" class="csl-entry" role="doc-biblioentry">
Woods, D., Parry, D., Cherwinski, H., Bosch, E., Lees, E., and McMahon, M. (1997). Raf-induced proliferation or cell cycle arrest is determined by the level of Raf activity with arrest mediated by p21Cip1. Mol Cell Biol <em>17</em>, 5598–5611. <a href="https://doi.org/10.1128/mcb.17.9.5598">https://doi.org/10.1128/mcb.17.9.5598</a>.
</div>
<div id="ref-Mb1Q4pdk" class="csl-entry" role="doc-biblioentry">
Wu, X., Bayle, J.H., Olson, D., and Levine, A.J. (1993). The p53-mdm-2 autoregulatory feedback loop. Genes Dev <em>7</em>, 1126–1132. <a href="https://doi.org/10.1101/gad.7.7a.1126">https://doi.org/10.1101/gad.7.7a.1126</a>.
</div>
<div id="ref-FfxdQ0tV" class="csl-entry" role="doc-biblioentry">
Xue, Y., Hou, S., Ji, H., and Han, X. (2016). Evolution from genetics to phenotype: reinterpretation of NSCLC plasticity, heterogeneity, and drug resistance. Protein Cell <em>8</em>, 178–190. <a href="https://doi.org/10.1007/s13238-016-0330-1">https://doi.org/10.1007/s13238-016-0330-1</a>.
</div>
<div id="ref-2NpsUe92" class="csl-entry" role="doc-biblioentry">
Xue, Y., Martelotto, L., Baslan, T., Vides, A., Solomon, M., Mai, T.T., Chaudhary, N., Riely, G.J., Li, B.T., Scott, K., et al. (2017). An approach to suppress the evolution of resistance in BRAF. Nat Med <em>23</em>, 929–937. <a href="https://doi.org/10.1038/nm.4369">https://doi.org/10.1038/nm.4369</a>.
</div>
<div id="ref-NqyqCnIL" class="csl-entry" role="doc-biblioentry">
Yaeger, R., and Corcoran, R.B. (2019). Targeting Alterations in the RAF-MEK Pathway. Cancer Discov <em>9</em>, 329–341. <a href="https://doi.org/10.1158/2159-8290.cd-18-1321">https://doi.org/10.1158/2159-8290.cd-18-1321</a>.
</div>
<div id="ref-3xXZMpos" class="csl-entry" role="doc-biblioentry">
Yoshida, K., Gowers, K.H.C., Lee-Six, H., Chandrasekharan, D.P., Coorens, T., Maughan, E.F., Beal, K., Menzies, A., Millar, F.R., Anderson, E., et al. (2020). Tobacco smoking and somatic mutations in human bronchial epithelium. Nature <em>578</em>, 266–272. <a href="https://doi.org/10.1038/s41586-020-1961-1">https://doi.org/10.1038/s41586-020-1961-1</a>.
</div>
<div id="ref-1FBeCW3xl" class="csl-entry" role="doc-biblioentry">
Zafra, M.P., Parsons, M.J., Kim, J., Alonso-Curbelo, D., Goswami, S., Schatoff, E.M., Han, T., Katti, A., Fernandez, M.T.C., Wilkinson, J.E., et al. (2020). An. Cancer Discov <em>10</em>, 1654–1671. <a href="https://doi.org/10.1158/2159-8290.cd-20-0442">https://doi.org/10.1158/2159-8290.cd-20-0442</a>.
</div>
<div id="ref-u4tRgATP" class="csl-entry" role="doc-biblioentry">
Zewdu, R., Mehrabad, E.M., Ingram, K., Fang, P., Gillis, K.L., Camolotto, S.A., Orstad, G., Jones, A., Mendoza, M.C., Spike, B.T., et al. (2021). An NKX2-1/ERK/WNT feedback loop modulates gastric identity and response to targeted therapy in lung adenocarcinoma. Elife <em>10</em>. <a href="https://doi.org/10.7554/elife.66788">https://doi.org/10.7554/elife.66788</a>.
</div>
<div id="ref-kitgYOkt" class="csl-entry" role="doc-biblioentry">
Zhang, Y., Xiong, S., Liu, B., Pant, V., Celii, F., Chau, G., Elizondo-Fraire, A.C., Yang, P., You, M.J., El-Naggar, A.K., et al. (2018). Somatic Trp53 mutations differentially drive breast cancer and evolution of metastases. Nat Commun <em>9</em>, 3953. <a href="https://doi.org/10.1038/s41467-018-06146-9">https://doi.org/10.1038/s41467-018-06146-9</a>.
</div>
<div id="ref-bZo8j50s" class="csl-entry" role="doc-biblioentry">
Zheng, Q., Peacock, D.M., and Shokat, K.M. (2022). Drugging the Next Undruggable KRAS Allele-Gly12Asp. J Med Chem <em>65</em>, 3119–3122. <a href="https://doi.org/10.1021/acs.jmedchem.2c00099">https://doi.org/10.1021/acs.jmedchem.2c00099</a>.
</div>
<div id="ref-N2yNjRyX" class="csl-entry" role="doc-biblioentry">
Zhu, J., Woods, D., McMahon, M., and Bishop, J.M. (1998). Senescence of human fibroblasts induced by oncogenic Raf. Genes Dev <em>12</em>, 2997–3007. <a href="https://doi.org/10.1101/gad.12.19.2997">https://doi.org/10.1101/gad.12.19.2997</a>.
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
